{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Sanity checks for evaluating AIP execution on the RGP CAGI sample set."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import re\n",
    "import itertools\n",
    "\n",
    "from reanalysis.utils import read_json_from_path\n",
    "from cpg_utils import to_path\n",
    "\n",
    "from peddy import Ped"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Convenience functions.\n",
    "\n",
    "def hail_az_to_https(hail_az: str) -> str:\n",
    "    \"\"\"converts a hail-az prefixed Azure storage path to a standard https prefixed path.\"\"\"\n",
    "    return re.sub(r\"^hail-az://(?P<container>.*?)/\", \"https://\\g<container>.blob.core.windows.net/\", hail_az)\n",
    "\n",
    "def https_to_hail_az(https: str) -> str:\n",
    "    \"\"\"converts a https prefixed Azure storage path to a hail-az prefixed path.\"\"\"\n",
    "    return re.sub(r\"^https://(?P<container>.*?).blob.core.windows.net/\", \"hail-az://\\g<container>/\", https)\n",
    "\n",
    "def family_from_sid(sid: str) -> str:\n",
    "    \"\"\"Converts and RGP sample_id to the corresponding family_id.\"\"\"\n",
    "    return '_'.join(sid.split('_')[:2])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Relevant paths, https formatted.\n",
    "\n",
    "AIP_RESULTS_PATH = 'https://raregen001sa.blob.core.windows.net/test-analysis/reanalysis_train/2023-01-31/summary_results.json'\n",
    "COMPARISON_RESULTS_PATH = 'https://raregen001sa.blob.core.windows.net/test/reanalysis_train/comparison/2023-02-01/comparison_result_match_summary.json'\n",
    "ANSWER_KEY_PATH = 'https://raregen001sa.blob.core.windows.net/test/inputs/rgp/CAGI6_RGP Training Set Key.xlsx'\n",
    "PEDIGREE_PATH = 'https://raregen001sa.blob.core.windows.net/test/inputs/rgp/rgp_train.fam'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "EnvironmentCredential.get_token failed: EnvironmentCredential authentication unavailable. Environment variables are not fully configured.\n",
      "ImdsCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "ManagedIdentityCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "EnvironmentCredential.get_token failed: EnvironmentCredential authentication unavailable. Environment variables are not fully configured.\n",
      "ImdsCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "ManagedIdentityCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "EnvironmentCredential.get_token failed: EnvironmentCredential authentication unavailable. Environment variables are not fully configured.\n",
      "ImdsCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "ManagedIdentityCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "EnvironmentCredential.get_token failed: EnvironmentCredential authentication unavailable. Environment variables are not fully configured.\n",
      "ImdsCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n",
      "ManagedIdentityCredential.get_token failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\n"
     ]
    }
   ],
   "source": [
    "# Load and format the input pedigree.\n",
    "with open(to_path(PEDIGREE_PATH), 'r') as f:\n",
    "    pedigree = Ped(f)\n",
    "\n",
    "# Load and format the answer key.\n",
    "with open(to_path(ANSWER_KEY_PATH), 'rb') as f:\n",
    "    answer_key = pd.read_excel(f, dtype=str)\n",
    "\n",
    "answer_key = answer_key.rename(columns={'subject_id (to delete before sharing)': 'sample_id'})\n",
    "answer_key['family_id'] = answer_key.apply(lambda x: family_from_sid(x['sample_id']), axis=1)\n",
    "\n",
    "# Load and format the AIP results.\n",
    "aip_results = read_json_from_path(AIP_RESULTS_PATH)\n",
    "\n",
    "# Load and format the comparison results.\n",
    "comp_results = read_json_from_path(COMPARISON_RESULTS_PATH)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Package                           Version   Editable project location\n",
      "--------------------------------- --------- -------------------------------------------------\n",
      "aiohttp                           3.8.1\n",
      "aiohttp-session                   2.7.0\n",
      "aiosignal                         1.2.0\n",
      "airtable-python-wrapper           0.15.3\n",
      "analysis-runner-ms                0.9.9\n",
      "Arpeggio                          1.10.2\n",
      "asttokens                         2.0.8\n",
      "async-timeout                     4.0.2\n",
      "asyncinit                         0.2.4\n",
      "attrs                             22.1.0\n",
      "automated-interpretation-pipeline 0.1.0     /home/azureuser/automated-interpretation-pipeline\n",
      "avro                              1.11.1\n",
      "azure-common                      1.1.28\n",
      "azure-core                        1.26.0\n",
      "azure-identity                    1.6.0\n",
      "azure-keyvault-secrets            4.6.0\n",
      "azure-mgmt-core                   1.3.2\n",
      "azure-mgmt-storage                20.1.0\n",
      "azure-storage-blob                12.11.0\n",
      "backcall                          0.2.0\n",
      "bokeh                             1.4.0\n",
      "boto3                             1.24.94\n",
      "botocore                          1.27.94\n",
      "bump2version                      1.0.1\n",
      "cachetools                        4.2.4\n",
      "certifi                           2022.9.24\n",
      "cffi                              1.15.1\n",
      "chardet                           4.0.0\n",
      "charset-normalizer                2.1.1\n",
      "click                             8.1.3\n",
      "cloudpathlib                      0.9.0\n",
      "coloredlogs                       15.0.1\n",
      "commonmark                        0.9.1\n",
      "contourpy                         1.0.5\n",
      "cpg-utils-ms                      1.3.3\n",
      "cpg-workflows-ms                  0.1.4\n",
      "cryptography                      38.0.1\n",
      "cycler                            0.11.0\n",
      "Cython                            0.29.32\n",
      "cyvcf2                            0.30.18\n",
      "debugpy                           1.6.3\n",
      "decorator                         4.4.2\n",
      "Deprecated                        1.2.13\n",
      "dill                              0.3.5.1\n",
      "elastic-transport                 8.4.0\n",
      "elasticsearch                     8.5.0\n",
      "entrypoints                       0.4\n",
      "et-xmlfile                        1.1.0\n",
      "executing                         1.1.1\n",
      "flake8                            5.0.4\n",
      "flake8-bugbear                    22.9.23\n",
      "Flask                             2.2.2\n",
      "fonttools                         4.37.4\n",
      "frozendict                        2.3.4\n",
      "frozenlist                        1.3.1\n",
      "fsspec                            2022.11.0\n",
      "gnomad                            0.6.4\n",
      "google-api-core                   1.33.1\n",
      "google-api-python-client          2.10.0\n",
      "google-auth                       2.14.1\n",
      "google-auth-httplib2              0.1.0\n",
      "google-cloud-appengine-logging    1.1.4\n",
      "google-cloud-audit-log            0.2.4\n",
      "google-cloud-core                 2.3.2\n",
      "google-cloud-logging              3.1.1\n",
      "google-cloud-pubsub               2.3.0\n",
      "google-cloud-secret-manager       2.2.0\n",
      "google-cloud-storage              2.6.0\n",
      "google-crc32c                     1.5.0\n",
      "google-resumable-media            2.4.0\n",
      "googleapis-common-protos          1.56.4\n",
      "grpc-google-iam-v1                0.12.4\n",
      "grpcio                            1.50.0\n",
      "grpcio-status                     1.48.2\n",
      "hail                              0.2.105\n",
      "hdbscan                           0.8.29\n",
      "httplib2                          0.20.4\n",
      "humanfriendly                     10.0\n",
      "humanize                          1.0.0\n",
      "hurry.filesize                    0.9\n",
      "idna                              2.10\n",
      "ipykernel                         6.17.1\n",
      "ipython                           8.5.0\n",
      "ipywidgets                        8.0.2\n",
      "isodate                           0.6.1\n",
      "itsdangerous                      2.1.2\n",
      "janus                             1.0.0\n",
      "jedi                              0.18.1\n",
      "Jinja2                            3.0.3\n",
      "jmespath                          1.0.1\n",
      "joblib                            1.2.0\n",
      "jupyter_client                    7.4.7\n",
      "jupyter_core                      5.0.0\n",
      "jupyterlab-widgets                3.0.3\n",
      "kiwisolver                        1.4.4\n",
      "kubernetes                        24.2.0\n",
      "libcst                            0.4.7\n",
      "MarkupSafe                        2.1.1\n",
      "matplotlib                        3.6.0\n",
      "matplotlib-inline                 0.1.6\n",
      "mccabe                            0.7.0\n",
      "msal                              1.20.0\n",
      "msal-extensions                   0.3.1\n",
      "msrest                            0.7.1\n",
      "multidict                         6.0.2\n",
      "mypy-extensions                   0.4.3\n",
      "nest-asyncio                      1.5.4\n",
      "networkx                          2.8.3\n",
      "numpy                             1.22.4\n",
      "oauthlib                          3.2.2\n",
      "obonet                            0.3.0\n",
      "openpyxl                          3.1.0\n",
      "orjson                            3.6.4\n",
      "packaging                         21.3\n",
      "pandas                            1.4.3\n",
      "parsimonious                      0.8.1\n",
      "parso                             0.8.3\n",
      "parver                            0.3.1\n",
      "peddy                             0.4.8\n",
      "pexpect                           4.8.0\n",
      "pickleshare                       0.7.5\n",
      "Pillow                            9.2.0\n",
      "pip                               22.2.2\n",
      "platformdirs                      2.5.4\n",
      "plotly                            5.10.0\n",
      "portalocker                       2.6.0\n",
      "prompt-toolkit                    3.0.31\n",
      "proto-plus                        1.22.1\n",
      "protobuf                          3.20.2\n",
      "psutil                            5.9.4\n",
      "ptyprocess                        0.7.0\n",
      "pulumi                            3.35.3\n",
      "pulumi-gcp                        6.41.0\n",
      "pure-eval                         0.2.2\n",
      "py4j                              0.10.9\n",
      "pyasn1                            0.4.8\n",
      "pyasn1-modules                    0.2.8\n",
      "pycodestyle                       2.9.1\n",
      "pycparser                         2.21\n",
      "pyflakes                          2.5.0\n",
      "Pygments                          2.13.0\n",
      "PyJWT                             2.6.0\n",
      "pyparsing                         3.0.9\n",
      "pyspark                           3.1.3\n",
      "python-dateutil                   2.8.2\n",
      "python-json-logger                2.0.2\n",
      "pytz                              2022.5\n",
      "PyYAML                            5.4.1\n",
      "pyzmq                             24.0.1\n",
      "requests                          2.25.1\n",
      "requests-oauthlib                 1.3.1\n",
      "rich                              12.6.0\n",
      "rsa                               4.9\n",
      "s3transfer                        0.6.0\n",
      "sample-metadata-ms                0.7.1\n",
      "scikit-learn                      1.1.2\n",
      "scipy                             1.7.3\n",
      "seaborn                           0.12.1\n",
      "semver                            2.13.0\n",
      "seqr-loader                       1.2.5\n",
      "setuptools                        63.4.1\n",
      "six                               1.16.0\n",
      "slack-sdk                         3.19.4\n",
      "slackclient                       2.5.0\n",
      "sortedcontainers                  2.4.0\n",
      "stack-data                        0.5.1\n",
      "tabulate                          0.8.9\n",
      "tenacity                          8.1.0\n",
      "threadpoolctl                     3.1.0\n",
      "toml                              0.10.2\n",
      "toolshed                          0.4.6\n",
      "tornado                           6.2\n",
      "tqdm                              4.64.1\n",
      "traitlets                         5.5.0\n",
      "typing_extensions                 4.4.0\n",
      "typing-inspect                    0.8.0\n",
      "uritemplate                       3.0.1\n",
      "urllib3                           1.26.12\n",
      "uvloop                            0.16.0\n",
      "wcwidth                           0.2.5\n",
      "websocket-client                  1.4.1\n",
      "Werkzeug                          2.2.2\n",
      "wheel                             0.37.1\n",
      "widgetsnbextension                4.0.3\n",
      "wrapt                             1.14.1\n",
      "yarl                              1.8.1\n"
     ]
    }
   ],
   "source": [
    "from comparison.comparison import find_affected_samples, prep_rgp_dataframe, common_format_dataframe\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Here is a list of all discrepancies between the various family_id lists:\n",
      "  key_families-aip_families = {'RGP_74'}\n",
      "  pedigree_families-aip_families = {'RGP_74'}\n"
     ]
    }
   ],
   "source": [
    "# Sanity check 1 - the same set of families is represented in all the files under comparison\n",
    "aip_families = set([family_from_sid(sid) for sid in aip_results['results'].keys()])\n",
    "key_families = set(answer_key['family_id'])\n",
    "pedigree_families = set(pedigree.families)\n",
    "\n",
    "print('Here is a list of all discrepancies between the various family_id lists:')\n",
    "any_found = False\n",
    "for a_name, b_name in itertools.permutations(['aip_families', 'key_families', 'pedigree_families'], 2):\n",
    "    a = locals()[a_name]\n",
    "    b = locals()[b_name]\n",
    "    if diff := a.difference(b):\n",
    "        print(f'  {a_name}-{b_name} = {diff}')\n",
    "        any_found = True\n",
    "if not any_found:\n",
    "    print('  None found')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total number of families: 35\n",
      "Number of samples analyzed: 90\n",
      "Total number of families with solves: 35\n"
     ]
    }
   ],
   "source": [
    "# Cohort summary statistics.\n",
    "\n",
    "print(f'Total number of families: {len(pedigree.families)}')\n",
    "print(f'Number of samples analyzed: {len(list(pedigree.samples()))}')\n",
    "print(f'Total number of families with solves: {len(key_families)}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analyses included the following labels from SEQR: dict_keys(['KGFP', 'T1_NGAP', 'T1_NGFKP', 'T1_PE', 'T1_NMOI', 'T1_KGNP', 'T2_NGAP', 'T2_NGFKP', 'T2_PE', 'T2_PND', 'T2_KGNP'])\n",
      "The distribution of labeled variants in this set was: {'KGFP': 36, 'T1_NGAP': 3, 'T1_PE': 2}\n"
     ]
    }
   ],
   "source": [
    "# seqr labels compared with the \"answer key\".\n",
    "\n",
    "print(f'Analyses included the following labels from SEQR: {comp_results.keys()}')\n",
    "label_dist = {k: comp_results[k]['matched']['count'] + comp_results[k]['unmatched']['count'] for k in comp_results if comp_results[k]['matched']['count'] + comp_results[k]['unmatched']['count'] > 0}\n",
    "print(f'The distribution of labeled variants in this set was: {label_dist}')\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "data = []\n",
    "\n",
    "for label in comp_results:\n",
    "    for status in comp_results[label]:\n",
    "        for variant in comp_results[label][status]['details']:\n",
    "            m = re.match('(?P<sid>RGP_[0-9]+_[0-9]+)::(?P<var>[0-9X]+:[0-9]+_[A-Z]+>[A-Z]+) - Confidence.(?P<label>[0-9A-Za-z_]+)', variant)\n",
    "            if not m:\n",
    "                print(f'    UNMATCHED: {variant}')\n",
    "\n",
    "            assert m.group('label') == label\n",
    "            data.append([m.group('sid'), family_from_sid(m.group('sid')), m.group('var'), status, label])\n",
    "\n",
    "comp_results_df = pd.DataFrame(data, columns=['sample_id', 'family_id', 'variant_raw', 'match_status', 'seqr_label'])\n",
    "\n",
    "# TODO set reasonable index.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RGP_1105_3 : [1:155612073_C>G (1:155612209_G>C)- Confidence.EXPECTED]\n",
      "RGP_1133_3 : [18:23535687_A>G (18:23536811_G>A)- Confidence.EXPECTED]\n",
      "RGP_1173_3 : [2:240783791_A>G - Confidence.EXPECTED]\n",
      "RGP_1197_3 : [16:57901371_T>A - Confidence.EXPECTED]\n",
      "RGP_1203_3 : [15:45069018_CG>C - Confidence.EXPECTED]\n",
      "RGP_183_3 : [7:152222037_GT>G - Confidence.EXPECTED]\n",
      "RGP_19_3 : [17:38739601_G>A - Confidence.EXPECTED]\n",
      "RGP_218_3 : [3:132675903_G>A (3:132675248_G>C)- Confidence.EXPECTED]\n",
      "RGP_320_3 : [X:154031020_G>A - Confidence.EXPECTED]\n",
      "RGP_362_3 : [18:51078294_C>T - Confidence.EXPECTED]\n",
      "RGP_405_3 : [2:47808215_GAC>G - Confidence.EXPECTED]\n",
      "RGP_409_3 : [X:101412604_C>T - Confidence.EXPECTED]\n",
      "RGP_416_3 : [5:177267613_G>T - Confidence.EXPECTED]\n",
      "RGP_426_3 : [22:41152220_CTG>C - Confidence.EXPECTED]\n",
      "RGP_437_3 : [6:157200782_C>G - Confidence.EXPECTED]\n",
      "RGP_440_3 : [1:1806503_A>G - Confidence.EXPECTED]\n",
      "RGP_441_3 : [9:127676671_T>C - Confidence.EXPECTED]\n",
      "RGP_476_3 : [11:17772388_G>A - Confidence.EXPECTED]\n",
      "RGP_477_3 : [17:7508291_C>T - Confidence.EXPECTED]\n",
      "RGP_485_3 : [8:60856553_GC>G - Confidence.EXPECTED]\n",
      "RGP_508_3 : [X:153934450_T>G - Confidence.EXPECTED]\n",
      "RGP_56_3 : [19:41970476_A>T - Confidence.EXPECTED]\n",
      "RGP_676_3 : [X:18628716_C>T - Confidence.EXPECTED]\n",
      "RGP_681_3 : [3:41225731_CT>C - Confidence.EXPECTED]\n",
      "RGP_683_3 : [X:74592156_G>A - Confidence.EXPECTED]\n",
      "RGP_692_3 : [3:132668887_G>GGA (3:132675903_G>A)- Confidence.EXPECTED]\n",
      "RGP_697_3 : [4:6301742_CTA>C (4:6301415_G>A)- Confidence.EXPECTED]\n",
      "RGP_736_3 : [14:35007368_AAAC>A - Confidence.EXPECTED]\n",
      "RGP_74_3 : [X:65731150_C>T - Confidence.EXPECTED]\n",
      "RGP_830_3 : [3:49722057_G>A (3:49723648_C>G)- Confidence.EXPECTED]\n",
      "RGP_854_3 : [1:99876555_TG>T - Confidence.EXPECTED]\n",
      "RGP_904_3 : [17:44212851_C>T - Confidence.EXPECTED]\n",
      "RGP_910_3 : [1:25808786_GGAGGTGAGGACAGCTGGGGTGCGACGTGGGGCCCCTCC>G (1:25813890_G>A)- Confidence.EXPECTED]\n",
      "RGP_930_3 : [2:219421350_T>C - Confidence.EXPECTED]\n",
      "RGP_956_3 : [X:49075573_G>A - Confidence.EXPECTED]\n"
     ]
    }
   ],
   "source": [
    "from comparison.comparison import CommonFormatResult, Confidence\n",
    "from collections import defaultdict\n",
    "import math\n",
    "\n",
    "sample_dict = defaultdict(list)\n",
    "\n",
    "for _, row in answer_key.iterrows():\n",
    "    # There's probably a slicker way to do this, but this is easy to reason about and debug.\n",
    "\n",
    "    # Check whether this row is a proband.\n",
    "    if not row['CAGI-RGP ID'].endswith('PROBAND'):\n",
    "        continue\n",
    "\n",
    "    # Check for a compound het (a second variant), if found, treat it just like a primary variant (for now)\n",
    "    if isinstance(row['Gene-2'], str):\n",
    "        support_result = CommonFormatResult(row['Chrom-2'], int(row['Pos-2']), row['Ref-2'], row['Alt-2'], confidence=[Confidence.EXPECTED])\n",
    "        #sample_dict[row['sample_id']].append(support_result)\n",
    "    else:\n",
    "        support_result = None\n",
    "\n",
    "    sample_dict[row['sample_id']].append(\n",
    "        CommonFormatResult(row['Chrom-1'], int(row['Pos-1']), row['Ref-1'], row['Alt-1'], confidence=[Confidence.EXPECTED], support=support_result)\n",
    "    )\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n",
      "no\n"
     ]
    }
   ],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "None == None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'RGP_1105_3': {'variants': [{'sample': 'RGP_1105_3',\n",
       "    'family': 'RGP_1105',\n",
       "    'gene': 'ENSG00000125459',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 155612209,\n",
       "      'ref': 'G',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 10.15999984741211,\n",
       "      'baseqranksum': 3.75,\n",
       "      'dp': 48222,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 5.270999908447266,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 56.869998931884766,\n",
       "      'mqranksum': 0.42899999022483826,\n",
       "      'mq_dp': 96,\n",
       "      'negative_train_site': True,\n",
       "      'qd': 10.15999984741211,\n",
       "      'raw_mq': 310456.0,\n",
       "      'readposranksum': 1.0099999904632568,\n",
       "      'sor': 1.2339999675750732,\n",
       "      'vqslod': -1.3359999656677246,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 965267,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 6.0116399254184216e-05,\n",
       "      'gnomad_ex_an': 249516,\n",
       "      'gnomad_ex_ac': 15,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0001248439948540181,\n",
       "      'gnomad_an': 152190,\n",
       "      'gnomad_ac': 19,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.019999999552965164,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 23.899999618530273,\n",
       "      'revel': 0.5879999995231628,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000125459'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'MSTO1',\n",
       "       'gene': 'ENSG00000125459',\n",
       "       'feature': 'ENST00000245564',\n",
       "       'mane_select': 'NM_018116.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000245564.8:c.706G>C',\n",
       "       'hgvsp': 'ENSP00000245564.3:p.Asp236His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000245564',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MSTO1',\n",
       "       'gene': 'ENSG00000125459',\n",
       "       'feature': 'ENST00000368341',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/13',\n",
       "       'hgvsc': 'ENST00000368341.8:c.601G>C',\n",
       "       'hgvsp': 'ENSP00000357325.4:p.Asp201His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357325',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'MSTO1',\n",
       "       'gene': 'ENSG00000125459',\n",
       "       'feature': 'ENST00000462250',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000476693',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MSTO1',\n",
       "       'gene': 'ENSG00000125459',\n",
       "       'feature': 'ENST00000649846',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000649846.1:c.706G>C',\n",
       "       'hgvsp': 'ENSP00000498188.1:p.Asp236His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498188',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1105_1': 'Het',\n",
       "     'RGP_1105_2': 'WT',\n",
       "     'RGP_1105_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Myopathy - adult onset',\n",
       "     'Myopathy - paediatric onset',\n",
       "     'Muscular dystrophy_Paediatric',\n",
       "     'Myopathy Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F028679_rgp_1105',\n",
       "   'family_id': 'RGP_1105',\n",
       "   'members': {'RGP_1105_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F028679_rgp_1105'},\n",
       "    'RGP_1105_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F028679_rgp_1105'},\n",
       "    'RGP_1105_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F028679_rgp_1105'}},\n",
       "   'phenotypes': ['HP:0012642',\n",
       "    'HP:0005952',\n",
       "    'HP:0002194',\n",
       "    'HP:0003236',\n",
       "    'HP:0001263',\n",
       "    'HP:0008947',\n",
       "    'HP:0003713',\n",
       "    'HP:0003198',\n",
       "    'HP:0002033',\n",
       "    'HP:0003701',\n",
       "    'HP:0000360'],\n",
       "   'panel_ids': [3078, 72, 137, 329, 328, 141, 250, 3101, 3071],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Cerebellar and Pontocerebellar Hypoplasia',\n",
       "    'Mendeliome',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Myopathy - adult onset',\n",
       "    'Muscular dystrophy_Paediatric',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Myopathy Superpanel',\n",
       "    'Limb Girdle Muscular Dystrophy']}},\n",
       " 'RGP_1133_3': {'variants': [{'sample': 'RGP_1133_3',\n",
       "    'family': 'RGP_1133',\n",
       "    'gene': 'ENSG00000165091',\n",
       "    'var_data': {'coords': {'chrom': '9',\n",
       "      'pos': 72816213,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 9.319999694824219,\n",
       "      'baseqranksum': 0.7369999885559082,\n",
       "      'db': True,\n",
       "      'dp': 51680,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.930999994277954,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.190000057220459,\n",
       "      'mq_dp': 75,\n",
       "      'qd': 9.319999694824219,\n",
       "      'raw_mq': 270000.0,\n",
       "      'readposranksum': 2.309999942779541,\n",
       "      'sor': 0.8240000009536743,\n",
       "      'vqslod': 14.100000381469727,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 57028,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0006128380191512406,\n",
       "      'gnomad_ex_an': 251290,\n",
       "      'gnomad_ex_ac': 154,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0005123619921505451,\n",
       "      'gnomad_an': 152236,\n",
       "      'gnomad_ac': 78,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.5600000023841858,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 23.799999237060547,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000165091'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000297784',\n",
       "       'mane_select': 'NM_138691.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000297784.10:c.1763+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000297784',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000340019',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000340019.4:c.1763+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000341433',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000645208',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000645208.2:c.1763+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494684',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000645773',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000645773.1:c.1637+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493698',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000646619',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000646619.1:c.1325+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493726',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_region_variant&intron_variant',\n",
       "       'symbol': 'TMC1',\n",
       "       'gene': 'ENSG00000165091',\n",
       "       'feature': 'ENST00000651183',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000651183.1:c.1325+3A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498723',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '5']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1133_1': 'WT',\n",
       "     'RGP_1133_2': 'Het',\n",
       "     'RGP_1133_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1133_3',\n",
       "    'family': 'RGP_1133',\n",
       "    'gene': 'ENSG00000141458',\n",
       "    'var_data': {'coords': {'chrom': '18',\n",
       "      'pos': 23535687,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 15.359999656677246,\n",
       "      'baseqranksum': 1.4199999570846558,\n",
       "      'dp': 50519,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.8630000352859497,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.649999976158142,\n",
       "      'mq_dp': 77,\n",
       "      'qd': 15.359999656677246,\n",
       "      'raw_mq': 277200.0,\n",
       "      'readposranksum': 2.880000114440918,\n",
       "      'sor': 0.9409999847412109,\n",
       "      'vqslod': 13.0,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 548502,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.3144600416126195e-05,\n",
       "      'gnomad_an': 152154,\n",
       "      'gnomad_ac': 2,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.9359999895095825,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000141458'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000269228',\n",
       "       'mane_select': 'NM_000271.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/25',\n",
       "       'hgvsc': 'ENST00000269228.10:c.3259T>C',\n",
       "       'hgvsp': 'ENSP00000269228.4:p.Phe1087Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000269228',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000586150',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/4',\n",
       "       'hgvsc': 'ENST00000586150.5:c.16T>C',\n",
       "       'hgvsp': 'ENSP00000468321.1:p.Phe6Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468321',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000590723',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464755',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591051',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '15/18',\n",
       "       'hgvsc': 'ENST00000591051.1:c.2338T>C',\n",
       "       'hgvsp': 'ENSP00000467636.1:p.Phe780Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467636',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591107',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468438',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000593280',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467150',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1133_1': 'Het',\n",
       "     'RGP_1133_2': 'WT',\n",
       "     'RGP_1133_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Lysosomal Storage Disorder'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1133_3',\n",
       "    'family': 'RGP_1133',\n",
       "    'gene': 'ENSG00000141458',\n",
       "    'var_data': {'coords': {'chrom': '18',\n",
       "      'pos': 23535687,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 15.359999656677246,\n",
       "      'baseqranksum': 1.4199999570846558,\n",
       "      'dp': 50519,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.8630000352859497,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.649999976158142,\n",
       "      'mq_dp': 77,\n",
       "      'qd': 15.359999656677246,\n",
       "      'raw_mq': 277200.0,\n",
       "      'readposranksum': 2.880000114440918,\n",
       "      'sor': 0.9409999847412109,\n",
       "      'vqslod': 13.0,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 548502,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.3144600416126195e-05,\n",
       "      'gnomad_an': 152154,\n",
       "      'gnomad_ac': 2,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.9359999895095825,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000141458'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000269228',\n",
       "       'mane_select': 'NM_000271.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/25',\n",
       "       'hgvsc': 'ENST00000269228.10:c.3259T>C',\n",
       "       'hgvsp': 'ENSP00000269228.4:p.Phe1087Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000269228',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000586150',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/4',\n",
       "       'hgvsc': 'ENST00000586150.5:c.16T>C',\n",
       "       'hgvsp': 'ENSP00000468321.1:p.Phe6Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468321',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000590723',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464755',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591051',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '15/18',\n",
       "       'hgvsc': 'ENST00000591051.1:c.2338T>C',\n",
       "       'hgvsp': 'ENSP00000467636.1:p.Phe780Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467636',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591107',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468438',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000593280',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467150',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_1133_1': 'Het',\n",
       "     'RGP_1133_2': 'WT',\n",
       "     'RGP_1133_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['18-23536811-G-A'],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Lysosomal Storage Disorder'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1133_3',\n",
       "    'family': 'RGP_1133',\n",
       "    'gene': 'ENSG00000141458',\n",
       "    'var_data': {'coords': {'chrom': '18',\n",
       "      'pos': 23536811,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 10.010000228881836,\n",
       "      'baseqranksum': 0.6370000243186951,\n",
       "      'db': True,\n",
       "      'dp': 51255,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.7380000352859497,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.0199999809265137,\n",
       "      'mq_dp': 91,\n",
       "      'qd': 10.010000228881836,\n",
       "      'raw_mq': 327600.0,\n",
       "      'readposranksum': 1.5,\n",
       "      'sor': 0.4699999988079071,\n",
       "      'vqslod': 14.5,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 17997,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 2.7839900212711655e-05,\n",
       "      'gnomad_ex_an': 251438,\n",
       "      'gnomad_ex_ac': 7,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.9712999346666038e-05,\n",
       "      'gnomad_an': 152184,\n",
       "      'gnomad_ac': 3,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.600000381469727,\n",
       "      'revel': 0.8740000128746033,\n",
       "      'mutationtaster': 'A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000141458'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000269228',\n",
       "       'mane_select': 'NM_000271.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '21/25',\n",
       "       'hgvsc': 'ENST00000269228.10:c.3107C>T',\n",
       "       'hgvsp': 'ENSP00000269228.4:p.Thr1036Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000269228',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000586150',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468321',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000590723',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464755',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591051',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '14/18',\n",
       "       'hgvsc': 'ENST00000591051.1:c.2186C>T',\n",
       "       'hgvsp': 'ENSP00000467636.1:p.Thr729Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467636',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591107',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468438',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000593280',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467150',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1133_1': 'WT',\n",
       "     'RGP_1133_2': 'Het',\n",
       "     'RGP_1133_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Lysosomal Storage Disorder'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1133_3',\n",
       "    'family': 'RGP_1133',\n",
       "    'gene': 'ENSG00000141458',\n",
       "    'var_data': {'coords': {'chrom': '18',\n",
       "      'pos': 23536811,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 10.010000228881836,\n",
       "      'baseqranksum': 0.6370000243186951,\n",
       "      'db': True,\n",
       "      'dp': 51255,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.7380000352859497,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.0199999809265137,\n",
       "      'mq_dp': 91,\n",
       "      'qd': 10.010000228881836,\n",
       "      'raw_mq': 327600.0,\n",
       "      'readposranksum': 1.5,\n",
       "      'sor': 0.4699999988079071,\n",
       "      'vqslod': 14.5,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 17997,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 2.7839900212711655e-05,\n",
       "      'gnomad_ex_an': 251438,\n",
       "      'gnomad_ex_ac': 7,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.9712999346666038e-05,\n",
       "      'gnomad_an': 152184,\n",
       "      'gnomad_ac': 3,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.600000381469727,\n",
       "      'revel': 0.8740000128746033,\n",
       "      'mutationtaster': 'A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000141458'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000269228',\n",
       "       'mane_select': 'NM_000271.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '21/25',\n",
       "       'hgvsc': 'ENST00000269228.10:c.3107C>T',\n",
       "       'hgvsp': 'ENSP00000269228.4:p.Thr1036Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000269228',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000586150',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468321',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000590723',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464755',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591051',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '14/18',\n",
       "       'hgvsc': 'ENST00000591051.1:c.2186C>T',\n",
       "       'hgvsp': 'ENSP00000467636.1:p.Thr729Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467636',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000591107',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468438',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'NPC1',\n",
       "       'gene': 'ENSG00000141458',\n",
       "       'feature': 'ENST00000593280',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000467150',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_1133_1': 'WT',\n",
       "     'RGP_1133_2': 'Het',\n",
       "     'RGP_1133_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['18-23535687-A-G'],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Lysosomal Storage Disorder'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F028760_rgp_1133',\n",
       "   'family_id': 'RGP_1133',\n",
       "   'members': {'RGP_1133_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F028760_rgp_1133'},\n",
       "    'RGP_1133_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F028760_rgp_1133'},\n",
       "    'RGP_1133_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F028760_rgp_1133'}},\n",
       "   'phenotypes': ['HP:0002194',\n",
       "    'HP:0002540',\n",
       "    'HP:0010536',\n",
       "    'HP:0002835',\n",
       "    'HP:0001744'],\n",
       "   'panel_ids': [137, 250, 181, 106],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Lysosomal Storage Disorder',\n",
       "    'Glycogen Storage Diseases']}},\n",
       " 'RGP_1173_3': {'variants': [{'sample': 'RGP_1173_3',\n",
       "    'family': 'RGP_1173',\n",
       "    'gene': 'ENSG00000143631',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 152312600,\n",
       "      'ref': 'CACTG',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 49,\n",
       "      'af': 0.017000000923871994,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.0658000037074089,\n",
       "      'as_qd': 19.040000915527344,\n",
       "      'baseqranksum': 0.18199999630451202,\n",
       "      'dp': 49031,\n",
       "      'excesshet': 0.1437000036239624,\n",
       "      'fs': 0.5270000100135803,\n",
       "      'inbreedingcoeff': 0.0658000037074089,\n",
       "      'mleac': 49,\n",
       "      'mleaf': 0.017000000923871994,\n",
       "      'mq': 59.86000061035156,\n",
       "      'mqranksum': -0.00800000037997961,\n",
       "      'mq_dp': 1649,\n",
       "      'qd': 19.049999237060547,\n",
       "      'raw_mq': 5908690.0,\n",
       "      'readposranksum': 0.13600000739097595,\n",
       "      'sor': 0.7549999952316284,\n",
       "      'vqslod': 3.299999952316284,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 31359,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.01287159975618124,\n",
       "      'gnomad_ex_an': 251484,\n",
       "      'gnomad_ex_ac': 3237,\n",
       "      'gnomad_ex_hom': 34,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.01259750034660101,\n",
       "      'gnomad_an': 151856,\n",
       "      'gnomad_ac': 1913,\n",
       "      'gnomad_hom': 18,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 18.65999984741211,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000143631'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'FLG',\n",
       "       'gene': 'ENSG00000143631',\n",
       "       'feature': 'ENST00000368799',\n",
       "       'mane_select': 'NM_002016.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000368799.2:c.2282_2285del',\n",
       "       'hgvsp': 'ENSP00000357789.1:p.Ser761CysfsTer36',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000357789',\n",
       "       'lof': 'LC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1173_1': 'WT',\n",
       "     'RGP_1173_2_D2': 'Het',\n",
       "     'RGP_1173_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1173_3',\n",
       "    'family': 'RGP_1173',\n",
       "    'gene': 'ENSG00000130294',\n",
       "    'var_data': {'coords': {'chrom': '2',\n",
       "      'pos': 240783791,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 11.949999809265137,\n",
       "      'baseqranksum': 1.2799999713897705,\n",
       "      'dp': 59743,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 2.61899995803833,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.1340000033378601,\n",
       "      'mq_dp': 44,\n",
       "      'qd': 11.960000038146973,\n",
       "      'raw_mq': 158400.0,\n",
       "      'readposranksum': 0.15800000727176666,\n",
       "      'sor': 0.36800000071525574,\n",
       "      'vqslod': 11.699999809265137,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 973292,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 28.600000381469727,\n",
       "      'revel': 0.9779999852180481,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000130294'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000320389',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000320389.12:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000322791.8:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000322791',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000404283',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/50',\n",
       "       'hgvsc': 'ENST00000404283.9:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000384231.5:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000384231',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000428768',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/15',\n",
       "       'hgvsc': 'ENST00000428768.2:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000395443.2:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000395443',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000498729',\n",
       "       'mane_select': 'NM_001244008.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/49',\n",
       "       'hgvsc': 'ENST00000498729.9:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000438388.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000438388',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000647572',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497953',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000647731',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000647731.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000498099.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498099',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000647885',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000647885.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497739.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497739',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000648047',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497935',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000648129',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000648129.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497293.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497293',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000648364',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000648364.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000498196.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498196',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000648680',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/49',\n",
       "       'hgvsc': 'ENST00000648680.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497586.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497586',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000649096',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/47',\n",
       "       'hgvsc': 'ENST00000649096.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497030.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497030',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000649306',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000649306.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497678.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497678',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000650053',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/47',\n",
       "       'hgvsc': 'ENST00000650053.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000497824.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497824',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000650130',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/48',\n",
       "       'hgvsc': 'ENST00000650130.1:c.746T>C',\n",
       "       'hgvsp': 'ENSP00000498082.1:p.Leu249Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498082',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'KIF1A',\n",
       "       'gene': 'ENSG00000130294',\n",
       "       'feature': 'ENST00000675314',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000502584',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1173_1': 'WT',\n",
       "     'RGP_1173_2_D2': 'WT',\n",
       "     'RGP_1173_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Genetic Epilepsy',\n",
       "     'Tremors_Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1173_3',\n",
       "    'family': 'RGP_1173',\n",
       "    'gene': 'ENSG00000124587',\n",
       "    'var_data': {'coords': {'chrom': '6',\n",
       "      'pos': 42966802,\n",
       "      'ref': 'G',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 12.6899995803833,\n",
       "      'baseqranksum': 0.6159999966621399,\n",
       "      'dp': 52307,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 20.3439998626709,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.4490000009536743,\n",
       "      'mq_dp': 88,\n",
       "      'qd': 12.6899995803833,\n",
       "      'raw_mq': 316800.0,\n",
       "      'readposranksum': 0.7900000214576721,\n",
       "      'sor': 1.7640000581741333,\n",
       "      'vqslod': 12.119999885559082,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 560857,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 2.783879972412251e-05,\n",
       "      'gnomad_ex_an': 251448,\n",
       "      'gnomad_ex_ac': 7,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 6.582759851880837e-06,\n",
       "      'gnomad_an': 151912,\n",
       "      'gnomad_ac': 1,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 35.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000124587'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'PEX6',\n",
       "       'gene': 'ENSG00000124587',\n",
       "       'feature': 'ENST00000244546',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/15',\n",
       "       'hgvsc': 'ENST00000244546.4:c.1941C>A',\n",
       "       'hgvsp': 'ENSP00000244546.4:p.Cys647Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000244546',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PEX6',\n",
       "       'gene': 'ENSG00000124587',\n",
       "       'feature': 'ENST00000304611',\n",
       "       'mane_select': 'NM_000287.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/17',\n",
       "       'hgvsc': 'ENST00000304611.13:c.1941C>A',\n",
       "       'hgvsp': 'ENSP00000303511.8:p.Cys647Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000303511',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1173_1': 'WT',\n",
       "     'RGP_1173_2_D2': 'Het',\n",
       "     'RGP_1173_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Genetic Epilepsy',\n",
       "     'Intellectual disability syndromic and non-syndromic',\n",
       "     'Metabolic Disorders Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1173_3',\n",
       "    'family': 'RGP_1173',\n",
       "    'gene': 'ENSG00000204983',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 142751111,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.5618000030517578,\n",
       "      'as_qd': 0.5799999833106995,\n",
       "      'baseqranksum': 1.350000023841858,\n",
       "      'dp': 54409,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 10.100000381469727,\n",
       "      'inbreedingcoeff': 0.5616999864578247,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 56.68000030517578,\n",
       "      'mqranksum': -1.6200000047683716,\n",
       "      'mq_dp': 60,\n",
       "      'negative_train_site': True,\n",
       "      'qd': 0.5899999737739563,\n",
       "      'raw_mq': 192740.0,\n",
       "      'readposranksum': -0.13899999856948853,\n",
       "      'sor': 2.2669999599456787,\n",
       "      'vqslod': -6.734000205993652,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.3575699995271862e-05,\n",
       "      'gnomad_an': 147322,\n",
       "      'gnomad_ac': 2,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 14.460000038146973,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'N,N',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000204983'},\n",
       "     'transcript_consequences': [{'consequence': 'intron_variant',\n",
       "       'symbol': 'PRSS1',\n",
       "       'gene': 'ENSG00000204983',\n",
       "       'feature': 'ENST00000311737',\n",
       "       'mane_select': 'NM_002769.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000311737.12:c.200+397G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000308720',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'PRSS1',\n",
       "       'gene': 'ENSG00000204983',\n",
       "       'feature': 'ENST00000486171',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000486171.5:c.242+1G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417854',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'PRSS1',\n",
       "       'gene': 'ENSG00000204983',\n",
       "       'feature': 'ENST00000492062',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000492062.1:c.50+397G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419912',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1173_1': 'WT',\n",
       "     'RGP_1173_2_D2': 'WT',\n",
       "     'RGP_1173_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1173_3',\n",
       "    'family': 'RGP_1173',\n",
       "    'gene': 'ENSG00000140521',\n",
       "    'var_data': {'coords': {'chrom': '15',\n",
       "      'pos': 89327201,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 5,\n",
       "      'af': 0.0016959999920800328,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.0017000000225380063,\n",
       "      'as_qd': 14.529999732971191,\n",
       "      'baseqranksum': -0.03400000184774399,\n",
       "      'db': True,\n",
       "      'dp': 52928,\n",
       "      'excesshet': 3.025099992752075,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.0017000000225380063,\n",
       "      'mleac': 5,\n",
       "      'mleaf': 0.0016959999920800328,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.35199999809265137,\n",
       "      'mq_dp': 210,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 14.529999732971191,\n",
       "      'raw_mq': 756000.0,\n",
       "      'readposranksum': 0.5920000076293945,\n",
       "      'sor': 0.6759999990463257,\n",
       "      'vqslod': 15.6899995803833,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 28535,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0005129550117999315,\n",
       "      'gnomad_ex_an': 251484,\n",
       "      'gnomad_ex_ac': 129,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0006567970267497003,\n",
       "      'gnomad_an': 152254,\n",
       "      'gnomad_ac': 100,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 25.799999237060547,\n",
       "      'revel': 0.8840000033378601,\n",
       "      'mutationtaster': 'A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000140521'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000268124',\n",
       "       'mane_select': 'NM_002693.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/23',\n",
       "       'hgvsc': 'ENST00000268124.11:c.1399G>A',\n",
       "       'hgvsp': 'ENSP00000268124.5:p.Ala467Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000268124',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000442287',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/23',\n",
       "       'hgvsc': 'ENST00000442287.6:c.1399G>A',\n",
       "       'hgvsp': 'ENSP00000399851.2:p.Ala467Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000399851',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000526314',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000432389',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000637264',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/20',\n",
       "       'hgvsc': 'ENST00000637264.1:c.472G>A',\n",
       "       'hgvsp': 'ENSP00000489844.1:p.Ala158Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489844',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000637307',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490427',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000650303',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497242',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'POLG',\n",
       "       'gene': 'ENSG00000140521',\n",
       "       'feature': 'ENST00000670281',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000499709',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1173_1': 'WT',\n",
       "     'RGP_1173_2_D2': 'Het',\n",
       "     'RGP_1173_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Metabolic Disorders Superpanel',\n",
       "     'Tremors_Superpanel',\n",
       "     'Genetic Epilepsy',\n",
       "     'Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F029945_rgp_1173',\n",
       "   'family_id': 'RGP_1173',\n",
       "   'members': {'RGP_1173_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F029945_rgp_1173'},\n",
       "    'RGP_1173_2_D2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F029945_rgp_1173'},\n",
       "    'RGP_1173_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F029945_rgp_1173'}},\n",
       "   'phenotypes': ['HP:0002020',\n",
       "    'HP:0008738',\n",
       "    'HP:0000010',\n",
       "    'HP:0000218',\n",
       "    'HP:0000988',\n",
       "    'HP:0009747',\n",
       "    'HP:0100814',\n",
       "    'HP:0001773',\n",
       "    'HP:0200055',\n",
       "    'HP:0003128',\n",
       "    'HP:0002046',\n",
       "    'HP:0001252',\n",
       "    'HP:0001337',\n",
       "    'HP:0012758',\n",
       "    'HP:0000707',\n",
       "    'HP:0001250',\n",
       "    'HP:0010535',\n",
       "    'HP:0100547',\n",
       "    'HP:0007359',\n",
       "    'HP:0000326',\n",
       "    'HP:0000076'],\n",
       "   'panel_ids': [3078,\n",
       "    137,\n",
       "    3465,\n",
       "    329,\n",
       "    3531,\n",
       "    202,\n",
       "    3729,\n",
       "    275,\n",
       "    3796,\n",
       "    3795,\n",
       "    228,\n",
       "    231,\n",
       "    233,\n",
       "    250,\n",
       "    251,\n",
       "    61,\n",
       "    63],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Mendeliome',\n",
       "    'Metabolic Disorders Superpanel',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Tremors_Superpanel',\n",
       "    'Genetic Epilepsy',\n",
       "    'Hand and foot malformations',\n",
       "    'Kidneyome_SuperPanel',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Focal Epilepsy',\n",
       "    'Mendelian susceptibility to Immune Disorders',\n",
       "    'Defects of innate immunity',\n",
       "    'Phagocyte Defects',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Congenital abnormalities of the kidneys and urinary tract (CAKUT)_SuperPanel',\n",
       "    'Congenital anomalies of the kidney and urinary tract (CAKUT) Nonsyndromic',\n",
       "    'Congenital anomalies of the kidney and urinary tract (CAKUT) Syndromic']}},\n",
       " 'RGP_1197_3': {'variants': [{'sample': 'RGP_1197_3',\n",
       "    'family': 'RGP_1197',\n",
       "    'gene': 'ENSG00000007062',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 16000517,\n",
       "      'ref': 'G',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 9.0600004196167,\n",
       "      'baseqranksum': 1.0299999713897705,\n",
       "      'db': True,\n",
       "      'dp': 54422,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 3.128000020980835,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.12200000137090683,\n",
       "      'mq_dp': 35,\n",
       "      'qd': 9.069999694824219,\n",
       "      'raw_mq': 126000.0,\n",
       "      'readposranksum': 1.2400000095367432,\n",
       "      'sor': 0.28299999237060547,\n",
       "      'vqslod': 14.25,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 106454,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 2.8271400879020803e-05,\n",
       "      'gnomad_ex_an': 247600,\n",
       "      'gnomad_ex_ac': 7,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 7.886849925853312e-05,\n",
       "      'gnomad_an': 152152,\n",
       "      'gnomad_ac': 12,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 34.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,A,A,A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000007062'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000447510',\n",
       "       'mane_select': 'NM_006017.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '14/28',\n",
       "       'hgvsc': 'ENST00000447510.7:c.1557C>A',\n",
       "       'hgvsp': 'ENSP00000415481.2:p.Tyr519Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000415481',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000505450',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/27',\n",
       "       'hgvsc': 'ENST00000505450.5:c.1530C>A',\n",
       "       'hgvsp': 'ENSP00000426090.1:p.Tyr510Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000426090',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000508167',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/27',\n",
       "       'hgvsc': 'ENST00000508167.5:c.1530C>A',\n",
       "       'hgvsp': 'ENSP00000427346.1:p.Tyr510Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000427346',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000510224',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '14/28',\n",
       "       'hgvsc': 'ENST00000510224.5:c.1557C>A',\n",
       "       'hgvsp': 'ENSP00000426809.1:p.Tyr519Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000426809',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000539194',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '14/26',\n",
       "       'hgvsc': 'ENST00000539194.6:c.1557C>A',\n",
       "       'hgvsp': 'ENSP00000443620.1:p.Tyr519Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000443620',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000540805',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/25',\n",
       "       'hgvsc': 'ENST00000540805.6:c.1530C>A',\n",
       "       'hgvsp': 'ENSP00000438045.2:p.Tyr510Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000438045',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000675377',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/25',\n",
       "       'hgvsc': 'ENST00000675377.1:c.1557C>A',\n",
       "       'hgvsp': 'ENSP00000502545.1:p.Tyr519Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000502545',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'PROM1',\n",
       "       'gene': 'ENSG00000007062',\n",
       "       'feature': 'ENST00000675613',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/23',\n",
       "       'hgvsc': 'ENST00000675613.1:c.1530C>A',\n",
       "       'hgvsp': 'ENSP00000501741.1:p.Tyr510Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501741',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1197_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Cone-rod Dystrophy',\n",
       "     'Retinal Disorders Superpanel',\n",
       "     'Retinitis pigmentosa_Autosomal Recessive/X-linked',\n",
       "     'Retinitis Pigmentosa Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1197_3',\n",
       "    'family': 'RGP_1197',\n",
       "    'gene': 'ENSG00000065618',\n",
       "    'var_data': {'coords': {'chrom': '10',\n",
       "      'pos': 104037044,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 12.890000343322754,\n",
       "      'baseqranksum': 0.7549999952316284,\n",
       "      'db': True,\n",
       "      'dp': 55992,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 11.229000091552734,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.5849999785423279,\n",
       "      'mq_dp': 48,\n",
       "      'qd': 12.899999618530273,\n",
       "      'raw_mq': 172800.0,\n",
       "      'readposranksum': 1.1399999856948853,\n",
       "      'sor': 0.27000001072883606,\n",
       "      'vqslod': 13.850000381469727,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 585302,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 5.8833400544244796e-05,\n",
       "      'gnomad_ex_an': 237960,\n",
       "      'gnomad_ex_ac': 14,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 9.200839849654585e-05,\n",
       "      'gnomad_an': 152160,\n",
       "      'gnomad_ac': 14,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.9700000286102295,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 34.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000065618'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'COL17A1',\n",
       "       'gene': 'ENSG00000065618',\n",
       "       'feature': 'ENST00000369733',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000369733.8:c.3142+1G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000358748',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'COL17A1',\n",
       "       'gene': 'ENSG00000065618',\n",
       "       'feature': 'ENST00000433822',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388832',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'COL17A1',\n",
       "       'gene': 'ENSG00000065618',\n",
       "       'feature': 'ENST00000648076',\n",
       "       'mane_select': 'NM_000494.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000648076.2:c.3277+1G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497653',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3', '5']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1197_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1197_3',\n",
       "    'family': 'RGP_1197',\n",
       "    'gene': 'ENSG00000070729',\n",
       "    'var_data': {'coords': {'chrom': '16',\n",
       "      'pos': 57901371,\n",
       "      'ref': 'T',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 6,\n",
       "      'af': 0.0020349998958408833,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.3714999854564667,\n",
       "      'as_qd': 16.700000762939453,\n",
       "      'baseqranksum': 0.46000000834465027,\n",
       "      'db': True,\n",
       "      'dp': 52099,\n",
       "      'excesshet': 0.011099999770522118,\n",
       "      'fs': 1.753999948501587,\n",
       "      'inbreedingcoeff': 0.3714999854564667,\n",
       "      'mleac': 6,\n",
       "      'mleaf': 0.0020349998958408833,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.6520000100135803,\n",
       "      'mq_dp': 199,\n",
       "      'qd': 16.700000762939453,\n",
       "      'raw_mq': 716400.0,\n",
       "      'readposranksum': 0.35199999809265137,\n",
       "      'sor': 0.7379999756813049,\n",
       "      'vqslod': 14.800000190734863,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 177287,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0011421700473874807,\n",
       "      'gnomad_ex_an': 249524,\n",
       "      'gnomad_ex_ac': 285,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.001071110018528998,\n",
       "      'gnomad_an': 152178,\n",
       "      'gnomad_ac': 163,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 25.0,\n",
       "      'revel': 0.6949999928474426,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000070729'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'CNGB1',\n",
       "       'gene': 'ENSG00000070729',\n",
       "       'feature': 'ENST00000251102',\n",
       "       'mane_select': 'NM_001297.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '29/33',\n",
       "       'hgvsc': 'ENST00000251102.13:c.2957A>T',\n",
       "       'hgvsp': 'ENSP00000251102.8:p.Asn986Ile',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000251102',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'CNGB1',\n",
       "       'gene': 'ENSG00000070729',\n",
       "       'feature': 'ENST00000564448',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '29/33',\n",
       "       'hgvsc': 'ENST00000564448.5:c.2939A>T',\n",
       "       'hgvsp': 'ENSP00000454633.1:p.Asn980Ile',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000454633',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'CNGB1',\n",
       "       'gene': 'ENSG00000070729',\n",
       "       'feature': 'ENST00000565942',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/4',\n",
       "       'hgvsc': 'ENST00000565942.1:c.5A>T',\n",
       "       'hgvsp': 'ENSP00000455964.1:p.Asn2Ile',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000455964',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Homozygous'],\n",
       "    'genotypes': {'RGP_1197_3': 'Hom'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Retinitis Pigmentosa Superpanel',\n",
       "     'Retinitis pigmentosa_Autosomal Recessive/X-linked',\n",
       "     'Retinal Disorders Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F030169_rgp_1197',\n",
       "   'family_id': 'RGP_1197',\n",
       "   'members': {'RGP_1197_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F030169_rgp_1197'}},\n",
       "   'phenotypes': ['HP:0000510', 'HP:0000618'],\n",
       "   'panel_ids': [283, 3124, 277, 278, 137, 3147, 3100],\n",
       "   'panel_names': ['Congenital Stationary Night Blindness',\n",
       "    'Retinal Disorders Superpanel',\n",
       "    'Retinitis pigmentosa_Autosomal Recessive/X-linked',\n",
       "    'Retinitis pigmentosa_Autosomal Dominant',\n",
       "    'Mendeliome',\n",
       "    'Cone-rod Dystrophy',\n",
       "    'Retinitis Pigmentosa Superpanel']}},\n",
       " 'RGP_1203_3': {'variants': [{'sample': 'RGP_1203_3',\n",
       "    'family': 'RGP_1203',\n",
       "    'gene': 'ENSG00000134853',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 54281602,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 227,\n",
       "      'af': 0.07699999958276749,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.011900000274181366,\n",
       "      'as_qd': 13.989999771118164,\n",
       "      'baseqranksum': 0.7689999938011169,\n",
       "      'db': True,\n",
       "      'dp': 53658,\n",
       "      'excesshet': 1.6013000011444092,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.011900000274181366,\n",
       "      'mleac': 227,\n",
       "      'mleaf': 0.07699999958276749,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.125,\n",
       "      'mq_dp': 9171,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.989999771118164,\n",
       "      'raw_mq': 33011800.0,\n",
       "      'readposranksum': 0.3659999966621399,\n",
       "      'sor': 0.6930000185966492,\n",
       "      'vqslod': 19.010000228881836,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 137113,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.07246240228414536,\n",
       "      'gnomad_ex_an': 134042,\n",
       "      'gnomad_ex_ac': 9713,\n",
       "      'gnomad_ex_hom': 411,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.08386579900979996,\n",
       "      'gnomad_an': 152112,\n",
       "      'gnomad_ac': 12757,\n",
       "      'gnomad_hom': 582,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 5.215000152587891,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000134853'},\n",
       "     'transcript_consequences': [{'consequence': 'intron_variant',\n",
       "       'symbol': 'PDGFRA',\n",
       "       'gene': 'ENSG00000134853',\n",
       "       'feature': 'ENST00000257290',\n",
       "       'mane_select': 'NM_006206.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000257290.10:c.2323+1120C>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000257290',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1203_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1203_3',\n",
       "    'family': 'RGP_1203',\n",
       "    'gene': 'ENSG00000010704',\n",
       "    'var_data': {'coords': {'chrom': '6',\n",
       "      'pos': 26090951,\n",
       "      'ref': 'C',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 355,\n",
       "      'af': 0.11999999731779099,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.016699999570846558,\n",
       "      'as_qd': 15.130000114440918,\n",
       "      'baseqranksum': 0.1679999977350235,\n",
       "      'db': True,\n",
       "      'dp': 52248,\n",
       "      'excesshet': 1.2621999979019165,\n",
       "      'fs': 1.093000054359436,\n",
       "      'inbreedingcoeff': 0.016699999570846558,\n",
       "      'mleac': 355,\n",
       "      'mleaf': 0.11999999731779099,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.0,\n",
       "      'mq_dp': 12463,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 15.130000114440918,\n",
       "      'raw_mq': 44866100.0,\n",
       "      'readposranksum': 0.5070000290870667,\n",
       "      'sor': 0.6779999732971191,\n",
       "      'vqslod': 16.0,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 15049,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.10924000293016434,\n",
       "      'gnomad_ex_an': 251484,\n",
       "      'gnomad_ex_ac': 27472,\n",
       "      'gnomad_ex_hom': 1832,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.10164699703454971,\n",
       "      'gnomad_an': 152026,\n",
       "      'gnomad_ac': 15453,\n",
       "      'gnomad_hom': 1005,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.019999999552965164,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 15.609999656677246,\n",
       "      'revel': 0.3149999976158142,\n",
       "      'mutationtaster': 'P,P,P,P,P,P,P,P,P,P,P',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000010704'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000309234',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/6',\n",
       "       'hgvsc': 'ENST00000309234.10:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000311698.6:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311698',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000317896',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/5',\n",
       "       'hgvsc': 'ENST00000317896.11:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000313776.7:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000313776',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000336625',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/5',\n",
       "       'hgvsc': 'ENST00000336625.12:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000337819.8:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000337819',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000349999',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000349999.8:c.77-363C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000259699',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000352392',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000352392.8:c.77-2168C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000315936',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000353147',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000353147.9:c.77-1734C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000312342',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000357618',\n",
       "       'mane_select': 'NM_000410.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/6',\n",
       "       'hgvsc': 'ENST00000357618.10:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000417404.1:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417404',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000397022',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/6',\n",
       "       'hgvsc': 'ENST00000397022.7:c.118C>G',\n",
       "       'hgvsp': 'ENSP00000380217.3:p.His40Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000380217',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000461397',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/6',\n",
       "       'hgvsc': 'ENST00000461397.5:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000420802.1:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420802',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000470149',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/7',\n",
       "       'hgvsc': 'ENST00000470149.5:c.187C>G',\n",
       "       'hgvsp': 'ENSP00000419725.1:p.His63Asp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419725',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000485729',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417534',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000488199',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000488199.5:c.77-363C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420559',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_1203_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['6-26092913-G-A'],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1203_3',\n",
       "    'family': 'RGP_1203',\n",
       "    'gene': 'ENSG00000010704',\n",
       "    'var_data': {'coords': {'chrom': '6',\n",
       "      'pos': 26092913,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 164,\n",
       "      'af': 0.0560000017285347,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.02019999921321869,\n",
       "      'as_qd': 13.880000114440918,\n",
       "      'baseqranksum': 0.2290000021457672,\n",
       "      'db': True,\n",
       "      'dp': 52014,\n",
       "      'excesshet': 6.236000061035156,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.02019999921321869,\n",
       "      'mleac': 164,\n",
       "      'mleaf': 0.0560000017285347,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.12700000405311584,\n",
       "      'mq_dp': 6407,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.880000114440918,\n",
       "      'raw_mq': 23063100.0,\n",
       "      'readposranksum': 0.5630000233650208,\n",
       "      'sor': 0.6899999976158142,\n",
       "      'vqslod': 18.420000076293945,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 15048,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.03321180120110512,\n",
       "      'gnomad_ex_an': 251236,\n",
       "      'gnomad_ex_ac': 8344,\n",
       "      'gnomad_ex_hom': 244,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.03732160106301308,\n",
       "      'gnomad_an': 152164,\n",
       "      'gnomad_ac': 5679,\n",
       "      'gnomad_hom': 151,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.07999999821186066,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 25.799999237060547,\n",
       "      'revel': 0.871999979019165,\n",
       "      'mutationtaster': 'A,A,A,A,A,A,A,A,A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000010704'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000309234',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/6',\n",
       "       'hgvsc': 'ENST00000309234.10:c.845G>A',\n",
       "       'hgvsp': 'ENSP00000311698.6:p.Cys282Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311698',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000317896',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/5',\n",
       "       'hgvsc': 'ENST00000317896.11:c.569G>A',\n",
       "       'hgvsp': 'ENSP00000313776.7:p.Cys190Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000313776',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000336625',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/5',\n",
       "       'hgvsc': 'ENST00000336625.12:c.527G>A',\n",
       "       'hgvsp': 'ENSP00000337819.8:p.Cys176Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000337819',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000349999',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/5',\n",
       "       'hgvsc': 'ENST00000349999.8:c.581G>A',\n",
       "       'hgvsp': 'ENSP00000259699.6:p.Cys194Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000259699',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000352392',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000352392.8:c.77-206G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000315936',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000353147',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/4',\n",
       "       'hgvsc': 'ENST00000353147.9:c.305G>A',\n",
       "       'hgvsp': 'ENSP00000312342.5:p.Cys102Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000312342',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000357618',\n",
       "       'mane_select': 'NM_000410.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/6',\n",
       "       'hgvsc': 'ENST00000357618.10:c.845G>A',\n",
       "       'hgvsp': 'ENSP00000417404.1:p.Cys282Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417404',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000397022',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/6',\n",
       "       'hgvsc': 'ENST00000397022.7:c.776G>A',\n",
       "       'hgvsp': 'ENSP00000380217.3:p.Cys259Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000380217',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000461397',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/6',\n",
       "       'hgvsc': 'ENST00000461397.5:c.803G>A',\n",
       "       'hgvsp': 'ENSP00000420802.1:p.Cys268Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420802',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000470149',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/7',\n",
       "       'hgvsc': 'ENST00000470149.5:c.836G>A',\n",
       "       'hgvsp': 'ENSP00000419725.1:p.Cys279Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419725',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000485729',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417534',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'HFE',\n",
       "       'gene': 'ENSG00000010704',\n",
       "       'feature': 'ENST00000488199',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/5',\n",
       "       'hgvsc': 'ENST00000488199.5:c.539G>A',\n",
       "       'hgvsp': 'ENSP00000420559.1:p.Cys180Tyr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420559',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_1203_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['6-26090951-C-G'],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_1203_3',\n",
       "    'family': 'RGP_1203',\n",
       "    'gene': 'ENSG00000109103',\n",
       "    'var_data': {'coords': {'chrom': '17',\n",
       "      'pos': 28547800,\n",
       "      'ref': 'C',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 13.270000457763672,\n",
       "      'baseqranksum': 1.0800000429153442,\n",
       "      'dp': 52879,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 3.5999999046325684,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.42899999022483826,\n",
       "      'mq_dp': 60,\n",
       "      'qd': 13.270000457763672,\n",
       "      'raw_mq': 216000.0,\n",
       "      'readposranksum': 1.2000000476837158,\n",
       "      'sor': 0.4449999928474426,\n",
       "      'vqslod': 13.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 7.953360182000324e-06,\n",
       "      'gnomad_ex_an': 251466,\n",
       "      'gnomad_ex_ac': 2,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.314410019404022e-05,\n",
       "      'gnomad_an': 152160,\n",
       "      'gnomad_ac': 2,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 37.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,D,D',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000109103'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000301032',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/4',\n",
       "       'hgvsc': 'ENST00000301032.8:c.487G>T',\n",
       "       'hgvsp': 'ENSP00000301032.4:p.Glu163Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000301032',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000335765',\n",
       "       'mane_select': 'NM_005148.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/5',\n",
       "       'hgvsc': 'ENST00000335765.9:c.487G>T',\n",
       "       'hgvsp': 'ENSP00000337040.3:p.Glu163Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000337040',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000444148',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/5',\n",
       "       'hgvsc': 'ENST00000444148.1:c.466G>T',\n",
       "       'hgvsp': 'ENSP00000414639.1:p.Glu156Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000414639',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000470125',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000470125.5:c.202G>T',\n",
       "       'hgvsp': 'ENSP00000465323.1:p.Glu68Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000465323',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000484980',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/4',\n",
       "       'hgvsc': 'ENST00000484980.5:c.202G>T',\n",
       "       'hgvsp': 'ENSP00000466831.1:p.Glu68Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000466831',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'UNC119',\n",
       "       'gene': 'ENSG00000109103',\n",
       "       'feature': 'ENST00000581945',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000581945.1:c.*3G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000463893',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_1203_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F030171_rgp_1203',\n",
       "   'family_id': 'RGP_1203',\n",
       "   'members': {'RGP_1203_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F030171_rgp_1203'}},\n",
       "   'phenotypes': ['HP:0012719',\n",
       "    'HP:0002579',\n",
       "    'HP:0000364',\n",
       "    'HP:0200042',\n",
       "    'HP:0000975',\n",
       "    'HP:0001761',\n",
       "    'HP:0001780',\n",
       "    'HP:0030237',\n",
       "    'HP:0003394',\n",
       "    'HP:0007021',\n",
       "    'HP:0001260',\n",
       "    'HP:0002357',\n",
       "    'HP:0003457'],\n",
       "   'panel_ids': [3729, 3730, 137, 3087],\n",
       "   'panel_names': ['Hand and foot malformations',\n",
       "    'Limb and Digital Malformations SuperPanel',\n",
       "    'Mendeliome',\n",
       "    'Gastrointestinal neuromuscular disease']}},\n",
       " 'RGP_183_3': {'variants': [{'sample': 'RGP_183_3',\n",
       "    'family': 'RGP_183',\n",
       "    'gene': 'ENSG00000143631',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 152313385,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 64,\n",
       "      'af': 0.02199999988079071,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.022199999541044235,\n",
       "      'as_qd': 12.800000190734863,\n",
       "      'baseqranksum': 0.902999997138977,\n",
       "      'db': True,\n",
       "      'dp': 48573,\n",
       "      'excesshet': 6.045899868011475,\n",
       "      'fs': 0.5220000147819519,\n",
       "      'inbreedingcoeff': -0.022199999541044235,\n",
       "      'mleac': 64,\n",
       "      'mleaf': 0.02199999988079071,\n",
       "      'mq': 59.939998626708984,\n",
       "      'mqranksum': 0.0,\n",
       "      'mq_dp': 2142,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 12.800000190734863,\n",
       "      'raw_mq': 7695220.0,\n",
       "      'readposranksum': 0.3440000116825104,\n",
       "      'sor': 0.621999979019165,\n",
       "      'vqslod': 2.7899999618530273,\n",
       "      'culprit': 'DP',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 31358,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.009365430101752281,\n",
       "      'gnomad_ex_an': 251350,\n",
       "      'gnomad_ex_ac': 2354,\n",
       "      'gnomad_ex_hom': 18,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.010436099953949451,\n",
       "      'gnomad_an': 151398,\n",
       "      'gnomad_ac': 1580,\n",
       "      'gnomad_hom': 17,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 34.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000143631'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'FLG',\n",
       "       'gene': 'ENSG00000143631',\n",
       "       'feature': 'ENST00000368799',\n",
       "       'mane_select': 'NM_002016.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000368799.2:c.1501C>T',\n",
       "       'hgvsp': 'ENSP00000357789.1:p.Arg501Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357789',\n",
       "       'lof': 'LC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_183_1': 'WT', 'RGP_183_2': 'Het', 'RGP_183_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_183_3',\n",
       "    'family': 'RGP_183',\n",
       "    'gene': 'ENSG00000127990',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 94628288,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 15.779999732971191,\n",
       "      'baseqranksum': 1.2699999809265137,\n",
       "      'db': True,\n",
       "      'dp': 52025,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 3.3399999141693115,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.699999988079071,\n",
       "      'mq_dp': 68,\n",
       "      'qd': 15.770000457763672,\n",
       "      'raw_mq': 244800.0,\n",
       "      'readposranksum': 0.3709999918937683,\n",
       "      'sor': 1.1510000228881836,\n",
       "      'vqslod': 14.779999732971191,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 20808,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.3180600035411771e-05,\n",
       "      'gnomad_an': 151738,\n",
       "      'gnomad_ac': 2,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 38.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,A,A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000127990'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000415788',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000415788.3:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000405313.2:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000405313',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000428696',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000428696.7:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000397536.3:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000397536',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000437425',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000437425.7:c.181C>T',\n",
       "       'hgvsp': 'ENSP00000394061.2:p.Arg61Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000394061',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000445866',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000445866.7:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000398930.2:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000398930',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000447873',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000447873.6:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000388734.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388734',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000642394',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/9',\n",
       "       'hgvsc': 'ENST00000642394.1:c.181C>T',\n",
       "       'hgvsp': 'ENSP00000493751.1:p.Arg61Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493751',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000642441',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000642441.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495994.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495994',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000642707',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000642707.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000495270.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495270',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000642933',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000642933.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000496237.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496237',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000643020',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000643020.1:c.253C>T',\n",
       "       'hgvsp': 'ENSP00000494457.1:p.Arg85Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494457',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000643128',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/9',\n",
       "       'hgvsc': 'ENST00000643128.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000496678.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496678',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000643193',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000643193.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000496559.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496559',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000643272',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000643272.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000494488.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494488',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000643903',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/8',\n",
       "       'hgvsc': 'ENST00000643903.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495165.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495165',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644116',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000644116.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495276.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495276',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644122',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/13',\n",
       "       'hgvsc': 'ENST00000644122.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495236.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495236',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/13',\n",
       "       'hgvsc': 'ENST00000644375.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000494315.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494315',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644551',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000644551.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000493981.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493981',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644609',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000644609.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000496045.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496045',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644681',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000644681.1:c.31C>T',\n",
       "       'hgvsp': 'ENSP00000496455.1:p.Arg11Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496455',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644682',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000644682.1:c.259C>T',\n",
       "       'hgvsp': 'ENSP00000493509.1:p.Arg87Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493509',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000644816',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000644816.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000494898.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494898',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645101',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000645101.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000494975.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494975',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645109',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/10',\n",
       "       'hgvsc': 'ENST00000645109.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000495931.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495931',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645262',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000645262.1:c.181C>T',\n",
       "       'hgvsp': 'ENSP00000494164.1:p.Arg61Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494164',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645445',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000645445.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000493603.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493603',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645535',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000645535.1:c.31C>T',\n",
       "       'hgvsp': 'ENSP00000493984.1:p.Arg11Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493984',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000645725',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000645725.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495480.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495480',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646098',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000646098.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495591.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495591',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646119',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000646119.1:c.280C>T',\n",
       "       'hgvsp': 'ENSP00000494615.1:p.Arg94Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494615',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646137',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000646137.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495199.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495199',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646265',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000646265.1:c.394C>T',\n",
       "       'hgvsp': 'ENSP00000494587.1:p.Arg132Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494587',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646489',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000646489.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000496268.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496268',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646879',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/9',\n",
       "       'hgvsc': 'ENST00000646879.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000495209.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495209',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000646943',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/13',\n",
       "       'hgvsc': 'ENST00000646943.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000494666.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494666',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647018',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000647018.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000493722.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493722',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647048',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000647048.1:c.130C>T',\n",
       "       'hgvsp': 'ENSP00000495468.1:p.Arg44Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495468',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647096',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/13',\n",
       "       'hgvsc': 'ENST00000647096.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000494192.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494192',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647110',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/12',\n",
       "       'hgvsc': 'ENST00000647110.1:c.199C>T',\n",
       "       'hgvsp': 'ENSP00000494738.1:p.Arg67Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494738',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647334',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000647334.1:c.412C>T',\n",
       "       'hgvsp': 'ENSP00000495114.1:p.Arg138Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495114',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000647351',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/13',\n",
       "       'hgvsc': 'ENST00000647351.1:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000494556.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494556',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SGCE',\n",
       "       'gene': 'ENSG00000127990',\n",
       "       'feature': 'ENST00000648936',\n",
       "       'mane_select': 'NM_003919.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/11',\n",
       "       'hgvsc': 'ENST00000648936.2:c.304C>T',\n",
       "       'hgvsp': 'ENSP00000497130.1:p.Arg102Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497130',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_183_1': 'Het', 'RGP_183_2': 'WT', 'RGP_183_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_183_3',\n",
       "    'family': 'RGP_183',\n",
       "    'gene': 'ENSG00000055609',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 152222037,\n",
       "      'ref': 'GT',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.5577999949455261,\n",
       "      'as_qd': 16.790000915527344,\n",
       "      'baseqranksum': 0.5460000038146973,\n",
       "      'dp': 50408,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.5577999949455261,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -1.0260000228881836,\n",
       "      'mq_dp': 29,\n",
       "      'qd': 16.799999237060547,\n",
       "      'raw_mq': 104400.0,\n",
       "      'readposranksum': -0.7639999985694885,\n",
       "      'sor': 0.5690000057220459,\n",
       "      'vqslod': 0.722000002861023,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 654481,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.019999999552965164,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000055609'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000262189',\n",
       "       'mane_select': 'NM_170606.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/59',\n",
       "       'hgvsc': 'ENST00000262189.11:c.3462del',\n",
       "       'hgvsp': 'ENSP00000262189.6:p.Val1156Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000262189',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000418673',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/8',\n",
       "       'hgvsc': 'ENST00000418673.2:c.810del',\n",
       "       'hgvsp': 'ENSP00000403483.2:p.Val272Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000403483',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000679882',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '21/56',\n",
       "       'hgvsc': 'ENST00000679882.1:c.3342del',\n",
       "       'hgvsp': 'ENSP00000506154.1:p.Val1116Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000506154',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000680969',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/29',\n",
       "       'hgvsc': 'ENST00000680969.1:c.810del',\n",
       "       'hgvsp': 'ENSP00000505951.1:p.Val272Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000505951',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000681033',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/31',\n",
       "       'hgvsc': 'ENST00000681033.1:c.2163del',\n",
       "       'hgvsp': 'ENSP00000505058.1:p.Val723Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000505058',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000682176',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/20',\n",
       "       'hgvsc': 'ENST00000682176.1:c.139del',\n",
       "       'hgvsp': 'ENSP00000507665.1:p.Val49Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507665',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000682283',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/60',\n",
       "       'hgvsc': 'ENST00000682283.1:c.3462del',\n",
       "       'hgvsp': 'ENSP00000507485.1:p.Val1156Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507485',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000683200',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/44',\n",
       "       'hgvsc': 'ENST00000683200.1:c.810del',\n",
       "       'hgvsp': 'ENSP00000508052.1:p.Val272Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000508052',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000683490',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/25',\n",
       "       'hgvsc': 'ENST00000683490.1:c.3462del',\n",
       "       'hgvsp': 'ENSP00000507385.1:p.Val1156Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507385',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'KMT2C',\n",
       "       'gene': 'ENSG00000055609',\n",
       "       'feature': 'ENST00000684550',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/26',\n",
       "       'hgvsc': 'ENST00000684550.1:c.3462del',\n",
       "       'hgvsp': 'ENSP00000507135.1:p.Val1156Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507135',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_183_1': 'WT', 'RGP_183_2': 'WT', 'RGP_183_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic', 'Autism'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F019702_rgp_183',\n",
       "   'family_id': 'RGP_183',\n",
       "   'members': {'RGP_183_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019702_rgp_183'},\n",
       "    'RGP_183_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019702_rgp_183'},\n",
       "    'RGP_183_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F019702_rgp_183'}},\n",
       "   'phenotypes': ['HP:0000717',\n",
       "    'HP:0002579',\n",
       "    'HP:0001263',\n",
       "    'HP:0000365',\n",
       "    'HP:0001250',\n",
       "    'HP:0004322',\n",
       "    'HP:0004755'],\n",
       "   'panel_ids': [3241, 137, 202, 3087, 3631, 209, 48, 51, 3796, 183, 250, 254],\n",
       "   'panel_names': ['Deafness_Isolated',\n",
       "    'Mendeliome',\n",
       "    'Genetic Epilepsy',\n",
       "    'Gastrointestinal neuromuscular disease',\n",
       "    'Growth failure',\n",
       "    'Deafness_IsolatedAndComplex',\n",
       "    'Arrhythmogenic Cardiomyopathy',\n",
       "    'Autism',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Ventricular Fibrillation',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Arrhythmia_SuperPanel']}},\n",
       " 'RGP_19_3': {'variants': [{'sample': 'RGP_19_3',\n",
       "    'family': 'RGP_19',\n",
       "    'gene': 'ENSG00000277258',\n",
       "    'var_data': {'coords': {'chrom': '17',\n",
       "      'pos': 38739601,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.3095000088214874,\n",
       "      'as_qd': 13.529999732971191,\n",
       "      'baseqranksum': 0.22699999809265137,\n",
       "      'dp': 52848,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 1.406000018119812,\n",
       "      'inbreedingcoeff': 0.3095000088214874,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.25,\n",
       "      'mq_dp': 32,\n",
       "      'qd': 13.539999961853027,\n",
       "      'raw_mq': 115200.0,\n",
       "      'readposranksum': 0.1889999955892563,\n",
       "      'sor': 0.9860000014305115,\n",
       "      'vqslod': 13.949999809265137,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 610591,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 24.799999237060547,\n",
       "      'revel': 0.26100000739097595,\n",
       "      'mutationtaster': 'D,D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000277258'},\n",
       "     'transcript_consequences': [{'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000610747',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000483741',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000611883',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000611883.4:c.194C>T',\n",
       "       'hgvsp': 'ENSP00000478970.1:p.Pro65Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000478970',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000616129',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000616129.4:c.194C>T',\n",
       "       'hgvsp': 'ENSP00000484201.1:p.Pro65Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000484201',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000616199',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/12',\n",
       "       'hgvsc': 'ENST00000616199.4:c.194C>T',\n",
       "       'hgvsp': 'ENSP00000482063.1:p.Pro65Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000482063',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000618506',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/9',\n",
       "       'hgvsc': 'ENST00000618506.1:c.32C>T',\n",
       "       'hgvsp': 'ENSP00000484721.1:p.Pro11Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000484721',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000618941',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/10',\n",
       "       'hgvsc': 'ENST00000618941.4:c.194C>T',\n",
       "       'hgvsp': 'ENSP00000481656.1:p.Pro65Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000481656',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'PCGF2',\n",
       "       'gene': 'ENSG00000277258',\n",
       "       'feature': 'ENST00000620225',\n",
       "       'mane_select': 'NM_007144.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000620225.5:c.194C>T',\n",
       "       'hgvsp': 'ENSP00000482815.1:p.Pro65Leu',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000482815',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_19_1': 'WT', 'RGP_19_2': 'WT', 'RGP_19_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F019226_rgp_19',\n",
       "   'family_id': 'RGP_19',\n",
       "   'members': {'RGP_19_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019226_rgp_19'},\n",
       "    'RGP_19_2': {'sex': 'male', 'affected': False, 'ext_id': 'F019226_rgp_19'},\n",
       "    'RGP_19_3': {'sex': 'male', 'affected': True, 'ext_id': 'F019226_rgp_19'}},\n",
       "   'phenotypes': ['HP:0000717',\n",
       "    'HP:0012385',\n",
       "    'HP:0002019',\n",
       "    'HP:0009901',\n",
       "    'HP:0002020',\n",
       "    'HP:0001249',\n",
       "    'HP:0000369',\n",
       "    'HP:0001252',\n",
       "    'HP:0000505'],\n",
       "   'panel_ids': [51, 3078, 329, 137, 250, 47],\n",
       "   'panel_names': ['Autism',\n",
       "    'Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Arthrogryposis']}},\n",
       " 'RGP_218_3': {'variants': [{'sample': 'RGP_218_3',\n",
       "    'family': 'RGP_218',\n",
       "    'gene': 'ENSG00000177628',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 155235843,\n",
       "      'ref': 'T',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 17,\n",
       "      'af': 0.0057669999077916145,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.005900000222027302,\n",
       "      'as_qd': 12.5,\n",
       "      'baseqranksum': 0.2370000034570694,\n",
       "      'db': True,\n",
       "      'dp': 52689,\n",
       "      'excesshet': 3.2118000984191895,\n",
       "      'fs': 3.2170000076293945,\n",
       "      'inbreedingcoeff': -0.005900000222027302,\n",
       "      'mleac': 17,\n",
       "      'mleaf': 0.0057669999077916145,\n",
       "      'mq': 59.68000030517578,\n",
       "      'mqranksum': -0.10199999809265137,\n",
       "      'mq_dp': 706,\n",
       "      'negative_train_site': True,\n",
       "      'qd': 12.5,\n",
       "      'raw_mq': 2514150.0,\n",
       "      'readposranksum': 0.44699999690055847,\n",
       "      'sor': 0.6399999856948853,\n",
       "      'vqslod': -1.9210000038146973,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 19329,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0023106499575078487,\n",
       "      'gnomad_ex_an': 251444,\n",
       "      'gnomad_ex_ac': 581,\n",
       "      'gnomad_ex_hom': 4,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0018495400436222553,\n",
       "      'gnomad_an': 151930,\n",
       "      'gnomad_ac': 281,\n",
       "      'gnomad_hom': 3,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.20999999344348907,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 24.100000381469727,\n",
       "      'revel': 0.6729999780654907,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000177628'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'GBA',\n",
       "       'gene': 'ENSG00000177628',\n",
       "       'feature': 'ENST00000327247',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/12',\n",
       "       'hgvsc': 'ENST00000327247.9:c.1226A>G',\n",
       "       'hgvsp': 'ENSP00000314508.5:p.Asn409Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000314508',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'GBA',\n",
       "       'gene': 'ENSG00000177628',\n",
       "       'feature': 'ENST00000368373',\n",
       "       'mane_select': 'NM_000157.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/11',\n",
       "       'hgvsc': 'ENST00000368373.8:c.1226A>G',\n",
       "       'hgvsp': 'ENSP00000357357.3:p.Asn409Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357357',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'GBA',\n",
       "       'gene': 'ENSG00000177628',\n",
       "       'feature': 'ENST00000427500',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/10',\n",
       "       'hgvsc': 'ENST00000427500.7:c.1079A>G',\n",
       "       'hgvsp': 'ENSP00000402577.2:p.Asn360Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000402577',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'GBA',\n",
       "       'gene': 'ENSG00000177628',\n",
       "       'feature': 'ENST00000428024',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/10',\n",
       "       'hgvsc': 'ENST00000428024.3:c.965A>G',\n",
       "       'hgvsp': 'ENSP00000397986.2:p.Asn322Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000397986',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_218_1': 'Het', 'RGP_218_2': 'WT', 'RGP_218_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Genetic Epilepsy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_218_3',\n",
       "    'family': 'RGP_218',\n",
       "    'gene': 'ENSG00000198734',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 169549811,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 82,\n",
       "      'af': 0.02800000086426735,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.028699999675154686,\n",
       "      'as_qd': 13.069999694824219,\n",
       "      'baseqranksum': 0.5350000262260437,\n",
       "      'dp': 52499,\n",
       "      'excesshet': 8.042499542236328,\n",
       "      'fs': 1.093999981880188,\n",
       "      'inbreedingcoeff': -0.028699999675154686,\n",
       "      'mleac': 82,\n",
       "      'mleaf': 0.02800000086426735,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.1379999965429306,\n",
       "      'mq_dp': 3102,\n",
       "      'qd': 13.069999694824219,\n",
       "      'raw_mq': 11165400.0,\n",
       "      'readposranksum': 0.5220000147819519,\n",
       "      'sor': 0.6169999837875366,\n",
       "      'vqslod': 14.279999732971191,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 15681,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.017332499846816063,\n",
       "      'gnomad_an': 152200,\n",
       "      'gnomad_ac': 2638,\n",
       "      'gnomad_hom': 37,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'missing',\n",
       "      'cadd': 27.899999618530273,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000198734'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'F5',\n",
       "       'gene': 'ENSG00000198734',\n",
       "       'feature': 'ENST00000367796',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/25',\n",
       "       'hgvsc': 'ENST00000367796.3:c.1601G>A',\n",
       "       'hgvsp': 'ENSP00000356770.3:p.Arg534Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000356770',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'F5',\n",
       "       'gene': 'ENSG00000198734',\n",
       "       'feature': 'ENST00000367797',\n",
       "       'mane_select': 'NM_000130.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/25',\n",
       "       'hgvsc': 'ENST00000367797.9:c.1601G>A',\n",
       "       'hgvsp': 'ENSP00000356771.3:p.Arg534Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000356771',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_218_1': 'WT', 'RGP_218_2': 'Het', 'RGP_218_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_218_3',\n",
       "    'family': 'RGP_218',\n",
       "    'gene': 'ENSG00000081307',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 132675248,\n",
       "      'ref': 'G',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.43130001425743103,\n",
       "      'as_qd': 10.970000267028809,\n",
       "      'baseqranksum': 1.2400000095367432,\n",
       "      'dp': 50315,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.4311999976634979,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.3490000069141388,\n",
       "      'mq_dp': 73,\n",
       "      'qd': 10.970000267028809,\n",
       "      'raw_mq': 262800.0,\n",
       "      'readposranksum': 1.3799999952316284,\n",
       "      'sor': 0.6159999966621399,\n",
       "      'vqslod': 14.039999961853027,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.6299999952316284,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 34.0,\n",
       "      'revel': 0.4569999873638153,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000081307'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000264991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/12',\n",
       "       'hgvsc': 'ENST00000264991.8:c.645G>C',\n",
       "       'hgvsp': 'ENSP00000264991.4:p.Lys215Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000264991',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000356232',\n",
       "       'mane_select': 'NM_024818.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/12',\n",
       "       'hgvsc': 'ENST00000356232.10:c.813G>C',\n",
       "       'hgvsp': 'ENSP00000348565.4:p.Lys271Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000348565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000464068',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420055',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000473651',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/11',\n",
       "       'hgvsc': 'ENST00000473651.5:c.813G>C',\n",
       "       'hgvsp': 'ENSP00000424984.1:p.Lys271Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424984',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000489361',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417905',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000493720',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/12',\n",
       "       'hgvsc': 'ENST00000493720.6:c.813G>C',\n",
       "       'hgvsp': 'ENSP00000417879.2:p.Lys271Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417879',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000494238',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/12',\n",
       "       'hgvsc': 'ENST00000494238.6:c.645G>C',\n",
       "       'hgvsp': 'ENSP00000418807.2:p.Lys215Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418807',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000683741',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/13',\n",
       "       'hgvsc': 'ENST00000683741.1:c.984G>C',\n",
       "       'hgvsp': 'ENSP00000507396.1:p.Lys328Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507396',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['5', 'support']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_218_1': 'Het', 'RGP_218_2': 'WT', 'RGP_218_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-132675903-G-A'],\n",
       "    'flags': ['Microcephaly',\n",
       "     'Intellectual disability syndromic and non-syndromic',\n",
       "     'Genetic Epilepsy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_218_3',\n",
       "    'family': 'RGP_218',\n",
       "    'gene': 'ENSG00000081307',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 132675903,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 9,\n",
       "      'af': 0.0030529999639838934,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.2313999980688095,\n",
       "      'as_qd': 12.899999618530273,\n",
       "      'baseqranksum': 0.3970000147819519,\n",
       "      'db': True,\n",
       "      'dp': 50538,\n",
       "      'excesshet': 3.063499927520752,\n",
       "      'fs': 2.4749999046325684,\n",
       "      'inbreedingcoeff': 0.2313999980688095,\n",
       "      'mleac': 9,\n",
       "      'mleaf': 0.0030529999639838934,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.17599999904632568,\n",
       "      'mq_dp': 309,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 12.899999618530273,\n",
       "      'raw_mq': 1112400.0,\n",
       "      'readposranksum': 0.36500000953674316,\n",
       "      'sor': 0.5609999895095825,\n",
       "      'vqslod': 15.239999771118164,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 260377,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0018695499747991562,\n",
       "      'gnomad_ex_an': 243374,\n",
       "      'gnomad_ex_ac': 455,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0016192300245165825,\n",
       "      'gnomad_an': 151924,\n",
       "      'gnomad_ac': 246,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.799999237060547,\n",
       "      'revel': 0.6790000200271606,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000081307'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000264991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000264991.8:c.943G>A',\n",
       "       'hgvsp': 'ENSP00000264991.4:p.Ala315Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000264991',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000356232',\n",
       "       'mane_select': 'NM_024818.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000356232.10:c.1111G>A',\n",
       "       'hgvsp': 'ENSP00000348565.4:p.Ala371Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000348565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000464068',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420055',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000473651',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000473651.5:c.1024+223G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424984',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000493720',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000493720.6:c.1111G>A',\n",
       "       'hgvsp': 'ENSP00000417879.2:p.Ala371Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417879',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000494238',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000494238.6:c.943G>A',\n",
       "       'hgvsp': 'ENSP00000418807.2:p.Ala315Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418807',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000683741',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/13',\n",
       "       'hgvsc': 'ENST00000683741.1:c.1282G>A',\n",
       "       'hgvsp': 'ENSP00000507396.1:p.Ala428Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507396',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_218_1': 'WT', 'RGP_218_2': 'Het', 'RGP_218_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-132675248-G-C'],\n",
       "    'flags': ['Microcephaly',\n",
       "     'Intellectual disability syndromic and non-syndromic',\n",
       "     'Genetic Epilepsy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F019727_rgp_218',\n",
       "   'family_id': 'RGP_218',\n",
       "   'members': {'RGP_218_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019727_rgp_218'},\n",
       "    'RGP_218_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019727_rgp_218'},\n",
       "    'RGP_218_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F019727_rgp_218'}},\n",
       "   'phenotypes': ['HP:0000193',\n",
       "    'HP:0100704',\n",
       "    'HP:0012332',\n",
       "    'HP:0011968',\n",
       "    'HP:0001263',\n",
       "    'HP:0012469',\n",
       "    'HP:0005484',\n",
       "    'HP:0001250'],\n",
       "   'panel_ids': [3368, 137, 138, 3114, 202, 3796, 250],\n",
       "   'panel_names': ['Clefting disorders',\n",
       "    'Mendeliome',\n",
       "    'Microcephaly',\n",
       "    'Stickler Syndrome',\n",
       "    'Genetic Epilepsy',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_320_3': {'variants': [{'sample': 'RGP_320_3',\n",
       "    'family': 'RGP_320',\n",
       "    'gene': 'ENSG00000197912',\n",
       "    'var_data': {'coords': {'chrom': '16',\n",
       "      'pos': 89546737,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 10,\n",
       "      'af': 0.0033919999841600657,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.0035000001080334187,\n",
       "      'as_qd': 13.40999984741211,\n",
       "      'baseqranksum': 0.3050000071525574,\n",
       "      'db': True,\n",
       "      'dp': 54584,\n",
       "      'excesshet': 3.0768001079559326,\n",
       "      'fs': 0.5210000276565552,\n",
       "      'inbreedingcoeff': -0.0035000001080334187,\n",
       "      'mleac': 10,\n",
       "      'mleaf': 0.0033919999841600657,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.4860000014305115,\n",
       "      'mq_dp': 417,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.40999984741211,\n",
       "      'raw_mq': 1501200.0,\n",
       "      'readposranksum': 1.7799999713897705,\n",
       "      'sor': 0.7599999904632568,\n",
       "      'vqslod': 15.039999961853027,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 51184,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.002922720042988658,\n",
       "      'gnomad_ex_an': 251478,\n",
       "      'gnomad_ex_ac': 735,\n",
       "      'gnomad_ex_hom': 2,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0035878100898116827,\n",
       "      'gnomad_an': 152182,\n",
       "      'gnomad_ac': 546,\n",
       "      'gnomad_hom': 2,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 25.399999618530273,\n",
       "      'revel': 0.9229999780654907,\n",
       "      'mutationtaster': 'D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000197912'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000268704',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000268704.7:c.1508C>T',\n",
       "       'hgvsp': 'ENSP00000268704.3:p.Ala503Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000268704',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000643649',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/16',\n",
       "       'hgvsc': 'ENST00000643649.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000494806.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494806',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000644671',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/15',\n",
       "       'hgvsc': 'ENST00000644671.1:c.1187C>T',\n",
       "       'hgvsp': 'ENSP00000495999.1:p.Ala396Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495999',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000644781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000644781.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000495473.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495473',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645063',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/18',\n",
       "       'hgvsc': 'ENST00000645063.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000493590.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493590',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645818',\n",
       "       'mane_select': 'NM_003119.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000645818.2:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000495795.2:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495795',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645897',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000645897.1:c.1067C>T',\n",
       "       'hgvsp': 'ENSP00000495293.1:p.Ala356Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495293',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646303',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000646303.1:c.1397C>T',\n",
       "       'hgvsp': 'ENSP00000494160.1:p.Ala466Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494160',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646445',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/10',\n",
       "       'hgvsc': 'ENST00000646445.1:c.389C>T',\n",
       "       'hgvsp': 'ENSP00000496434.1:p.Ala130Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496434',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646716',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/12',\n",
       "       'hgvsc': 'ENST00000646716.1:c.581C>T',\n",
       "       'hgvsp': 'ENSP00000495593.1:p.Ala194Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495593',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000647032',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000647032.1:c.1145C>T',\n",
       "       'hgvsp': 'ENSP00000496047.1:p.Ala382Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496047',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000647079',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000647079.1:c.1121C>T',\n",
       "       'hgvsp': 'ENSP00000495967.1:p.Ala374Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495967',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_320_1': 'WT', 'RGP_320_2': 'Het', 'RGP_320_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_320_3',\n",
       "    'family': 'RGP_320',\n",
       "    'gene': 'ENSG00000130283',\n",
       "    'var_data': {'coords': {'chrom': '19',\n",
       "      'pos': 18869035,\n",
       "      'ref': 'G',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 6,\n",
       "      'af': 0.0020349998958408833,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.09109999984502792,\n",
       "      'as_qd': 12.079999923706055,\n",
       "      'baseqranksum': 0.08500000089406967,\n",
       "      'db': True,\n",
       "      'dp': 54848,\n",
       "      'excesshet': 3.032399892807007,\n",
       "      'fs': 3.763000011444092,\n",
       "      'inbreedingcoeff': 0.09109999984502792,\n",
       "      'mleac': 6,\n",
       "      'mleaf': 0.0020349998958408833,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.2460000067949295,\n",
       "      'mq_dp': 219,\n",
       "      'qd': 12.079999923706055,\n",
       "      'raw_mq': 788400.0,\n",
       "      'readposranksum': 0.8840000033378601,\n",
       "      'sor': 0.7570000290870667,\n",
       "      'vqslod': 14.510000228881836,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 21786,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 306,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0005362689844332635,\n",
       "      'gnomad_an': 147314,\n",
       "      'gnomad_ac': 79,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 33.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000130283'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'GDF1',\n",
       "       'gene': 'ENSG00000130283',\n",
       "       'feature': 'ENST00000247005',\n",
       "       'mane_select': 'NM_001492.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000247005.8:c.681C>A',\n",
       "       'hgvsp': 'ENSP00000247005.5:p.Cys227Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000247005',\n",
       "       'lof': 'LC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_320_1': 'Het', 'RGP_320_2': 'WT', 'RGP_320_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021077_rgp_320',\n",
       "   'family_id': 'RGP_320',\n",
       "   'members': {'RGP_320_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021077_rgp_320'},\n",
       "    'RGP_320_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021077_rgp_320'},\n",
       "    'RGP_320_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021077_rgp_320'}},\n",
       "   'phenotypes': ['HP:0001344',\n",
       "    'HP:0010536',\n",
       "    'HP:0010862',\n",
       "    'HP:0002194',\n",
       "    'HP:0001508',\n",
       "    'HP:0002540',\n",
       "    'HP:0001252'],\n",
       "   'panel_ids': [3078, 329, 137, 250, 3631],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Growth failure']}},\n",
       " 'RGP_362_3': {'variants': [{'sample': 'RGP_362_3',\n",
       "    'family': 'RGP_362',\n",
       "    'gene': 'ENSG00000141646',\n",
       "    'var_data': {'coords': {'chrom': '18',\n",
       "      'pos': 51078294,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 11.8100004196167,\n",
       "      'baseqranksum': 1.1200000047683716,\n",
       "      'dp': 50880,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.5239999890327454,\n",
       "      'mq_dp': 48,\n",
       "      'qd': 11.819999694824219,\n",
       "      'raw_mq': 172800.0,\n",
       "      'readposranksum': 1.3799999952316284,\n",
       "      'sor': 0.6389999985694885,\n",
       "      'vqslod': 14.109999656677246,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 94253,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 7.954439752211329e-06,\n",
       "      'gnomad_ex_an': 251432,\n",
       "      'gnomad_ex_ac': 2,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.12999999523162842,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.9610000252723694,\n",
       "      'mutationtaster': 'D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000141646'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SMAD4',\n",
       "       'gene': 'ENSG00000141646',\n",
       "       'feature': 'ENST00000342988',\n",
       "       'mane_select': 'NM_005359.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/12',\n",
       "       'hgvsc': 'ENST00000342988.8:c.1486C>T',\n",
       "       'hgvsp': 'ENSP00000341551.3:p.Arg496Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000341551',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SMAD4',\n",
       "       'gene': 'ENSG00000141646',\n",
       "       'feature': 'ENST00000398417',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/12',\n",
       "       'hgvsc': 'ENST00000398417.6:c.1486C>T',\n",
       "       'hgvsp': 'ENSP00000381452.1:p.Arg496Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000381452',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SMAD4',\n",
       "       'gene': 'ENSG00000141646',\n",
       "       'feature': 'ENST00000588745',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000588745.5:c.1198C>T',\n",
       "       'hgvsp': 'ENSP00000464901.1:p.Arg400Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464901',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'SMAD4',\n",
       "       'gene': 'ENSG00000141646',\n",
       "       'feature': 'ENST00000593223',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000466118',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_362_1': 'WT', 'RGP_362_2': 'WT', 'RGP_362_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021099_rgp_362',\n",
       "   'family_id': 'RGP_362',\n",
       "   'members': {'RGP_362_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021099_rgp_362'},\n",
       "    'RGP_362_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021099_rgp_362'},\n",
       "    'RGP_362_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021099_rgp_362'}},\n",
       "   'phenotypes': ['HP:0002028',\n",
       "    'HP:0002020',\n",
       "    'HP:0001508',\n",
       "    'HP:0001263',\n",
       "    'HP:0001376',\n",
       "    'HP:0000473'],\n",
       "   'panel_ids': [137, 3123, 290, 291, 89, 250, 3631],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Dystonia_Superpanel',\n",
       "    'Dystonia - complex',\n",
       "    'Dystonia - isolated/combined',\n",
       "    'Congenital Diarrhoea',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Growth failure']}},\n",
       " 'RGP_405_3': {'variants': [{'sample': 'RGP_405_3',\n",
       "    'family': 'RGP_405',\n",
       "    'gene': 'ENSG00000138081',\n",
       "    'var_data': {'coords': {'chrom': '2',\n",
       "      'pos': 47808215,\n",
       "      'ref': 'GAC',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 26.389999389648438,\n",
       "      'baseqranksum': 0.6119999885559082,\n",
       "      'dp': 51434,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 59.79999923706055,\n",
       "      'mqranksum': -0.07900000363588333,\n",
       "      'mq_dp': 38,\n",
       "      'qd': 26.399999618530273,\n",
       "      'raw_mq': 135904.0,\n",
       "      'readposranksum': -0.2669999897480011,\n",
       "      'sor': 0.6930000185966492,\n",
       "      'vqslod': 5.909999847412109,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 792598,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000138081'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'FBXO11',\n",
       "       'gene': 'ENSG00000138081',\n",
       "       'feature': 'ENST00000402508',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '23/23',\n",
       "       'hgvsc': 'ENST00000402508.5:c.2433_2434del',\n",
       "       'hgvsp': 'ENSP00000385398.1:p.Ser812Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000385398',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'FBXO11',\n",
       "       'gene': 'ENSG00000138081',\n",
       "       'feature': 'ENST00000403359',\n",
       "       'mane_select': 'NM_001190274.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '23/23',\n",
       "       'hgvsc': 'ENST00000403359.8:c.2685_2686del',\n",
       "       'hgvsp': 'ENSP00000384823.4:p.Ser896Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000384823',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'FBXO11',\n",
       "       'gene': 'ENSG00000138081',\n",
       "       'feature': 'ENST00000405808',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/4',\n",
       "       'hgvsc': 'ENST00000405808.5:c.147_148del',\n",
       "       'hgvsp': 'ENSP00000385127.1:p.Ser50Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000385127',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'FBXO11',\n",
       "       'gene': 'ENSG00000138081',\n",
       "       'feature': 'ENST00000493962',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000428118',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'FBXO11',\n",
       "       'gene': 'ENSG00000138081',\n",
       "       'feature': 'ENST00000683894',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '22/22',\n",
       "       'hgvsc': 'ENST00000683894.1:c.2514_2515del',\n",
       "       'hgvsp': 'ENSP00000507789.1:p.Ser839Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507789',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_405_1': 'WT', 'RGP_405_2': 'WT', 'RGP_405_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021127_rgp_405',\n",
       "   'family_id': 'RGP_405',\n",
       "   'members': {'RGP_405_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021127_rgp_405'},\n",
       "    'RGP_405_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021127_rgp_405'},\n",
       "    'RGP_405_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021127_rgp_405'}},\n",
       "   'phenotypes': ['HP:0000565',\n",
       "    'HP:0002020',\n",
       "    'HP:0001263',\n",
       "    'HP:0001252',\n",
       "    'HP:0001513'],\n",
       "   'panel_ids': [137, 3764, 3078, 151, 329, 250],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Severe early-onset obesity',\n",
       "    'Congenital Myasthenia',\n",
       "    'Overgrowth',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_409_3': {'variants': [{'sample': 'RGP_409_3',\n",
       "    'family': 'RGP_409',\n",
       "    'gene': 'ENSG00000021488',\n",
       "    'var_data': {'coords': {'chrom': '19',\n",
       "      'pos': 32864261,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 13.3100004196167,\n",
       "      'baseqranksum': 1.7799999713897705,\n",
       "      'db': True,\n",
       "      'dp': 57611,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 2.130000114440918,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.7730000019073486,\n",
       "      'mq_dp': 66,\n",
       "      'qd': 13.300000190734863,\n",
       "      'raw_mq': 237600.0,\n",
       "      'readposranksum': 0.23399999737739563,\n",
       "      'sor': 0.42500001192092896,\n",
       "      'vqslod': 13.729999542236328,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 20820,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.00027486999169923365,\n",
       "      'gnomad_ex_an': 251028,\n",
       "      'gnomad_ex_ac': 69,\n",
       "      'gnomad_ex_hom': 1,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.00019056600285694003,\n",
       "      'gnomad_an': 152178,\n",
       "      'gnomad_ac': 29,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 27.100000381469727,\n",
       "      'revel': 0.906000018119812,\n",
       "      'mutationtaster': 'A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000021488'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC7A9',\n",
       "       'gene': 'ENSG00000021488',\n",
       "       'feature': 'ENST00000023064',\n",
       "       'mane_select': 'NM_014270.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/13',\n",
       "       'hgvsc': 'ENST00000023064.9:c.313G>A',\n",
       "       'hgvsp': 'ENSP00000023064.3:p.Gly105Arg',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000023064',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC7A9',\n",
       "       'gene': 'ENSG00000021488',\n",
       "       'feature': 'ENST00000587772',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/13',\n",
       "       'hgvsc': 'ENST00000587772.1:c.313G>A',\n",
       "       'hgvsp': 'ENSP00000468439.1:p.Gly105Arg',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468439',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC7A9',\n",
       "       'gene': 'ENSG00000021488',\n",
       "       'feature': 'ENST00000590341',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/13',\n",
       "       'hgvsc': 'ENST00000590341.5:c.313G>A',\n",
       "       'hgvsp': 'ENSP00000464822.1:p.Gly105Arg',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464822',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_409_1': 'Het', 'RGP_409_2': 'WT', 'RGP_409_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021132_rgp_409',\n",
       "   'family_id': 'RGP_409',\n",
       "   'members': {'RGP_409_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021132_rgp_409'},\n",
       "    'RGP_409_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021132_rgp_409'},\n",
       "    'RGP_409_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021132_rgp_409'}},\n",
       "   'phenotypes': ['HP:0001344',\n",
       "    'HP:0011451',\n",
       "    'HP:0100704',\n",
       "    'HP:0001263',\n",
       "    'HP:0005692',\n",
       "    'HP:0001382',\n",
       "    'HP:0001252',\n",
       "    'HP:0011147'],\n",
       "   'panel_ids': [3796, 3078, 138, 329, 137, 250, 202],\n",
       "   'panel_names': ['Familial Generalised Epilepsy',\n",
       "    'Congenital Myasthenia',\n",
       "    'Microcephaly',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Genetic Epilepsy']}},\n",
       " 'RGP_416_3': {'variants': [{'sample': 'RGP_416_3',\n",
       "    'family': 'RGP_416',\n",
       "    'gene': 'ENSG00000165671',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 177267613,\n",
       "      'ref': 'G',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 15.779999732971191,\n",
       "      'baseqranksum': 1.1200000047683716,\n",
       "      'dp': 49021,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.8610000014305115,\n",
       "      'mq_dp': 50,\n",
       "      'qd': 15.789999961853027,\n",
       "      'raw_mq': 180000.0,\n",
       "      'readposranksum': 1.649999976158142,\n",
       "      'sor': 0.5830000042915344,\n",
       "      'vqslod': 13.680000305175781,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 830681,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 29.700000762939453,\n",
       "      'revel': 0.953000009059906,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000165671'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000347982',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/24',\n",
       "       'hgvsc': 'ENST00000347982.9:c.4325G>T',\n",
       "       'hgvsp': 'ENSP00000343209.5:p.Cys1442Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000343209',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000354179',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/24',\n",
       "       'hgvsc': 'ENST00000354179.9:c.4325G>T',\n",
       "       'hgvsp': 'ENSP00000346111.5:p.Cys1442Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000346111',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000439151',\n",
       "       'mane_select': 'NM_022455.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '15/23',\n",
       "       'hgvsc': 'ENST00000439151.7:c.5198G>T',\n",
       "       'hgvsp': 'ENSP00000395929.2:p.Cys1733Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000395929',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000503056',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000503056.6:c.-161G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424024',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000508029',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000508029.6:c.-161G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000425120',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000686993',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/24',\n",
       "       'hgvsc': 'ENST00000686993.1:c.4325G>T',\n",
       "       'hgvsp': 'ENSP00000510020.1:p.Cys1442Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000510020',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000687453',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/20',\n",
       "       'hgvsc': 'ENST00000687453.1:c.4889G>T',\n",
       "       'hgvsp': 'ENSP00000508426.1:p.Cys1630Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000508426',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'NSD1',\n",
       "       'gene': 'ENSG00000165671',\n",
       "       'feature': 'ENST00000689345',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/24',\n",
       "       'hgvsc': 'ENST00000689345.1:c.4325G>T',\n",
       "       'hgvsp': 'ENSP00000509711.1:p.Cys1442Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509711',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_416_1': 'WT', 'RGP_416_2': 'WT', 'RGP_416_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Autism',\n",
       "     'Overgrowth'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021138_rgp_416',\n",
       "   'family_id': 'RGP_416',\n",
       "   'members': {'RGP_416_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021138_rgp_416'},\n",
       "    'RGP_416_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021138_rgp_416'},\n",
       "    'RGP_416_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021138_rgp_416'}},\n",
       "   'phenotypes': ['HP:0000098',\n",
       "    'HP:0001252',\n",
       "    'HP:0007018',\n",
       "    'HP:0000717',\n",
       "    'HP:0000708',\n",
       "    'HP:0100543',\n",
       "    'HP:0001263',\n",
       "    'HP:0002003'],\n",
       "   'panel_ids': [137, 51, 3078, 151, 24, 329, 250],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Autism',\n",
       "    'Congenital Myasthenia',\n",
       "    'Overgrowth',\n",
       "    'Early-onset Dementia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_426_3': {'variants': [{'sample': 'RGP_426_3',\n",
       "    'family': 'RGP_426',\n",
       "    'gene': 'ENSG00000100393',\n",
       "    'var_data': {'coords': {'chrom': '22',\n",
       "      'pos': 41152220,\n",
       "      'ref': 'CTG',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 15.079999923706055,\n",
       "      'baseqranksum': -0.847000002861023,\n",
       "      'dp': 51155,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 2.5269999504089355,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.159999966621399,\n",
       "      'mq_dp': 49,\n",
       "      'qd': 15.09000015258789,\n",
       "      'raw_mq': 176400.0,\n",
       "      'readposranksum': -0.9520000219345093,\n",
       "      'sor': 0.7049999833106995,\n",
       "      'vqslod': 0.8830000162124634,\n",
       "      'culprit': 'MQRankSum',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918471,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000100393'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'EP300',\n",
       "       'gene': 'ENSG00000100393',\n",
       "       'feature': 'ENST00000263253',\n",
       "       'mane_select': 'NM_001429.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/31',\n",
       "       'hgvsc': 'ENST00000263253.9:c.3014_3015del',\n",
       "       'hgvsp': 'ENSP00000263253.7:p.Cys1005Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000263253',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'EP300',\n",
       "       'gene': 'ENSG00000100393',\n",
       "       'feature': 'ENST00000674155',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '15/30',\n",
       "       'hgvsc': 'ENST00000674155.1:c.2936_2937del',\n",
       "       'hgvsp': 'ENSP00000501078.1:p.Cys979Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000501078',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_426_1': 'WT', 'RGP_426_2': 'WT', 'RGP_426_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Autism',\n",
       "     'Callosome'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021470_rgp_426',\n",
       "   'family_id': 'RGP_426',\n",
       "   'members': {'RGP_426_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021470_rgp_426'},\n",
       "    'RGP_426_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021470_rgp_426'},\n",
       "    'RGP_426_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021470_rgp_426'}},\n",
       "   'phenotypes': ['HP:0000717',\n",
       "    'HP:0000678',\n",
       "    'HP:0002079',\n",
       "    'HP:0001249',\n",
       "    'HP:0009765',\n",
       "    'HP:0002119'],\n",
       "   'panel_ids': [51, 115, 137, 250, 205],\n",
       "   'panel_names': ['Autism',\n",
       "    'Hydrocephalus_Ventriculomegaly',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Callosome']}},\n",
       " 'RGP_437_3': {'variants': [{'sample': 'RGP_437_3',\n",
       "    'family': 'RGP_437',\n",
       "    'gene': 'ENSG00000143631',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 152311957,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 16.40999984741211,\n",
       "      'baseqranksum': -0.1120000034570694,\n",
       "      'db': True,\n",
       "      'dp': 49906,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 1.4850000143051147,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.9210000038146973,\n",
       "      'mq_dp': 29,\n",
       "      'qd': 16.43000030517578,\n",
       "      'raw_mq': 104400.0,\n",
       "      'readposranksum': -1.055999994277954,\n",
       "      'sor': 0.8009999990463257,\n",
       "      'vqslod': 14.59000015258789,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 259629,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.00011929299944313243,\n",
       "      'gnomad_ex_an': 251482,\n",
       "      'gnomad_ex_ac': 30,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.00015777800581417978,\n",
       "      'gnomad_an': 152112,\n",
       "      'gnomad_ac': 24,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 36.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000143631'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'FLG',\n",
       "       'gene': 'ENSG00000143631',\n",
       "       'feature': 'ENST00000368799',\n",
       "       'mane_select': 'NM_002016.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000368799.2:c.2929C>T',\n",
       "       'hgvsp': 'ENSP00000357789.1:p.Gln977Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357789',\n",
       "       'lof': 'LC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_437_1': 'WT', 'RGP_437_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_437_3',\n",
       "    'family': 'RGP_437',\n",
       "    'gene': 'ENSG00000049618',\n",
       "    'var_data': {'coords': {'chrom': '6',\n",
       "      'pos': 157200782,\n",
       "      'ref': 'C',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 8.880000114440918,\n",
       "      'baseqranksum': -0.367000013589859,\n",
       "      'dp': 51187,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 1.3769999742507935,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.22699999809265137,\n",
       "      'mq_dp': 34,\n",
       "      'qd': 8.890000343322754,\n",
       "      'raw_mq': 122400.0,\n",
       "      'readposranksum': -0.7870000004768372,\n",
       "      'sor': 0.3919999897480011,\n",
       "      'vqslod': 13.779999732971191,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918273,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.700000762939453,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000049618'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000346085',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '19/21',\n",
       "       'hgvsc': 'ENST00000346085.10:c.4437C>G',\n",
       "       'hgvsp': 'ENSP00000344546.5:p.Tyr1479Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000344546',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000350026',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/19',\n",
       "       'hgvsc': 'ENST00000350026.10:c.4149C>G',\n",
       "       'hgvsp': 'ENSP00000055163.7:p.Tyr1383Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000055163',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000414678',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/19',\n",
       "       'hgvsc': 'ENST00000414678.7:c.2715C>G',\n",
       "       'hgvsp': 'ENSP00000412835.2:p.Tyr905Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000412835',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000635849',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '15/17',\n",
       "       'hgvsc': 'ENST00000635849.1:c.1878C>G',\n",
       "       'hgvsp': 'ENSP00000490948.1:p.Tyr626Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490948',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000635957',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/13',\n",
       "       'hgvsc': 'ENST00000635957.1:c.1509C>G',\n",
       "       'hgvsp': 'ENSP00000490385.1:p.Tyr503Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490385',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000636930',\n",
       "       'mane_select': 'NM_001374828.1',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '18/20',\n",
       "       'hgvsc': 'ENST00000636930.2:c.4557C>G',\n",
       "       'hgvsp': 'ENSP00000490491.2:p.Tyr1519Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490491',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000637015',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/14',\n",
       "       'hgvsc': 'ENST00000637015.1:c.1926C>G',\n",
       "       'hgvsp': 'ENSP00000489729.1:p.Tyr642Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489729',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000637810',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/15',\n",
       "       'hgvsc': 'ENST00000637810.1:c.1899C>G',\n",
       "       'hgvsp': 'ENSP00000489636.1:p.Tyr633Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489636',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000637904',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '16/18',\n",
       "       'hgvsc': 'ENST00000637904.1:c.2058C>G',\n",
       "       'hgvsp': 'ENSP00000490550.1:p.Tyr686Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490550',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'ARID1B',\n",
       "       'gene': 'ENSG00000049618',\n",
       "       'feature': 'ENST00000647938',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '18/20',\n",
       "       'hgvsc': 'ENST00000647938.1:c.4188C>G',\n",
       "       'hgvsp': 'ENSP00000498155.1:p.Tyr1396Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498155',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_437_1': 'WT', 'RGP_437_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Genetic Epilepsy',\n",
       "     'Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021500_rgp_437',\n",
       "   'family_id': 'RGP_437',\n",
       "   'members': {'RGP_437_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021500_rgp_437'},\n",
       "    'RGP_437_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021500_rgp_437'}},\n",
       "   'phenotypes': ['HP:0000164',\n",
       "    'HP:0001488',\n",
       "    'HP:0000414',\n",
       "    'HP:0005824',\n",
       "    'HP:0000286',\n",
       "    'HP:0001263',\n",
       "    'HP:0000218',\n",
       "    'HP:0001007',\n",
       "    'HP:0000316',\n",
       "    'HP:0007925',\n",
       "    'HP:0000368',\n",
       "    'HP:0000252',\n",
       "    'HP:0001212',\n",
       "    'HP:0001250',\n",
       "    'HP:0003196',\n",
       "    'HP:0000219'],\n",
       "   'panel_ids': [3796, 3269, 138, 137, 250, 202],\n",
       "   'panel_names': ['Familial Generalised Epilepsy',\n",
       "    'Hair disorders',\n",
       "    'Microcephaly',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Genetic Epilepsy']}},\n",
       " 'RGP_440_3': {'variants': [{'sample': 'RGP_440_3',\n",
       "    'family': 'RGP_440',\n",
       "    'gene': 'ENSG00000078369',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 1806503,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 11.699999809265137,\n",
       "      'baseqranksum': 0.7850000262260437,\n",
       "      'dp': 50091,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 1.1890000104904175,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.2980000078678131,\n",
       "      'mq_dp': 40,\n",
       "      'qd': 11.710000038146973,\n",
       "      'raw_mq': 144000.0,\n",
       "      'readposranksum': 2.759999990463257,\n",
       "      'sor': 0.9139999747276306,\n",
       "      'vqslod': 13.40999984741211,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 205216,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 152188,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 27.100000381469727,\n",
       "      'revel': 0.5149999856948853,\n",
       "      'mutationtaster': 'D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000078369'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000378609',\n",
       "       'mane_select': 'NM_002074.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/12',\n",
       "       'hgvsc': 'ENST00000378609.9:c.239T>C',\n",
       "       'hgvsp': 'ENSP00000367872.3:p.Ile80Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000367872',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000434686',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/9',\n",
       "       'hgvsc': 'ENST00000434686.6:c.239T>C',\n",
       "       'hgvsp': 'ENSP00000392765.2:p.Ile80Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000392765',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000437146',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/6',\n",
       "       'hgvsc': 'ENST00000437146.1:c.239T>C',\n",
       "       'hgvsp': 'ENSP00000416651.1:p.Ile80Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000416651',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000439272',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/8',\n",
       "       'hgvsc': 'ENST00000439272.6:c.200T>C',\n",
       "       'hgvsp': 'ENSP00000399741.2:p.Ile67Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000399741',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000610897',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/11',\n",
       "       'hgvsc': 'ENST00000610897.4:c.239T>C',\n",
       "       'hgvsp': 'ENSP00000481878.1:p.Ile80Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000481878',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'GNB1',\n",
       "       'gene': 'ENSG00000078369',\n",
       "       'feature': 'ENST00000615252',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/10',\n",
       "       'hgvsc': 'ENST00000615252.4:c.-62T>C',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000483532',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_440_1': 'WT', 'RGP_440_2': 'WT', 'RGP_440_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_440_3',\n",
       "    'family': 'RGP_440',\n",
       "    'gene': 'ENSG00000197375',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 132370108,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 4,\n",
       "      'af': 0.0013569999719038606,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00139999995008111,\n",
       "      'as_qd': 13.210000038146973,\n",
       "      'baseqranksum': 0.2199999988079071,\n",
       "      'db': True,\n",
       "      'dp': 57408,\n",
       "      'excesshet': 3.0190999507904053,\n",
       "      'fs': 4.616000175476074,\n",
       "      'inbreedingcoeff': -0.00139999995008111,\n",
       "      'mleac': 4,\n",
       "      'mleaf': 0.0013569999719038606,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.9079999923706055,\n",
       "      'mq_dp': 166,\n",
       "      'qd': 13.210000038146973,\n",
       "      'raw_mq': 597600.0,\n",
       "      'readposranksum': 0.010999999940395355,\n",
       "      'sor': 0.4359999895095825,\n",
       "      'vqslod': 13.579999923706055,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 190414,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.00044634801452048123,\n",
       "      'gnomad_ex_an': 244204,\n",
       "      'gnomad_ex_ac': 109,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0004730180080514401,\n",
       "      'gnomad_an': 152214,\n",
       "      'gnomad_ac': 72,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.299999237060547,\n",
       "      'revel': 0.8989999890327454,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000197375'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000245407',\n",
       "       'mane_select': 'NM_003060.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/10',\n",
       "       'hgvsc': 'ENST00000245407.8:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000245407.3:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000245407',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000415928',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/8',\n",
       "       'hgvsc': 'ENST00000415928.6:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000388838.2:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388838',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000435065',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/11',\n",
       "       'hgvsc': 'ENST00000435065.7:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000402760.2:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000402760',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000689271',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/9',\n",
       "       'hgvsc': 'ENST00000689271.1:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000510797.1:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000510797',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000692413',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/10',\n",
       "       'hgvsc': 'ENST00000692413.1:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000509374.1:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509374',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000693308',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/10',\n",
       "       'hgvsc': 'ENST00000693308.1:c.136C>T',\n",
       "       'hgvsp': 'ENSP00000509770.1:p.Pro46Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509770',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_440_1': 'WT', 'RGP_440_2': 'Het', 'RGP_440_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['5-132370583-C-G'],\n",
       "    'flags': ['Myopathy - paediatric onset'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_440_3',\n",
       "    'family': 'RGP_440',\n",
       "    'gene': 'ENSG00000197375',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 132370583,\n",
       "      'ref': 'C',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 8,\n",
       "      'af': 0.002713999943807721,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00279999990016222,\n",
       "      'as_qd': 13.119999885559082,\n",
       "      'baseqranksum': 0.5709999799728394,\n",
       "      'dp': 54901,\n",
       "      'excesshet': 3.0517001152038574,\n",
       "      'fs': 2.7809998989105225,\n",
       "      'inbreedingcoeff': -0.00279999990016222,\n",
       "      'mleac': 8,\n",
       "      'mleaf': 0.002713999943807721,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.7699999809265137,\n",
       "      'mq_dp': 360,\n",
       "      'qd': 13.119999885559082,\n",
       "      'raw_mq': 1296000.0,\n",
       "      'readposranksum': 0.6499999761581421,\n",
       "      'sor': 0.7379999756813049,\n",
       "      'vqslod': 13.680000305175781,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.002884669927880168,\n",
       "      'gnomad_an': 152184,\n",
       "      'gnomad_ac': 439,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.6299999952316284,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 8.32800006866455,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000197375'},\n",
       "     'transcript_consequences': [{'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000245407',\n",
       "       'mane_select': 'NM_003060.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000245407.8:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000245407',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000415928',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000415928.6:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388838',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000435065',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000435065.7:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000402760',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000689271',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000689271.1:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000510797',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000692413',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000692413.1:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509374',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'SLC22A5',\n",
       "       'gene': 'ENSG00000197375',\n",
       "       'feature': 'ENST00000693308',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000693308.1:c.393+218C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509770',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['5']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_440_1': 'Het', 'RGP_440_2': 'WT', 'RGP_440_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['5-132370108-C-T'],\n",
       "    'flags': ['Myopathy - paediatric onset'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_440_3',\n",
       "    'family': 'RGP_440',\n",
       "    'gene': 'ENSG00000164266',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 147828115,\n",
       "      'ref': 'T',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 20,\n",
       "      'af': 0.006783999968320131,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.02449999935925007,\n",
       "      'as_qd': 12.130000114440918,\n",
       "      'baseqranksum': 0.6790000200271606,\n",
       "      'db': True,\n",
       "      'dp': 51018,\n",
       "      'excesshet': 3.292099952697754,\n",
       "      'fs': 1.093999981880188,\n",
       "      'inbreedingcoeff': 0.02449999935925007,\n",
       "      'mleac': 20,\n",
       "      'mleaf': 0.006783999968320131,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.40299999713897705,\n",
       "      'mq_dp': 729,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 12.130000114440918,\n",
       "      'raw_mq': 2624400.0,\n",
       "      'readposranksum': 0.5070000290870667,\n",
       "      'sor': 0.8069999814033508,\n",
       "      'vqslod': 15.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 28799,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.009062569588422775,\n",
       "      'gnomad_ex_an': 249598,\n",
       "      'gnomad_ex_ac': 2262,\n",
       "      'gnomad_ex_hom': 20,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.008252410218119621,\n",
       "      'gnomad_an': 152198,\n",
       "      'gnomad_ac': 1256,\n",
       "      'gnomad_hom': 7,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0010000000474974513,\n",
       "      'revel': 0.19900000095367432,\n",
       "      'mutationtaster': 'N,N',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000164266'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SPINK1',\n",
       "       'gene': 'ENSG00000164266',\n",
       "       'feature': 'ENST00000296695',\n",
       "       'mane_select': 'NM_001379610.1',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/4',\n",
       "       'hgvsc': 'ENST00000296695.10:c.101A>G',\n",
       "       'hgvsp': 'ENSP00000296695.5:p.Asn34Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000296695',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPINK1',\n",
       "       'gene': 'ENSG00000164266',\n",
       "       'feature': 'ENST00000510027',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000510027.2:c.101A>G',\n",
       "       'hgvsp': 'ENSP00000427376.1:p.Asn34Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000427376',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_440_1': 'Het', 'RGP_440_2': 'WT', 'RGP_440_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F021736_rgp_440',\n",
       "   'family_id': 'RGP_440',\n",
       "   'members': {'RGP_440_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021736_rgp_440'},\n",
       "    'RGP_440_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F021736_rgp_440'},\n",
       "    'RGP_440_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F021736_rgp_440'}},\n",
       "   'phenotypes': ['HP:0002750',\n",
       "    'HP:0002007',\n",
       "    'HP:0002342',\n",
       "    'HP:0000256',\n",
       "    'HP:0001270',\n",
       "    'HP:0001252',\n",
       "    'HP:0000639',\n",
       "    'HP:0002144'],\n",
       "   'panel_ids': [3762, 3379, 3078, 135, 329, 137, 250],\n",
       "   'panel_names': ['Congenital nystagmus',\n",
       "    'Congenital ophthalmoplegia',\n",
       "    'Congenital Myasthenia',\n",
       "    'Macrocephaly_Megalencephaly',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_441_2': {'variants': [],\n",
       "  'metadata': {'ext_id': 'F022164_rgp_441',\n",
       "   'family_id': 'RGP_441',\n",
       "   'members': {'RGP_441_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F022164_rgp_441'},\n",
       "    'RGP_441_2': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F022164_rgp_441'},\n",
       "    'RGP_441_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F022164_rgp_441'}},\n",
       "   'phenotypes': ['HP:0007302'],\n",
       "   'panel_ids': [137],\n",
       "   'panel_names': ['Mendeliome']}},\n",
       " 'RGP_441_3': {'variants': [],\n",
       "  'metadata': {'ext_id': 'F022164_rgp_441',\n",
       "   'family_id': 'RGP_441',\n",
       "   'members': {'RGP_441_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F022164_rgp_441'},\n",
       "    'RGP_441_2': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F022164_rgp_441'},\n",
       "    'RGP_441_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F022164_rgp_441'}},\n",
       "   'phenotypes': ['HP:0002540', 'HP:0004794', 'HP:0000252', 'HP:0000076'],\n",
       "   'panel_ids': [137, 138],\n",
       "   'panel_names': ['Mendeliome', 'Microcephaly']}},\n",
       " 'RGP_476_3': {'variants': [{'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117548628,\n",
       "      'ref': 'GTT',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 92,\n",
       "      'af': [0.0820000022649765,\n",
       "       0.024000000208616257,\n",
       "       0.03099999949336052,\n",
       "       0.000678399985190481],\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': [-0.01119999960064888,\n",
       "       0.04230000078678131,\n",
       "       0.040699999779462814,\n",
       "       0.2939999997615814],\n",
       "      'as_qd': [10.9399995803833,\n",
       "       4.860000133514404,\n",
       "       6.449999809265137,\n",
       "       0.20999999344348907],\n",
       "      'baseqranksum': 0.39500001072883606,\n",
       "      'db': True,\n",
       "      'dp': 52568,\n",
       "      'excesshet': 1.8043999671936035,\n",
       "      'fs': 2.427999973297119,\n",
       "      'inbreedingcoeff': 0.057100001722574234,\n",
       "      'mleac': [251, 75, 95, 2],\n",
       "      'mleaf': [0.08500000089406967,\n",
       "       0.02500000037252903,\n",
       "       0.03200000151991844,\n",
       "       0.000678399985190481],\n",
       "      'mq': 58.61000061035156,\n",
       "      'mqranksum': -0.12600000202655792,\n",
       "      'mq_dp': 12957,\n",
       "      'qd': 15.539999961853027,\n",
       "      'raw_mq': 44509100.0,\n",
       "      'readposranksum': 0.32199999690055847,\n",
       "      'sor': 0.8100000023841858,\n",
       "      'vqslod': 1.590000033378601,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 205614,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.02231140062212944,\n",
       "      'gnomad_ex_an': 205052,\n",
       "      'gnomad_ex_ac': 4575,\n",
       "      'gnomad_ex_hom': 13,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0357137992978096,\n",
       "      'gnomad_an': 142494,\n",
       "      'gnomad_ac': 5089,\n",
       "      'gnomad_hom': 122,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.05000000074505806,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 3.9200000762939453,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1210-7_1210-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1120-7_1120-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000673785',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000673785.1:c.967-7_967-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501235',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'WT', 'RGP_476_2': 'Het', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117548630-T-G'],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117548628,\n",
       "      'ref': 'GTT',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 92,\n",
       "      'af': [0.0820000022649765,\n",
       "       0.024000000208616257,\n",
       "       0.03099999949336052,\n",
       "       0.000678399985190481],\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': [-0.01119999960064888,\n",
       "       0.04230000078678131,\n",
       "       0.040699999779462814,\n",
       "       0.2939999997615814],\n",
       "      'as_qd': [10.9399995803833,\n",
       "       4.860000133514404,\n",
       "       6.449999809265137,\n",
       "       0.20999999344348907],\n",
       "      'baseqranksum': 0.39500001072883606,\n",
       "      'db': True,\n",
       "      'dp': 52568,\n",
       "      'excesshet': 1.8043999671936035,\n",
       "      'fs': 2.427999973297119,\n",
       "      'inbreedingcoeff': 0.057100001722574234,\n",
       "      'mleac': [251, 75, 95, 2],\n",
       "      'mleaf': [0.08500000089406967,\n",
       "       0.02500000037252903,\n",
       "       0.03200000151991844,\n",
       "       0.000678399985190481],\n",
       "      'mq': 58.61000061035156,\n",
       "      'mqranksum': -0.12600000202655792,\n",
       "      'mq_dp': 12957,\n",
       "      'qd': 15.539999961853027,\n",
       "      'raw_mq': 44509100.0,\n",
       "      'readposranksum': 0.32199999690055847,\n",
       "      'sor': 0.8100000023841858,\n",
       "      'vqslod': 1.590000033378601,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 205614,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.02231140062212944,\n",
       "      'gnomad_ex_an': 205052,\n",
       "      'gnomad_ex_ac': 4575,\n",
       "      'gnomad_ex_hom': 13,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0357137992978096,\n",
       "      'gnomad_an': 142494,\n",
       "      'gnomad_ac': 5089,\n",
       "      'gnomad_hom': 122,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.05000000074505806,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 3.9200000762939453,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1210-7_1210-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1120-7_1120-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1209+6526_1209+6527del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000673785',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000673785.1:c.967-7_967-6del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501235',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'WT', 'RGP_476_2': 'Het', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117559590-ATCT-A'],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117548630,\n",
       "      'ref': 'T',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 47,\n",
       "      'af': [0.01600000075995922, 0.03099999949336052, 0.0006788999889977276],\n",
       "      'an': 2946,\n",
       "      'as_inbreedingcoeff': [0.06729999929666519,\n",
       "       0.0348999984562397,\n",
       "       0.19910000264644623],\n",
       "      'as_qd': [4.989999771118164, 13.449999809265137, 0.38999998569488525],\n",
       "      'baseqranksum': 0.09399999678134918,\n",
       "      'db': True,\n",
       "      'dp': 52802,\n",
       "      'excesshet': 0.0,\n",
       "      'fs': 1.097000002861023,\n",
       "      'inbreedingcoeff': 0.24089999496936798,\n",
       "      'mleac': [47, 91, 3],\n",
       "      'mleaf': [0.01600000075995922,\n",
       "       0.03099999949336052,\n",
       "       0.0010179999517276883],\n",
       "      'mq': 59.06999969482422,\n",
       "      'mqranksum': 0.20399999618530273,\n",
       "      'mq_dp': 1190,\n",
       "      'qd': 3.440000057220459,\n",
       "      'raw_mq': 4151790.0,\n",
       "      'readposranksum': 0.9610000252723694,\n",
       "      'sor': 0.7139999866485596,\n",
       "      'vqslod': -0.9280999898910522,\n",
       "      'culprit': 'QD',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 174036,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.008609729818999767,\n",
       "      'gnomad_ex_an': 237638,\n",
       "      'gnomad_ex_ac': 2046,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.009419900365173817,\n",
       "      'gnomad_an': 142252,\n",
       "      'gnomad_ac': 1340,\n",
       "      'gnomad_hom': 10,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.07999999821186066,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 13.0600004196167,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1210-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1120-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000673785',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000673785.1:c.967-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501235',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'WT', 'RGP_476_2': 'WT', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117548628-GTT-G'],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117548630,\n",
       "      'ref': 'T',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 47,\n",
       "      'af': [0.01600000075995922, 0.03099999949336052, 0.0006788999889977276],\n",
       "      'an': 2946,\n",
       "      'as_inbreedingcoeff': [0.06729999929666519,\n",
       "       0.0348999984562397,\n",
       "       0.19910000264644623],\n",
       "      'as_qd': [4.989999771118164, 13.449999809265137, 0.38999998569488525],\n",
       "      'baseqranksum': 0.09399999678134918,\n",
       "      'db': True,\n",
       "      'dp': 52802,\n",
       "      'excesshet': 0.0,\n",
       "      'fs': 1.097000002861023,\n",
       "      'inbreedingcoeff': 0.24089999496936798,\n",
       "      'mleac': [47, 91, 3],\n",
       "      'mleaf': [0.01600000075995922,\n",
       "       0.03099999949336052,\n",
       "       0.0010179999517276883],\n",
       "      'mq': 59.06999969482422,\n",
       "      'mqranksum': 0.20399999618530273,\n",
       "      'mq_dp': 1190,\n",
       "      'qd': 3.440000057220459,\n",
       "      'raw_mq': 4151790.0,\n",
       "      'readposranksum': 0.9610000252723694,\n",
       "      'sor': 0.7139999866485596,\n",
       "      'vqslod': -0.9280999898910522,\n",
       "      'culprit': 'QD',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 174036,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.008609729818999767,\n",
       "      'gnomad_ex_an': 237638,\n",
       "      'gnomad_ex_ac': 2046,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.009419900365173817,\n",
       "      'gnomad_an': 142252,\n",
       "      'gnomad_ac': 1340,\n",
       "      'gnomad_hom': 10,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.07999999821186066,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 13.0600004196167,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1210-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1120-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1209+6522T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_polypyrimidine_tract_variant&intron_variant',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000673785',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000673785.1:c.967-11T>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501235',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'WT', 'RGP_476_2': 'WT', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117559590-ATCT-A'],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117559590,\n",
       "      'ref': 'ATCT',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 36,\n",
       "      'af': 0.012000000104308128,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.03830000013113022,\n",
       "      'as_qd': 19.079999923706055,\n",
       "      'baseqranksum': 0.1550000011920929,\n",
       "      'db': True,\n",
       "      'dp': 50321,\n",
       "      'excesshet': 3.949700117111206,\n",
       "      'fs': 0.5180000066757202,\n",
       "      'inbreedingcoeff': 0.03830000013113022,\n",
       "      'mleac': 36,\n",
       "      'mleaf': 0.012000000104308128,\n",
       "      'mq': 59.9900016784668,\n",
       "      'mqranksum': -0.4129999876022339,\n",
       "      'mq_dp': 1328,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 19.079999923706055,\n",
       "      'raw_mq': 4779700.0,\n",
       "      'readposranksum': -0.028999999165534973,\n",
       "      'sor': 0.6629999876022339,\n",
       "      'vqslod': 2.5799999237060547,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 4,\n",
       "      'clinvar_allele': 22144,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.007068490143865347,\n",
       "      'gnomad_ex_an': 251256,\n",
       "      'gnomad_ex_ac': 1776,\n",
       "      'gnomad_ex_hom': 1,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.007890170440077782,\n",
       "      'gnomad_an': 152088,\n",
       "      'gnomad_ac': 1200,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 19.209999084472656,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/27',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1521_1523del',\n",
       "       'hgvsp': 'ENSP00000003084.6:p.Phe508del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/26',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1431_1433del',\n",
       "       'hgvsp': 'ENSP00000389119.1:p.Phe478del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/17',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497957.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/21',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497965.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/26',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497203.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'Het', 'RGP_476_2': 'WT', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117548628-GTT-G'],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_476_3',\n",
       "    'family': 'RGP_476',\n",
       "    'gene': 'ENSG00000001626',\n",
       "    'var_data': {'coords': {'chrom': '7',\n",
       "      'pos': 117559590,\n",
       "      'ref': 'ATCT',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 36,\n",
       "      'af': 0.012000000104308128,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.03830000013113022,\n",
       "      'as_qd': 19.079999923706055,\n",
       "      'baseqranksum': 0.1550000011920929,\n",
       "      'db': True,\n",
       "      'dp': 50321,\n",
       "      'excesshet': 3.949700117111206,\n",
       "      'fs': 0.5180000066757202,\n",
       "      'inbreedingcoeff': 0.03830000013113022,\n",
       "      'mleac': 36,\n",
       "      'mleaf': 0.012000000104308128,\n",
       "      'mq': 59.9900016784668,\n",
       "      'mqranksum': -0.4129999876022339,\n",
       "      'mq_dp': 1328,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 19.079999923706055,\n",
       "      'raw_mq': 4779700.0,\n",
       "      'readposranksum': -0.028999999165534973,\n",
       "      'sor': 0.6629999876022339,\n",
       "      'vqslod': 2.5799999237060547,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 4,\n",
       "      'clinvar_allele': 22144,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.007068490143865347,\n",
       "      'gnomad_ex_an': 251256,\n",
       "      'gnomad_ex_ac': 1776,\n",
       "      'gnomad_ex_hom': 1,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.007890170440077782,\n",
       "      'gnomad_an': 152088,\n",
       "      'gnomad_ac': 1200,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 19.209999084472656,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000001626'},\n",
       "     'transcript_consequences': [{'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000003084',\n",
       "       'mane_select': 'NM_000492.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/27',\n",
       "       'hgvsc': 'ENST00000003084.11:c.1521_1523del',\n",
       "       'hgvsp': 'ENSP00000003084.6:p.Phe508del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000003084',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000426809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/26',\n",
       "       'hgvsc': 'ENST00000426809.5:c.1431_1433del',\n",
       "       'hgvsp': 'ENSP00000389119.1:p.Phe478del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000389119',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000648260',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/17',\n",
       "       'hgvsc': 'ENST00000648260.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497957.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497957',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649406',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/21',\n",
       "       'hgvsc': 'ENST00000649406.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497965.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497965',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'inframe_deletion',\n",
       "       'symbol': 'CFTR',\n",
       "       'gene': 'ENSG00000001626',\n",
       "       'feature': 'ENST00000649781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/26',\n",
       "       'hgvsc': 'ENST00000649781.1:c.1338_1340del',\n",
       "       'hgvsp': 'ENSP00000497203.1:p.Phe447del',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000497203',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_476_1': 'Het', 'RGP_476_2': 'WT', 'RGP_476_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['7-117548630-T-G'],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F022195_rgp_476',\n",
       "   'family_id': 'RGP_476',\n",
       "   'members': {'RGP_476_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F022195_rgp_476'},\n",
       "    'RGP_476_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F022195_rgp_476'},\n",
       "    'RGP_476_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F022195_rgp_476'}},\n",
       "   'phenotypes': ['HP:0003388',\n",
       "    'HP:0011968',\n",
       "    'HP:0001290',\n",
       "    'HP:0002123',\n",
       "    'HP:0001263',\n",
       "    'HP:0000540',\n",
       "    'HP:0002540',\n",
       "    'HP:0002370'],\n",
       "   'panel_ids': [137, 250, 3796, 3078, 329, 202, 331],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Genetic Epilepsy',\n",
       "    'Progressive Myoclonic Epilepsy']}},\n",
       " 'RGP_477_3': {'variants': [{'sample': 'RGP_477_3',\n",
       "    'family': 'RGP_477',\n",
       "    'gene': 'ENSG00000181222',\n",
       "    'var_data': {'coords': {'chrom': '17',\n",
       "      'pos': 7508291,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.3098999857902527,\n",
       "      'as_qd': 8.90999984741211,\n",
       "      'baseqranksum': 1.100000023841858,\n",
       "      'dp': 53627,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.30979999899864197,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.08699999749660492,\n",
       "      'mq_dp': 34,\n",
       "      'qd': 8.920000076293945,\n",
       "      'raw_mq': 122400.0,\n",
       "      'readposranksum': 0.9269999861717224,\n",
       "      'sor': 0.5830000042915344,\n",
       "      'vqslod': 13.979999542236328,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 861271,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 18.229999542236328,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000181222'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'POLR2A',\n",
       "       'gene': 'ENSG00000181222',\n",
       "       'feature': 'ENST00000674977',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '20/30',\n",
       "       'hgvsc': 'ENST00000674977.2:c.3281C>T',\n",
       "       'hgvsp': 'ENSP00000502190.2:p.Ser1094Phe',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000502190',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_477_1': 'WT', 'RGP_477_2': 'WT', 'RGP_477_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F023021_rgp_477',\n",
       "   'family_id': 'RGP_477',\n",
       "   'members': {'RGP_477_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023021_rgp_477'},\n",
       "    'RGP_477_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023021_rgp_477'},\n",
       "    'RGP_477_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F023021_rgp_477'}},\n",
       "   'phenotypes': ['HP:0000750', 'HP:0001290', 'HP:0001263'],\n",
       "   'panel_ids': [137, 3078, 329, 250],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_485_3': {'variants': [{'sample': 'RGP_485_3',\n",
       "    'family': 'RGP_485',\n",
       "    'gene': 'ENSG00000171316',\n",
       "    'var_data': {'coords': {'chrom': '8',\n",
       "      'pos': 60856553,\n",
       "      'ref': 'GC',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 15.890000343322754,\n",
       "      'baseqranksum': 0.41100001335144043,\n",
       "      'dp': 51636,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 2.742000102996826,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -1.027999997138977,\n",
       "      'mq_dp': 40,\n",
       "      'qd': 15.899999618530273,\n",
       "      'raw_mq': 144000.0,\n",
       "      'readposranksum': -1.5789999961853027,\n",
       "      'sor': 0.5419999957084656,\n",
       "      'vqslod': 0.31200000643730164,\n",
       "      'culprit': 'ReadPosRankSum',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918297,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.019999999552965164,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000171316'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CHD7',\n",
       "       'gene': 'ENSG00000171316',\n",
       "       'feature': 'ENST00000423902',\n",
       "       'mane_select': 'NM_017780.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '34/38',\n",
       "       'hgvsc': 'ENST00000423902.7:c.7276del',\n",
       "       'hgvsp': 'ENSP00000392028.1:p.Gln2426SerfsTer17',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000392028',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'CHD7',\n",
       "       'gene': 'ENSG00000171316',\n",
       "       'feature': 'ENST00000524602',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000524602.5:c.1717-5673del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000437061',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_485_1': 'WT', 'RGP_485_2': 'WT', 'RGP_485_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Anophthalmia_Microphthalmia_Coloboma',\n",
       "     'Deafness_Isolated',\n",
       "     'Deafness_IsolatedAndComplex',\n",
       "     'Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F023012_rgp_485',\n",
       "   'family_id': 'RGP_485',\n",
       "   'members': {'RGP_485_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023012_rgp_485'},\n",
       "    'RGP_485_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023012_rgp_485'},\n",
       "    'RGP_485_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F023012_rgp_485'}},\n",
       "   'phenotypes': ['HP:0000589',\n",
       "    'HP:0001263',\n",
       "    'HP:0000365',\n",
       "    'HP:0001562',\n",
       "    'HP:0002650',\n",
       "    'HP:0000076'],\n",
       "   'panel_ids': [209, 250, 3241, 137, 42],\n",
       "   'panel_names': ['Deafness_IsolatedAndComplex',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Deafness_Isolated',\n",
       "    'Mendeliome',\n",
       "    'Anophthalmia_Microphthalmia_Coloboma']}},\n",
       " 'RGP_508_3': {'variants': [],\n",
       "  'metadata': {'ext_id': 'F023347_rgp_508',\n",
       "   'family_id': 'RGP_508',\n",
       "   'members': {'RGP_508_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023347_rgp_508'},\n",
       "    'RGP_508_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F023347_rgp_508'},\n",
       "    'RGP_508_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F023347_rgp_508'}},\n",
       "   'phenotypes': ['HP:0100543', 'HP:0001290', 'HP:0001263', 'HP:0005517'],\n",
       "   'panel_ids': [3078, 24, 137, 250, 329],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Early-onset Dementia',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Myopathy - paediatric onset']}},\n",
       " 'RGP_56_3': {'variants': [{'sample': 'RGP_56_3',\n",
       "    'family': 'RGP_56',\n",
       "    'gene': 'ENSG00000165474',\n",
       "    'var_data': {'coords': {'chrom': '13',\n",
       "      'pos': 20189481,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 26,\n",
       "      'af': 0.008820000104606152,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.008899999782443047,\n",
       "      'as_qd': 13.109999656677246,\n",
       "      'baseqranksum': -0.18199999630451202,\n",
       "      'db': True,\n",
       "      'dp': 53641,\n",
       "      'excesshet': 3.493299961090088,\n",
       "      'fs': 1.725000023841858,\n",
       "      'inbreedingcoeff': -0.008899999782443047,\n",
       "      'mleac': 26,\n",
       "      'mleaf': 0.008820000104606152,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.14100000262260437,\n",
       "      'mq_dp': 1089,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.109999656677246,\n",
       "      'raw_mq': 3920400.0,\n",
       "      'readposranksum': 0.32199999690055847,\n",
       "      'sor': 0.5889999866485596,\n",
       "      'vqslod': 15.289999961853027,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 3,\n",
       "      'clinvar_allele': 32039,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.008678450249135494,\n",
       "      'gnomad_ex_an': 250736,\n",
       "      'gnomad_ex_ac': 2176,\n",
       "      'gnomad_ex_hom': 23,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.009419710375368595,\n",
       "      'gnomad_an': 152234,\n",
       "      'gnomad_ac': 1434,\n",
       "      'gnomad_hom': 9,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 20.899999618530273,\n",
       "      'revel': 0.7020000219345093,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000165474'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GJB2',\n",
       "       'gene': 'ENSG00000165474',\n",
       "       'feature': 'ENST00000382844',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/1',\n",
       "       'hgvsc': 'ENST00000382844.2:c.101T>C',\n",
       "       'hgvsp': 'ENSP00000372295.1:p.Met34Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000372295',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GJB2',\n",
       "       'gene': 'ENSG00000165474',\n",
       "       'feature': 'ENST00000382848',\n",
       "       'mane_select': 'NM_004004.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000382848.5:c.101T>C',\n",
       "       'hgvsp': 'ENSP00000372299.4:p.Met34Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000372299',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_56_1': 'WT', 'RGP_56_2': 'Het', 'RGP_56_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_56_3',\n",
       "    'family': 'RGP_56',\n",
       "    'gene': 'ENSG00000104044',\n",
       "    'var_data': {'coords': {'chrom': '15',\n",
       "      'pos': 27985101,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 14,\n",
       "      'af': 0.004749000072479248,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.004800000227987766,\n",
       "      'as_qd': 13.109999656677246,\n",
       "      'baseqranksum': 0.6159999966621399,\n",
       "      'db': True,\n",
       "      'dp': 55164,\n",
       "      'excesshet': 3.1449999809265137,\n",
       "      'fs': 0.5210000276565552,\n",
       "      'inbreedingcoeff': -0.004800000227987766,\n",
       "      'mleac': 14,\n",
       "      'mleaf': 0.004749000072479248,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.028999999165534973,\n",
       "      'mq_dp': 506,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.109999656677246,\n",
       "      'raw_mq': 1821600.0,\n",
       "      'readposranksum': 0.37299999594688416,\n",
       "      'sor': 0.7739999890327454,\n",
       "      'vqslod': 17.0,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 15994,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0030473899096250534,\n",
       "      'gnomad_ex_an': 250050,\n",
       "      'gnomad_ex_ac': 762,\n",
       "      'gnomad_ex_hom': 3,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0033718100748956203,\n",
       "      'gnomad_an': 152144,\n",
       "      'gnomad_ac': 513,\n",
       "      'gnomad_hom': 3,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 25.799999237060547,\n",
       "      'revel': 0.746999979019165,\n",
       "      'mutationtaster': 'A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000104044'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'OCA2',\n",
       "       'gene': 'ENSG00000104044',\n",
       "       'feature': 'ENST00000353809',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/23',\n",
       "       'hgvsc': 'ENST00000353809.9:c.1255G>A',\n",
       "       'hgvsp': 'ENSP00000261276.8:p.Val419Ile',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000261276',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'OCA2',\n",
       "       'gene': 'ENSG00000104044',\n",
       "       'feature': 'ENST00000354638',\n",
       "       'mane_select': 'NM_000275.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '13/24',\n",
       "       'hgvsc': 'ENST00000354638.8:c.1327G>A',\n",
       "       'hgvsp': 'ENSP00000346659.3:p.Val443Ile',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000346659',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_56_1': 'Het', 'RGP_56_2': 'WT', 'RGP_56_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_56_3',\n",
       "    'family': 'RGP_56',\n",
       "    'gene': 'ENSG00000105409',\n",
       "    'var_data': {'coords': {'chrom': '19',\n",
       "      'pos': 41970476,\n",
       "      'ref': 'A',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 10.0,\n",
       "      'baseqranksum': 1.0099999904632568,\n",
       "      'dp': 56506,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 4.460999965667725,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.1799999475479126,\n",
       "      'mq_dp': 43,\n",
       "      'qd': 10.010000228881836,\n",
       "      'raw_mq': 154800.0,\n",
       "      'readposranksum': 0.33500000834465027,\n",
       "      'sor': 0.8970000147819519,\n",
       "      'vqslod': 12.90999984741211,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 654885,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 28.600000381469727,\n",
       "      'revel': 0.9139999747276306,\n",
       "      'mutationtaster': 'D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000105409'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'ATP1A3',\n",
       "       'gene': 'ENSG00000105409',\n",
       "       'feature': 'ENST00000543770',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/23',\n",
       "       'hgvsc': 'ENST00000543770.5:c.2363T>A',\n",
       "       'hgvsp': 'ENSP00000437577.1:p.Ile788Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000437577',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ATP1A3',\n",
       "       'gene': 'ENSG00000105409',\n",
       "       'feature': 'ENST00000545399',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/23',\n",
       "       'hgvsc': 'ENST00000545399.6:c.2369T>A',\n",
       "       'hgvsp': 'ENSP00000444688.1:p.Ile790Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000444688',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ATP1A3',\n",
       "       'gene': 'ENSG00000105409',\n",
       "       'feature': 'ENST00000602133',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/23',\n",
       "       'hgvsc': 'ENST00000602133.5:c.2240T>A',\n",
       "       'hgvsp': 'ENSP00000471581.1:p.Ile747Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000471581',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ATP1A3',\n",
       "       'gene': 'ENSG00000105409',\n",
       "       'feature': 'ENST00000648268',\n",
       "       'mane_select': 'NM_152296.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '17/23',\n",
       "       'hgvsc': 'ENST00000648268.1:c.2330T>A',\n",
       "       'hgvsp': 'ENSP00000498113.1:p.Ile777Asn',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498113',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_56_1': 'WT', 'RGP_56_2': 'WT', 'RGP_56_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Dystonia - isolated/combined',\n",
       "     'Dystonia_Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F019251_rgp_56',\n",
       "   'family_id': 'RGP_56',\n",
       "   'members': {'RGP_56_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F019251_rgp_56'},\n",
       "    'RGP_56_2': {'sex': 'male', 'affected': False, 'ext_id': 'F019251_rgp_56'},\n",
       "    'RGP_56_3': {'sex': 'male', 'affected': True, 'ext_id': 'F019251_rgp_56'}},\n",
       "   'phenotypes': ['HP:0002020',\n",
       "    'HP:0100704',\n",
       "    'HP:0001508',\n",
       "    'HP:0006895',\n",
       "    'HP:0001385',\n",
       "    'HP:0001252',\n",
       "    'HP:0000938',\n",
       "    'HP:0012448',\n",
       "    'HP:0001263',\n",
       "    'HP:0002870',\n",
       "    'HP:0001332'],\n",
       "   'panel_ids': [290, 291, 3078, 329, 137, 3631, 3730, 3123, 250],\n",
       "   'panel_names': ['Dystonia - complex',\n",
       "    'Dystonia - isolated/combined',\n",
       "    'Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Growth failure',\n",
       "    'Limb and Digital Malformations SuperPanel',\n",
       "    'Dystonia_Superpanel',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_676_3': {'variants': [{'sample': 'RGP_676_3',\n",
       "    'family': 'RGP_676',\n",
       "    'gene': 'ENSG00000104884',\n",
       "    'var_data': {'coords': {'chrom': '19',\n",
       "      'pos': 45368929,\n",
       "      'ref': 'C',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 13.229999542236328,\n",
       "      'baseqranksum': 0.6169999837875366,\n",
       "      'dp': 51314,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 1.9119999408721924,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.6499999761581421,\n",
       "      'mq_dp': 75,\n",
       "      'qd': 13.220000267028809,\n",
       "      'raw_mq': 270000.0,\n",
       "      'readposranksum': 0.492000013589859,\n",
       "      'sor': 0.46399998664855957,\n",
       "      'vqslod': 14.09000015258789,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 6.569439847226022e-06,\n",
       "      'gnomad_an': 152220,\n",
       "      'gnomad_ac': 1,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 1.0,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 34.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000104884'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000391941',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000391941.6:c.174+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000375805',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000391944',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000391944.8:c.246+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000375808',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000391945',\n",
       "       'mane_select': 'NM_000400.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000391945.10:c.246+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000375809',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000485403',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000485403.6:c.174+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000431229',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000586131',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000586131.6:c.174+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000464887',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000586856',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000586856.1:c.96+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000466998',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant',\n",
       "       'symbol': 'ERCC2',\n",
       "       'gene': 'ENSG00000104884',\n",
       "       'feature': 'ENST00000684407',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000684407.1:c.123+1G>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507775',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['3', '5']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_676_1': 'WT', 'RGP_676_2': 'WT', 'RGP_676_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['19-45352228-A-G'],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026336_rgp_676',\n",
       "   'family_id': 'RGP_676',\n",
       "   'members': {'RGP_676_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026336_rgp_676'},\n",
       "    'RGP_676_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026336_rgp_676'},\n",
       "    'RGP_676_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026336_rgp_676'}},\n",
       "   'phenotypes': ['HP:0012469', 'HP:0001250'],\n",
       "   'panel_ids': [3796, 137, 202],\n",
       "   'panel_names': ['Familial Generalised Epilepsy',\n",
       "    'Mendeliome',\n",
       "    'Genetic Epilepsy']}},\n",
       " 'RGP_681_3': {'variants': [{'sample': 'RGP_681_3',\n",
       "    'family': 'RGP_681',\n",
       "    'gene': 'ENSG00000168036',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 41225731,\n",
       "      'ref': 'CT',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 13.029999732971191,\n",
       "      'baseqranksum': 1.8200000524520874,\n",
       "      'dp': 50613,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 10.35200023651123,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -1.9500000476837158,\n",
       "      'mq_dp': 35,\n",
       "      'negative_train_site': True,\n",
       "      'qd': 13.039999961853027,\n",
       "      'raw_mq': 126000.0,\n",
       "      'readposranksum': 0.03500000014901161,\n",
       "      'sor': 1.3049999475479126,\n",
       "      'vqslod': -0.8129000067710876,\n",
       "      'culprit': 'MQRankSum',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918237,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000168036'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000349496',\n",
       "       'mane_select': 'NM_001904.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000349496.11:c.807del',\n",
       "       'hgvsp': 'ENSP00000344456.5:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000344456',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000396183',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000396183.7:c.807del',\n",
       "       'hgvsp': 'ENSP00000379486.3:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000379486',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000396185',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000396185.8:c.807del',\n",
       "       'hgvsp': 'ENSP00000379488.3:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000379488',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000405570',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000405570.6:c.807del',\n",
       "       'hgvsp': 'ENSP00000385604.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000385604',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000426215',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000400508',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000431914',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000431914.6:c.807del',\n",
       "       'hgvsp': 'ENSP00000412219.2:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000412219',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000433400',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000433400.6:c.807del',\n",
       "       'hgvsp': 'ENSP00000387455.2:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000387455',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000441708',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000441708.2:c.807del',\n",
       "       'hgvsp': 'ENSP00000401599.2:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000401599',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000450969',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000450969.6:c.807del',\n",
       "       'hgvsp': 'ENSP00000409302.2:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000409302',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000453024',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000453024.6:c.786del',\n",
       "       'hgvsp': 'ENSP00000411226.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000411226',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642248',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000642248.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495244.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495244',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642315',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000642315.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495076.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495076',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642426',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000642426.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495719.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495719',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642836',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000642836.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000496295.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496295',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642886',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000642886.1:c.696del',\n",
       "       'hgvsp': 'ENSP00000496020.1:p.Met234TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496020',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642986',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/17',\n",
       "       'hgvsc': 'ENST00000642986.1:c.675del',\n",
       "       'hgvsp': 'ENSP00000494422.1:p.Met227TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494422',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000642992',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000642992.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000496385.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496385',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000643031',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000643031.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495450.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495450',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000643297',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000643297.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000494677.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494677',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000643541',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000643541.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000494411.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494411',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000643977',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000643977.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000494053.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494053',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000643992',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000643992.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000493610.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000493610',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000644138',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/13',\n",
       "       'hgvsc': 'ENST00000644138.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000496649.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496649',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000644524',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000644524.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000494780.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494780',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000644678',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000644678.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000495794.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495794',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000644867',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000644867.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495992.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495992',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000644873',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000644873.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000496511.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496511',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645210',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000645210.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000496180.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496180',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645276',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000645276.1:c.813del',\n",
       "       'hgvsp': 'ENSP00000494654.1:p.Met273TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494654',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645320',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000645320.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000495360.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495360',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645493',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000645493.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000494467.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494467',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645900',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000645900.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000495286.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495286',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645927',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495287',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000645982',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000645982.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000494845.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494845',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000646116',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000646116.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000495426.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495426',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000646174',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/14',\n",
       "       'hgvsc': 'ENST00000646174.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000495161.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000495161',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000646369',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000646369.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000494914.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494914',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000646381',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/17',\n",
       "       'hgvsc': 'ENST00000646381.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000496067.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496067',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000646725',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/17',\n",
       "       'hgvsc': 'ENST00000646725.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000496021.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000496021',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000647264',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/16',\n",
       "       'hgvsc': 'ENST00000647264.1:c.786del',\n",
       "       'hgvsp': 'ENSP00000494849.1:p.Met264TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494849',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000647390',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/16',\n",
       "       'hgvsc': 'ENST00000647390.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000493533.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000493533',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'CTNNB1',\n",
       "       'gene': 'ENSG00000168036',\n",
       "       'feature': 'ENST00000647413',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/15',\n",
       "       'hgvsc': 'ENST00000647413.1:c.807del',\n",
       "       'hgvsp': 'ENSP00000493583.1:p.Met271TrpfsTer5',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000493583',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_681_1': 'WT', 'RGP_681_2': 'WT', 'RGP_681_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026361_rgp_681',\n",
       "   'family_id': 'RGP_681',\n",
       "   'members': {'RGP_681_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026361_rgp_681'},\n",
       "    'RGP_681_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026361_rgp_681'},\n",
       "    'RGP_681_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026361_rgp_681'}},\n",
       "   'phenotypes': ['HP:0001999',\n",
       "    'HP:0002500',\n",
       "    'HP:0000164',\n",
       "    'HP:0001344',\n",
       "    'HP:0000717',\n",
       "    'HP:0002194',\n",
       "    'HP:0000565',\n",
       "    'HP:0001290',\n",
       "    'HP:0001263',\n",
       "    'HP:0001276',\n",
       "    'HP:0000817',\n",
       "    'HP:0000217'],\n",
       "   'panel_ids': [3078, 329, 137, 298, 299, 51, 3094, 250],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Mendeliome',\n",
       "    'Leukodystrophy - paediatric',\n",
       "    'Leukodystrophy - adult onset',\n",
       "    'Autism',\n",
       "    'Leukodystrophy_Superpanel',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_683_3': {'variants': [{'sample': 'RGP_683_3',\n",
       "    'family': 'RGP_683',\n",
       "    'gene': 'ENSG00000167508',\n",
       "    'var_data': {'coords': {'chrom': '16',\n",
       "      'pos': 88663006,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 8,\n",
       "      'af': 0.002713999943807721,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00279999990016222,\n",
       "      'as_qd': 12.460000038146973,\n",
       "      'baseqranksum': -0.03200000151991844,\n",
       "      'db': True,\n",
       "      'dp': 59648,\n",
       "      'excesshet': 3.0517001152038574,\n",
       "      'fs': 3.191999912261963,\n",
       "      'inbreedingcoeff': -0.00279999990016222,\n",
       "      'mleac': 8,\n",
       "      'mleaf': 0.002713999943807721,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.017000000923871994,\n",
       "      'mq_dp': 434,\n",
       "      'qd': 12.460000038146973,\n",
       "      'raw_mq': 1562400.0,\n",
       "      'readposranksum': 0.9789999723434448,\n",
       "      'sor': 0.7110000252723694,\n",
       "      'vqslod': 12.649999618530273,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 1336585,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0019698599353432655,\n",
       "      'gnomad_ex_an': 225904,\n",
       "      'gnomad_ex_ac': 445,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0029553500935435295,\n",
       "      'gnomad_an': 152266,\n",
       "      'gnomad_ac': 450,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.8600000143051147,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000167508'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_5th_base_variant&intron_variant',\n",
       "       'symbol': 'MVD',\n",
       "       'gene': 'ENSG00000167508',\n",
       "       'feature': 'ENST00000301012',\n",
       "       'mane_select': 'NM_002461.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000301012.8:c.70+5G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000301012',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_5th_base_variant&intron_variant',\n",
       "       'symbol': 'MVD',\n",
       "       'gene': 'ENSG00000167508',\n",
       "       'feature': 'ENST00000569177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000569177.5:c.70+5G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000455131',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '5']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_683_1': 'WT', 'RGP_683_2': 'Het', 'RGP_683_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_683_3',\n",
       "    'family': 'RGP_683',\n",
       "    'gene': 'ENSG00000131263',\n",
       "    'var_data': {'coords': {'chrom': 'X',\n",
       "      'pos': 74592156,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 3,\n",
       "      'af': 0.0010179999517276883,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.6345000267028809,\n",
       "      'as_qd': 21.459999084472656,\n",
       "      'baseqranksum': -0.4189999997615814,\n",
       "      'dp': 39614,\n",
       "      'excesshet': 0.002199999988079071,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.6344000101089478,\n",
       "      'mleac': 3,\n",
       "      'mleaf': 0.0010179999517276883,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.4189999997615814,\n",
       "      'mq_dp': 50,\n",
       "      'qd': 21.469999313354492,\n",
       "      'raw_mq': 180000.0,\n",
       "      'readposranksum': 0.16699999570846558,\n",
       "      'sor': 0.6140000224113464,\n",
       "      'vqslod': 15.970000267028809,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 247499,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 24.600000381469727,\n",
       "      'revel': 0.32600000500679016,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000131263'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'RLIM',\n",
       "       'gene': 'ENSG00000131263',\n",
       "       'feature': 'ENST00000332687',\n",
       "       'mane_select': 'NM_016120.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/4',\n",
       "       'hgvsc': 'ENST00000332687.11:c.1159C>T',\n",
       "       'hgvsp': 'ENSP00000328059.6:p.Arg387Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000328059',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'RLIM',\n",
       "       'gene': 'ENSG00000131263',\n",
       "       'feature': 'ENST00000349225',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/5',\n",
       "       'hgvsc': 'ENST00000349225.2:c.1159C>T',\n",
       "       'hgvsp': 'ENSP00000253571.3:p.Arg387Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000253571',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['X_Recessive Male'],\n",
       "    'genotypes': {'RGP_683_1': 'Het', 'RGP_683_2': 'WT', 'RGP_683_3': 'Hemi'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026449_rgp_683',\n",
       "   'family_id': 'RGP_683',\n",
       "   'members': {'RGP_683_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026449_rgp_683'},\n",
       "    'RGP_683_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026449_rgp_683'},\n",
       "    'RGP_683_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026449_rgp_683'}},\n",
       "   'phenotypes': ['HP:0011024',\n",
       "    'HP:0012332',\n",
       "    'HP:0100543',\n",
       "    'HP:0001508',\n",
       "    'HP:0001263',\n",
       "    'HP:0001252',\n",
       "    'HP:0004396',\n",
       "    'HP:0001250',\n",
       "    'HP:0004322'],\n",
       "   'panel_ids': [3078, 137, 202, 329, 3631, 3796, 24, 250],\n",
       "   'panel_names': ['Congenital Myasthenia',\n",
       "    'Mendeliome',\n",
       "    'Genetic Epilepsy',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Growth failure',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Early-onset Dementia',\n",
       "    'Intellectual disability syndromic and non-syndromic']}},\n",
       " 'RGP_692_3': {'variants': [{'sample': 'RGP_692_3',\n",
       "    'family': 'RGP_692',\n",
       "    'gene': 'ENSG00000081307',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 132668887,\n",
       "      'ref': 'G',\n",
       "      'alt': 'GGA'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.38580000400543213,\n",
       "      'as_qd': 17.8799991607666,\n",
       "      'baseqranksum': 0.34700000286102295,\n",
       "      'dp': 49924,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 3.0429999828338623,\n",
       "      'inbreedingcoeff': 0.38580000400543213,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.7429999709129333,\n",
       "      'mq_dp': 78,\n",
       "      'qd': 17.8799991607666,\n",
       "      'raw_mq': 280800.0,\n",
       "      'readposranksum': -0.21400000154972076,\n",
       "      'sor': 0.414000004529953,\n",
       "      'vqslod': 1.0700000524520874,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'missing',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000081307'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000264991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000264991.8:c.199_200insGA',\n",
       "       'hgvsp': 'ENSP00000264991.4:p.Ala67GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000264991',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000356232',\n",
       "       'mane_select': 'NM_024818.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000356232.10:c.367_368insGA',\n",
       "       'hgvsp': 'ENSP00000348565.4:p.Ala123GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000348565',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000464068',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/5',\n",
       "       'hgvsc': 'ENST00000464068.5:c.97_98insGA',\n",
       "       'hgvsp': 'ENSP00000420055.1:p.Ala33GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000420055',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000468022',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/6',\n",
       "       'hgvsc': 'ENST00000468022.5:c.199_200insGA',\n",
       "       'hgvsp': 'ENSP00000418569.1:p.Ala67GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000418569',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000473651',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/11',\n",
       "       'hgvsc': 'ENST00000473651.5:c.367_368insGA',\n",
       "       'hgvsp': 'ENSP00000424984.1:p.Ala123GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000424984',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000489361',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/4',\n",
       "       'hgvsc': 'ENST00000489361.1:c.199_200insGA',\n",
       "       'hgvsp': 'ENSP00000417905.1:p.Ala67GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000417905',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000493720',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000493720.6:c.367_368insGA',\n",
       "       'hgvsp': 'ENSP00000417879.2:p.Ala123GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000417879',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000494238',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000494238.6:c.199_200insGA',\n",
       "       'hgvsp': 'ENSP00000418807.2:p.Ala67GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000418807',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000683741',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/13',\n",
       "       'hgvsc': 'ENST00000683741.1:c.538_539insGA',\n",
       "       'hgvsp': 'ENSP00000507396.1:p.Ala180GlyfsTer4',\n",
       "       'variant_class': 'insertion',\n",
       "       'ensp': 'ENSP00000507396',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['3']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_692_1': 'WT', 'RGP_692_2': 'Het', 'RGP_692_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-132675903-G-A'],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Genetic Epilepsy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_692_3',\n",
       "    'family': 'RGP_692',\n",
       "    'gene': 'ENSG00000081307',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 132675903,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 9,\n",
       "      'af': 0.0030529999639838934,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.2313999980688095,\n",
       "      'as_qd': 12.899999618530273,\n",
       "      'baseqranksum': 0.3970000147819519,\n",
       "      'db': True,\n",
       "      'dp': 50538,\n",
       "      'excesshet': 3.063499927520752,\n",
       "      'fs': 2.4749999046325684,\n",
       "      'inbreedingcoeff': 0.2313999980688095,\n",
       "      'mleac': 9,\n",
       "      'mleaf': 0.0030529999639838934,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.17599999904632568,\n",
       "      'mq_dp': 309,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 12.899999618530273,\n",
       "      'raw_mq': 1112400.0,\n",
       "      'readposranksum': 0.36500000953674316,\n",
       "      'sor': 0.5609999895095825,\n",
       "      'vqslod': 15.239999771118164,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 260377,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0018695499747991562,\n",
       "      'gnomad_ex_an': 243374,\n",
       "      'gnomad_ex_ac': 455,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0016192300245165825,\n",
       "      'gnomad_an': 151924,\n",
       "      'gnomad_ac': 246,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.799999237060547,\n",
       "      'revel': 0.6790000200271606,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000081307'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000264991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000264991.8:c.943G>A',\n",
       "       'hgvsp': 'ENSP00000264991.4:p.Ala315Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000264991',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000356232',\n",
       "       'mane_select': 'NM_024818.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000356232.10:c.1111G>A',\n",
       "       'hgvsp': 'ENSP00000348565.4:p.Ala371Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000348565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000464068',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420055',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'intron_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000473651',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000473651.5:c.1024+223G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424984',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000493720',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000493720.6:c.1111G>A',\n",
       "       'hgvsp': 'ENSP00000417879.2:p.Ala371Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417879',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000494238',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/12',\n",
       "       'hgvsc': 'ENST00000494238.6:c.943G>A',\n",
       "       'hgvsp': 'ENSP00000418807.2:p.Ala315Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418807',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBA5',\n",
       "       'gene': 'ENSG00000081307',\n",
       "       'feature': 'ENST00000683741',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/13',\n",
       "       'hgvsc': 'ENST00000683741.1:c.1282G>A',\n",
       "       'hgvsp': 'ENSP00000507396.1:p.Ala428Thr',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507396',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_692_1': 'Het', 'RGP_692_2': 'WT', 'RGP_692_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-132668887-G-GGA'],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Genetic Epilepsy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026460_rgp_692',\n",
       "   'family_id': 'RGP_692',\n",
       "   'members': {'RGP_692_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026460_rgp_692'},\n",
       "    'RGP_692_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026460_rgp_692'},\n",
       "    'RGP_692_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026460_rgp_692'}},\n",
       "   'phenotypes': ['HP:0011968', 'HP:0001263', 'HP:0012469', 'HP:0001250'],\n",
       "   'panel_ids': [3796, 137, 250, 202],\n",
       "   'panel_names': ['Familial Generalised Epilepsy',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Genetic Epilepsy']}},\n",
       " 'RGP_697_3': {'variants': [{'sample': 'RGP_697_3',\n",
       "    'family': 'RGP_697',\n",
       "    'gene': 'ENSG00000109501',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 6301415,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 14.720000267028809,\n",
       "      'baseqranksum': 1.149999976158142,\n",
       "      'dp': 60983,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 2.434000015258789,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -1.3040000200271606,\n",
       "      'mq_dp': 50,\n",
       "      'qd': 14.729999542236328,\n",
       "      'raw_mq': 180000.0,\n",
       "      'readposranksum': 0.4090000092983246,\n",
       "      'sor': 0.7570000290870667,\n",
       "      'vqslod': 10.15999984741211,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918249,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 52.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000109501'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000226760',\n",
       "       'mane_select': 'NM_006005.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000226760.5:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000226760.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000226760',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000503569',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000503569.5:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000423337.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000423337',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000506362',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000506362.2:c.1371G>A',\n",
       "       'hgvsp': 'ENSP00000424103.2:p.Trp457Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424103',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673642',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000673642.1:c.1279G>A',\n",
       "       'hgvsp': 'ENSP00000501242.1:p.Val427Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501242',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000673991.1:c.1656G>A',\n",
       "       'hgvsp': 'ENSP00000501033.1:p.Trp552Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501033',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000682275',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000682275.1:c.1656G>A',\n",
       "       'hgvsp': 'ENSP00000507852.1:p.Trp552Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507852',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000683395',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000683395.1:c.*761G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507124',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000684087',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000684087.1:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000506978.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000506978',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_697_1': 'WT', 'RGP_697_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Cataract',\n",
       "     'Deafness_Isolated',\n",
       "     'Deafness_IsolatedAndComplex',\n",
       "     'Monogenic Diabetes',\n",
       "     'Optic Atrophy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_697_3',\n",
       "    'family': 'RGP_697',\n",
       "    'gene': 'ENSG00000109501',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 6301415,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 14.720000267028809,\n",
       "      'baseqranksum': 1.149999976158142,\n",
       "      'dp': 60983,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 2.434000015258789,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -1.3040000200271606,\n",
       "      'mq_dp': 50,\n",
       "      'qd': 14.729999542236328,\n",
       "      'raw_mq': 180000.0,\n",
       "      'readposranksum': 0.4090000092983246,\n",
       "      'sor': 0.7570000290870667,\n",
       "      'vqslod': 10.15999984741211,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918249,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 52.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000109501'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000226760',\n",
       "       'mane_select': 'NM_006005.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000226760.5:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000226760.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000226760',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000503569',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000503569.5:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000423337.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000423337',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000506362',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000506362.2:c.1371G>A',\n",
       "       'hgvsp': 'ENSP00000424103.2:p.Trp457Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424103',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673642',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000673642.1:c.1279G>A',\n",
       "       'hgvsp': 'ENSP00000501242.1:p.Val427Met',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501242',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000673991.1:c.1656G>A',\n",
       "       'hgvsp': 'ENSP00000501033.1:p.Trp552Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000501033',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000682275',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000682275.1:c.1656G>A',\n",
       "       'hgvsp': 'ENSP00000507852.1:p.Trp552Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507852',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000683395',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000683395.1:c.*761G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000507124',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000684087',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000684087.1:c.1620G>A',\n",
       "       'hgvsp': 'ENSP00000506978.1:p.Trp540Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000506978',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_697_1': 'WT', 'RGP_697_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['4-6301742-CTA-C'],\n",
       "    'flags': ['Cataract',\n",
       "     'Deafness_Isolated',\n",
       "     'Deafness_IsolatedAndComplex',\n",
       "     'Monogenic Diabetes',\n",
       "     'Optic Atrophy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_697_3',\n",
       "    'family': 'RGP_697',\n",
       "    'gene': 'ENSG00000109501',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 6301742,\n",
       "      'ref': 'CTA',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 20.860000610351562,\n",
       "      'baseqranksum': 0.23499999940395355,\n",
       "      'dp': 58127,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 5.01800012588501,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.20999999344348907,\n",
       "      'mq_dp': 102,\n",
       "      'qd': 20.860000610351562,\n",
       "      'raw_mq': 367200.0,\n",
       "      'readposranksum': -0.2770000100135803,\n",
       "      'sor': 0.9800000190734863,\n",
       "      'vqslod': -0.218299999833107,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918251,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000109501'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000226760',\n",
       "       'mane_select': 'NM_006005.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000226760.5:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000226760.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000226760',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000503569',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000503569.5:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000423337.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000423337',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000506362',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000506362.2:c.1700_1701del',\n",
       "       'hgvsp': 'ENSP00000424103.2:p.Tyr567CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000424103',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673642',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000673642.1:c.*324_*325del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000501242',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000673991.1:c.1985_1986del',\n",
       "       'hgvsp': 'ENSP00000501033.1:p.Tyr662CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000501033',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000682275',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000682275.1:c.1985_1986del',\n",
       "       'hgvsp': 'ENSP00000507852.1:p.Tyr662CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507852',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000683395',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000683395.1:c.*1090_*1091del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507124',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000684087',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000684087.1:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000506978.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000506978',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_697_1': 'Het', 'RGP_697_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Cataract',\n",
       "     'Deafness_Isolated',\n",
       "     'Deafness_IsolatedAndComplex',\n",
       "     'Monogenic Diabetes',\n",
       "     'Optic Atrophy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_697_3',\n",
       "    'family': 'RGP_697',\n",
       "    'gene': 'ENSG00000109501',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 6301742,\n",
       "      'ref': 'CTA',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 20.860000610351562,\n",
       "      'baseqranksum': 0.23499999940395355,\n",
       "      'dp': 58127,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 5.01800012588501,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.20999999344348907,\n",
       "      'mq_dp': 102,\n",
       "      'qd': 20.860000610351562,\n",
       "      'raw_mq': 367200.0,\n",
       "      'readposranksum': -0.2770000100135803,\n",
       "      'sor': 0.9800000190734863,\n",
       "      'vqslod': -0.218299999833107,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918251,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000109501'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000226760',\n",
       "       'mane_select': 'NM_006005.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000226760.5:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000226760.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000226760',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000503569',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000503569.5:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000423337.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000423337',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000506362',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000506362.2:c.1700_1701del',\n",
       "       'hgvsp': 'ENSP00000424103.2:p.Tyr567CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000424103',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673642',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000673642.1:c.*324_*325del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000501242',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000673991',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000673991.1:c.1985_1986del',\n",
       "       'hgvsp': 'ENSP00000501033.1:p.Tyr662CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000501033',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000682275',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000682275.1:c.1985_1986del',\n",
       "       'hgvsp': 'ENSP00000507852.1:p.Tyr662CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507852',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000683395',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000683395.1:c.*1090_*1091del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000507124',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'WFS1',\n",
       "       'gene': 'ENSG00000109501',\n",
       "       'feature': 'ENST00000684087',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/7',\n",
       "       'hgvsc': 'ENST00000684087.1:c.1949_1950del',\n",
       "       'hgvsp': 'ENSP00000506978.1:p.Tyr650CysfsTer61',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000506978',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_697_1': 'Het', 'RGP_697_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['4-6301415-G-A'],\n",
       "    'flags': ['Cataract',\n",
       "     'Deafness_Isolated',\n",
       "     'Deafness_IsolatedAndComplex',\n",
       "     'Monogenic Diabetes',\n",
       "     'Optic Atrophy'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_697_3',\n",
       "    'family': 'RGP_697',\n",
       "    'gene': 'ENSG00000105722',\n",
       "    'var_data': {'coords': {'chrom': '19',\n",
       "      'pos': 42250332,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 0.699999988079071,\n",
       "      'baseqranksum': 2.5399999618530273,\n",
       "      'dp': 54216,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 3.177999973297119,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.7929999828338623,\n",
       "      'mq_dp': 69,\n",
       "      'qd': 0.699999988079071,\n",
       "      'raw_mq': 248400.0,\n",
       "      'readposranksum': 0.382999986410141,\n",
       "      'sor': 1.4019999504089355,\n",
       "      'vqslod': 9.529999732971191,\n",
       "      'culprit': 'QD',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 70567,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.029999999329447746,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 33.0,\n",
       "      'revel': 0.7649999856948853,\n",
       "      'mutationtaster': 'D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000105722'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'ERF',\n",
       "       'gene': 'ENSG00000105722',\n",
       "       'feature': 'ENST00000222329',\n",
       "       'mane_select': 'NM_006494.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/4',\n",
       "       'hgvsc': 'ENST00000222329.9:c.256C>T',\n",
       "       'hgvsp': 'ENSP00000222329.3:p.Arg86Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000222329',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'ERF',\n",
       "       'gene': 'ENSG00000105722',\n",
       "       'feature': 'ENST00000440177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/4',\n",
       "       'hgvsc': 'ENST00000440177.6:c.31C>T',\n",
       "       'hgvsp': 'ENSP00000388173.2:p.Arg11Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388173',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'ERF',\n",
       "       'gene': 'ENSG00000105722',\n",
       "       'feature': 'ENST00000593944',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/4',\n",
       "       'hgvsc': 'ENST00000593944.5:c.31C>T',\n",
       "       'hgvsp': 'ENSP00000469274.1:p.Arg11Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000469274',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant&splice_region_variant',\n",
       "       'symbol': 'ERF',\n",
       "       'gene': 'ENSG00000105722',\n",
       "       'feature': 'ENST00000598965',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/3',\n",
       "       'hgvsc': 'ENST00000598965.1:c.31C>T',\n",
       "       'hgvsp': 'ENSP00000468962.1:p.Arg11Cys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000468962',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_697_1': 'WT', 'RGP_697_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['AB Ratio'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026576_rgp_697',\n",
       "   'family_id': 'RGP_697',\n",
       "   'members': {'RGP_697_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026576_rgp_697'},\n",
       "    'RGP_697_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026576_rgp_697'}},\n",
       "   'phenotypes': ['HP:0000618',\n",
       "    'HP:0000518',\n",
       "    'HP:0002069',\n",
       "    'HP:0000365',\n",
       "    'HP:0000648',\n",
       "    'HP:0000024',\n",
       "    'HP:0100651',\n",
       "    'HP:0000016'],\n",
       "   'panel_ids': [66, 3241, 137, 202, 43, 301, 209, 3796, 149, 3093],\n",
       "   'panel_names': ['Cataract',\n",
       "    'Deafness_Isolated',\n",
       "    'Mendeliome',\n",
       "    'Genetic Epilepsy',\n",
       "    'Eye Anterior Segment Abnormalities',\n",
       "    'Maturity-onset Diabetes of the Young',\n",
       "    'Deafness_IsolatedAndComplex',\n",
       "    'Familial Generalised Epilepsy',\n",
       "    'Optic Atrophy',\n",
       "    'Monogenic Diabetes']}},\n",
       " 'RGP_736_3': {'variants': [{'sample': 'RGP_736_3',\n",
       "    'family': 'RGP_736',\n",
       "    'gene': 'ENSG00000197912',\n",
       "    'var_data': {'coords': {'chrom': '16',\n",
       "      'pos': 89546737,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 10,\n",
       "      'af': 0.0033919999841600657,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.0035000001080334187,\n",
       "      'as_qd': 13.40999984741211,\n",
       "      'baseqranksum': 0.3050000071525574,\n",
       "      'db': True,\n",
       "      'dp': 54584,\n",
       "      'excesshet': 3.0768001079559326,\n",
       "      'fs': 0.5210000276565552,\n",
       "      'inbreedingcoeff': -0.0035000001080334187,\n",
       "      'mleac': 10,\n",
       "      'mleaf': 0.0033919999841600657,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.4860000014305115,\n",
       "      'mq_dp': 417,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.40999984741211,\n",
       "      'raw_mq': 1501200.0,\n",
       "      'readposranksum': 1.7799999713897705,\n",
       "      'sor': 0.7599999904632568,\n",
       "      'vqslod': 15.039999961853027,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 51184,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.002922720042988658,\n",
       "      'gnomad_ex_an': 251478,\n",
       "      'gnomad_ex_ac': 735,\n",
       "      'gnomad_ex_hom': 2,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0035878100898116827,\n",
       "      'gnomad_an': 152182,\n",
       "      'gnomad_ac': 546,\n",
       "      'gnomad_hom': 2,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 25.399999618530273,\n",
       "      'revel': 0.9229999780654907,\n",
       "      'mutationtaster': 'D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000197912'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000268704',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000268704.7:c.1508C>T',\n",
       "       'hgvsp': 'ENSP00000268704.3:p.Ala503Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000268704',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000643649',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/16',\n",
       "       'hgvsc': 'ENST00000643649.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000494806.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494806',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000644671',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/15',\n",
       "       'hgvsc': 'ENST00000644671.1:c.1187C>T',\n",
       "       'hgvsp': 'ENSP00000495999.1:p.Ala396Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495999',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000644781',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000644781.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000495473.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495473',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645063',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/18',\n",
       "       'hgvsc': 'ENST00000645063.1:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000493590.1:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000493590',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645818',\n",
       "       'mane_select': 'NM_003119.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000645818.2:c.1529C>T',\n",
       "       'hgvsp': 'ENSP00000495795.2:p.Ala510Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495795',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000645897',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000645897.1:c.1067C>T',\n",
       "       'hgvsp': 'ENSP00000495293.1:p.Ala356Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495293',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646303',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000646303.1:c.1397C>T',\n",
       "       'hgvsp': 'ENSP00000494160.1:p.Ala466Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494160',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646445',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/10',\n",
       "       'hgvsc': 'ENST00000646445.1:c.389C>T',\n",
       "       'hgvsp': 'ENSP00000496434.1:p.Ala130Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496434',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000646716',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/12',\n",
       "       'hgvsc': 'ENST00000646716.1:c.581C>T',\n",
       "       'hgvsp': 'ENSP00000495593.1:p.Ala194Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495593',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000647032',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/14',\n",
       "       'hgvsc': 'ENST00000647032.1:c.1145C>T',\n",
       "       'hgvsp': 'ENSP00000496047.1:p.Ala382Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496047',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPG7',\n",
       "       'gene': 'ENSG00000197912',\n",
       "       'feature': 'ENST00000647079',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000647079.1:c.1121C>T',\n",
       "       'hgvsp': 'ENSP00000495967.1:p.Ala374Val',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000495967',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_736_1': 'Het', 'RGP_736_2': 'WT', 'RGP_736_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F026894_rgp_736',\n",
       "   'family_id': 'RGP_736',\n",
       "   'members': {'RGP_736_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026894_rgp_736'},\n",
       "    'RGP_736_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F026894_rgp_736'},\n",
       "    'RGP_736_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F026894_rgp_736'}},\n",
       "   'phenotypes': ['HP:0001875'],\n",
       "   'panel_ids': [137],\n",
       "   'panel_names': ['Mendeliome']}},\n",
       " 'RGP_830_3': {'variants': [{'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000162669',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 91375551,\n",
       "      'ref': 'G',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 15.199999809265137,\n",
       "      'baseqranksum': 1.5099999904632568,\n",
       "      'db': True,\n",
       "      'dp': 51712,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 9.557999610900879,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.19200000166893005,\n",
       "      'mq_dp': 25,\n",
       "      'qd': 15.210000038146973,\n",
       "      'raw_mq': 90000.0,\n",
       "      'readposranksum': 1.0700000524520874,\n",
       "      'sor': 1.090000033378601,\n",
       "      'vqslod': 14.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 1.9724400772247463e-05,\n",
       "      'gnomad_an': 152096,\n",
       "      'gnomad_ac': 3,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.25999999046325684,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 36.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,D',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162669'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'HFM1',\n",
       "       'gene': 'ENSG00000162669',\n",
       "       'feature': 'ENST00000370425',\n",
       "       'mane_select': 'NM_001017975.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '12/39',\n",
       "       'hgvsc': 'ENST00000370425.8:c.1572C>G',\n",
       "       'hgvsp': 'ENSP00000359454.3:p.Tyr524Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000359454',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'HFM1',\n",
       "       'gene': 'ENSG00000162669',\n",
       "       'feature': 'ENST00000448819',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000397369',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000160710',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 154602065,\n",
       "      'ref': 'G',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 8,\n",
       "      'af': 0.002713999943807721,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00279999990016222,\n",
       "      'as_qd': 12.529999732971191,\n",
       "      'baseqranksum': -0.11400000005960464,\n",
       "      'db': True,\n",
       "      'dp': 52756,\n",
       "      'excesshet': 3.0517001152038574,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00279999990016222,\n",
       "      'mleac': 8,\n",
       "      'mleaf': 0.002713999943807721,\n",
       "      'mq': 59.91999816894531,\n",
       "      'mqranksum': -0.19900000095367432,\n",
       "      'mq_dp': 298,\n",
       "      'qd': 12.529999732971191,\n",
       "      'raw_mq': 1069820.0,\n",
       "      'readposranksum': 0.6650000214576721,\n",
       "      'sor': 0.6970000267028809,\n",
       "      'vqslod': 2.119999885559082,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 131928,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0022071099374443293,\n",
       "      'gnomad_ex_an': 251460,\n",
       "      'gnomad_ex_ac': 555,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.002299640094861388,\n",
       "      'gnomad_an': 152198,\n",
       "      'gnomad_ac': 350,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 24.600000381469727,\n",
       "      'revel': 0.7409999966621399,\n",
       "      'mutationtaster': 'D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000160710'},\n",
       "     'transcript_consequences': [{'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000368471',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000368471.8:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357456',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000368474',\n",
       "       'mane_select': 'NM_001111.5',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000368474.9:c.577C>G',\n",
       "       'hgvsp': 'ENSP00000357459.4:p.Pro193Ala',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000357459',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000529168',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000529168.2:c.577C>G',\n",
       "       'hgvsp': 'ENSP00000431794.2:p.Pro193Ala',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000431794',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000648231',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/16',\n",
       "       'hgvsc': 'ENST00000648231.2:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497555',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000648311',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000648311.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000498137',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000648871',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/5',\n",
       "       'hgvsc': 'ENST00000648871.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497793',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000649022',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/16',\n",
       "       'hgvsc': 'ENST00000649022.2:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000496896',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000649042',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/16',\n",
       "       'hgvsc': 'ENST00000649042.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497790',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000649724',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000649724.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497932',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000649749',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000649749.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000497210',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000679899',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000679899.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000505996',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000680270',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/16',\n",
       "       'hgvsc': 'ENST00000680270.1:c.-162C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000505532',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000680305',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/14',\n",
       "       'hgvsc': 'ENST00000680305.1:c.577C>G',\n",
       "       'hgvsp': 'ENSP00000506312.1:p.Pro193Ala',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000506312',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000681056',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/15',\n",
       "       'hgvsc': 'ENST00000681056.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000506234',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'ADAR',\n",
       "       'gene': 'ENSG00000160710',\n",
       "       'feature': 'ENST00000681683',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/16',\n",
       "       'hgvsc': 'ENST00000681683.1:c.-309C>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000506666',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000173540',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 49722057,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.41839998960494995,\n",
       "      'as_qd': 14.75,\n",
       "      'baseqranksum': 0.40299999713897705,\n",
       "      'db': True,\n",
       "      'dp': 57832,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.41839998960494995,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.09300000220537186,\n",
       "      'mq_dp': 48,\n",
       "      'qd': 14.760000228881836,\n",
       "      'raw_mq': 172800.0,\n",
       "      'readposranksum': 0.21699999272823334,\n",
       "      'sor': 0.8460000157356262,\n",
       "      'vqslod': 13.960000038146973,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 227736,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.00010401000326965004,\n",
       "      'gnomad_ex_an': 249976,\n",
       "      'gnomad_ex_ac': 26,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 6.573150312760845e-05,\n",
       "      'gnomad_an': 152134,\n",
       "      'gnomad_ac': 10,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.05999999865889549,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 30.0,\n",
       "      'revel': 0.546999990940094,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000173540'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000308375.10:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000309092.6:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000309092',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308388',\n",
       "       'mane_select': 'NM_021971.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/9',\n",
       "       'hgvsc': 'ENST00000308388.7:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000311130.6:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311130',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000480687',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/10',\n",
       "       'hgvsc': 'ENST00000480687.5:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000418565.1:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000495627',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/9',\n",
       "       'hgvsc': 'ENST00000495627.2:c.967C>T',\n",
       "       'hgvsp': 'ENSP00000503768.1:p.Arg323Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503768',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000677393',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/7',\n",
       "       'hgvsc': 'ENST00000677393.1:c.652C>T',\n",
       "       'hgvsp': 'ENSP00000503880.1:p.Arg218Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503880',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000678010',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/6',\n",
       "       'hgvsc': 'ENST00000678010.1:c.493C>T',\n",
       "       'hgvsp': 'ENSP00000503176.1:p.Arg165Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503176',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-49722937-G-T'],\n",
       "    'flags': ['Muscular dystrophy_Paediatric'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000173540',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 49722057,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.41839998960494995,\n",
       "      'as_qd': 14.75,\n",
       "      'baseqranksum': 0.40299999713897705,\n",
       "      'db': True,\n",
       "      'dp': 57832,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.41839998960494995,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.09300000220537186,\n",
       "      'mq_dp': 48,\n",
       "      'qd': 14.760000228881836,\n",
       "      'raw_mq': 172800.0,\n",
       "      'readposranksum': 0.21699999272823334,\n",
       "      'sor': 0.8460000157356262,\n",
       "      'vqslod': 13.960000038146973,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 227736,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.00010401000326965004,\n",
       "      'gnomad_ex_an': 249976,\n",
       "      'gnomad_ex_ac': 26,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 6.573150312760845e-05,\n",
       "      'gnomad_an': 152134,\n",
       "      'gnomad_ac': 10,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.05999999865889549,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 30.0,\n",
       "      'revel': 0.546999990940094,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000173540'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/8',\n",
       "       'hgvsc': 'ENST00000308375.10:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000309092.6:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000309092',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308388',\n",
       "       'mane_select': 'NM_021971.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/9',\n",
       "       'hgvsc': 'ENST00000308388.7:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000311130.6:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311130',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000480687',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/10',\n",
       "       'hgvsc': 'ENST00000480687.5:c.859C>T',\n",
       "       'hgvsp': 'ENSP00000418565.1:p.Arg287Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000495627',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/9',\n",
       "       'hgvsc': 'ENST00000495627.2:c.967C>T',\n",
       "       'hgvsp': 'ENSP00000503768.1:p.Arg323Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503768',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000677393',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/7',\n",
       "       'hgvsc': 'ENST00000677393.1:c.652C>T',\n",
       "       'hgvsp': 'ENSP00000503880.1:p.Arg218Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503880',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000678010',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/6',\n",
       "       'hgvsc': 'ENST00000678010.1:c.493C>T',\n",
       "       'hgvsp': 'ENSP00000503176.1:p.Arg165Trp',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503176',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-49723648-C-G'],\n",
       "    'flags': ['Muscular dystrophy_Paediatric'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000173540',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 49723648,\n",
       "      'ref': 'C',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 4,\n",
       "      'af': 0.0013569999719038606,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00139999995008111,\n",
       "      'as_qd': 13.399999618530273,\n",
       "      'baseqranksum': 0.34700000286102295,\n",
       "      'db': True,\n",
       "      'dp': 56015,\n",
       "      'excesshet': 3.0190999507904053,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00139999995008111,\n",
       "      'mleac': 4,\n",
       "      'mleaf': 0.0013569999719038606,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.1550000011920929,\n",
       "      'mq_dp': 159,\n",
       "      'qd': 13.399999618530273,\n",
       "      'raw_mq': 572400.0,\n",
       "      'readposranksum': -0.17000000178813934,\n",
       "      'sor': 0.6690000295639038,\n",
       "      'vqslod': 14.619999885559082,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 75127,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0007044529775157571,\n",
       "      'gnomad_ex_an': 222868,\n",
       "      'gnomad_ex_ac': 157,\n",
       "      'gnomad_ex_hom': 1,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0005188289796933532,\n",
       "      'gnomad_an': 152266,\n",
       "      'gnomad_ac': 79,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.399999618530273,\n",
       "      'revel': 0.6230000257492065,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000173540'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/8',\n",
       "       'hgvsc': 'ENST00000308375.10:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000309092.6:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000309092',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308388',\n",
       "       'mane_select': 'NM_021971.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/9',\n",
       "       'hgvsc': 'ENST00000308388.7:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000311130.6:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311130',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000480687',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000480687.5:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000418565.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000495627',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/9',\n",
       "       'hgvsc': 'ENST00000495627.2:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503768.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503768',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000677393',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/7',\n",
       "       'hgvsc': 'ENST00000677393.1:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503880.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503880',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000678010',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/6',\n",
       "       'hgvsc': 'ENST00000678010.1:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503176.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503176',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-49722057-G-A'],\n",
       "    'flags': ['Muscular dystrophy_Paediatric'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000173540',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 49723648,\n",
       "      'ref': 'C',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 4,\n",
       "      'af': 0.0013569999719038606,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00139999995008111,\n",
       "      'as_qd': 13.399999618530273,\n",
       "      'baseqranksum': 0.34700000286102295,\n",
       "      'db': True,\n",
       "      'dp': 56015,\n",
       "      'excesshet': 3.0190999507904053,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00139999995008111,\n",
       "      'mleac': 4,\n",
       "      'mleaf': 0.0013569999719038606,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.1550000011920929,\n",
       "      'mq_dp': 159,\n",
       "      'qd': 13.399999618530273,\n",
       "      'raw_mq': 572400.0,\n",
       "      'readposranksum': -0.17000000178813934,\n",
       "      'sor': 0.6690000295639038,\n",
       "      'vqslod': 14.619999885559082,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 75127,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0007044529775157571,\n",
       "      'gnomad_ex_an': 222868,\n",
       "      'gnomad_ex_ac': 157,\n",
       "      'gnomad_ex_hom': 1,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0005188289796933532,\n",
       "      'gnomad_an': 152266,\n",
       "      'gnomad_ac': 79,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 26.399999618530273,\n",
       "      'revel': 0.6230000257492065,\n",
       "      'mutationtaster': 'D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000173540'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/8',\n",
       "       'hgvsc': 'ENST00000308375.10:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000309092.6:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000309092',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000308388',\n",
       "       'mane_select': 'NM_021971.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/9',\n",
       "       'hgvsc': 'ENST00000308388.7:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000311130.6:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000311130',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000480687',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/10',\n",
       "       'hgvsc': 'ENST00000480687.5:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000418565.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418565',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000495627',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/9',\n",
       "       'hgvsc': 'ENST00000495627.2:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503768.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503768',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000677393',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/7',\n",
       "       'hgvsc': 'ENST00000677393.1:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503880.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503880',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'GMPPB',\n",
       "       'gene': 'ENSG00000173540',\n",
       "       'feature': 'ENST00000678010',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '1/6',\n",
       "       'hgvsc': 'ENST00000678010.1:c.79G>C',\n",
       "       'hgvsp': 'ENSP00000503176.1:p.Asp27His',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000503176',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['3-49722937-G-T'],\n",
       "    'flags': ['Muscular dystrophy_Paediatric'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_830_3',\n",
       "    'family': 'RGP_830',\n",
       "    'gene': 'ENSG00000134853',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 54281602,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 227,\n",
       "      'af': 0.07699999958276749,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.011900000274181366,\n",
       "      'as_qd': 13.989999771118164,\n",
       "      'baseqranksum': 0.7689999938011169,\n",
       "      'db': True,\n",
       "      'dp': 53658,\n",
       "      'excesshet': 1.6013000011444092,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.011900000274181366,\n",
       "      'mleac': 227,\n",
       "      'mleaf': 0.07699999958276749,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.125,\n",
       "      'mq_dp': 9171,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.989999771118164,\n",
       "      'raw_mq': 33011800.0,\n",
       "      'readposranksum': 0.3659999966621399,\n",
       "      'sor': 0.6930000185966492,\n",
       "      'vqslod': 19.010000228881836,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 137113,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.07246240228414536,\n",
       "      'gnomad_ex_an': 134042,\n",
       "      'gnomad_ex_ac': 9713,\n",
       "      'gnomad_ex_hom': 411,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.08386579900979996,\n",
       "      'gnomad_an': 152112,\n",
       "      'gnomad_ac': 12757,\n",
       "      'gnomad_hom': 582,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 5.215000152587891,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000134853'},\n",
       "     'transcript_consequences': [{'consequence': 'intron_variant',\n",
       "       'symbol': 'PDGFRA',\n",
       "       'gene': 'ENSG00000134853',\n",
       "       'feature': 'ENST00000257290',\n",
       "       'mane_select': 'NM_006206.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000257290.10:c.2323+1120C>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000257290',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_830_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027466_rgp_830',\n",
       "   'family_id': 'RGP_830',\n",
       "   'members': {'RGP_830_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027466_rgp_830'}},\n",
       "   'phenotypes': ['HP:0006785'],\n",
       "   'panel_ids': [137, 3730, 141],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Limb and Digital Malformations SuperPanel',\n",
       "    'Muscular dystrophy_Paediatric']}},\n",
       " 'RGP_854_3': {'variants': [{'sample': 'RGP_854_3',\n",
       "    'family': 'RGP_854',\n",
       "    'gene': 'ENSG00000162688',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 99876555,\n",
       "      'ref': 'TG',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.8029999732971191,\n",
       "      'as_qd': 33.75,\n",
       "      'dp': 51562,\n",
       "      'excesshet': 0.000699999975040555,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.8029999732971191,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mq_dp': 34,\n",
       "      'qd': 33.7400016784668,\n",
       "      'raw_mq': 122400.0,\n",
       "      'sor': 0.8180000185966492,\n",
       "      'vqslod': 0.9120000004768372,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 186641,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 3.981489953730488e-06,\n",
       "      'gnomad_ex_an': 251162,\n",
       "      'gnomad_ex_ac': 1,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162688'},\n",
       "     'transcript_consequences': [{'consequence': 'frameshift_variant',\n",
       "       'symbol': 'AGL',\n",
       "       'gene': 'ENSG00000162688',\n",
       "       'feature': 'ENST00000294724',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/34',\n",
       "       'hgvsc': 'ENST00000294724.8:c.1384del',\n",
       "       'hgvsp': 'ENSP00000294724.4:p.Val462Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000294724',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'AGL',\n",
       "       'gene': 'ENSG00000162688',\n",
       "       'feature': 'ENST00000361915',\n",
       "       'mane_select': 'NM_000642.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/34',\n",
       "       'hgvsc': 'ENST00000361915.8:c.1384del',\n",
       "       'hgvsp': 'ENSP00000355106.3:p.Val462Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000355106',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'AGL',\n",
       "       'gene': 'ENSG00000162688',\n",
       "       'feature': 'ENST00000370161',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/33',\n",
       "       'hgvsc': 'ENST00000370161.6:c.1336del',\n",
       "       'hgvsp': 'ENSP00000359180.2:p.Val446Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000359180',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'AGL',\n",
       "       'gene': 'ENSG00000162688',\n",
       "       'feature': 'ENST00000370163',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/34',\n",
       "       'hgvsc': 'ENST00000370163.7:c.1384del',\n",
       "       'hgvsp': 'ENSP00000359182.3:p.Val462Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000359182',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'AGL',\n",
       "       'gene': 'ENSG00000162688',\n",
       "       'feature': 'ENST00000370165',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/34',\n",
       "       'hgvsc': 'ENST00000370165.7:c.1384del',\n",
       "       'hgvsp': 'ENSP00000359184.3:p.Val462Ter',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000359184',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Recessive Homozygous'],\n",
       "    'genotypes': {'RGP_854_3': 'Hom'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Myopathy - adult onset',\n",
       "     'Myopathy Superpanel',\n",
       "     'Myopathy - paediatric onset'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_854_3',\n",
       "    'family': 'RGP_854',\n",
       "    'gene': 'ENSG00000163631',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 73415104,\n",
       "      'ref': 'A',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 6.360000133514404,\n",
       "      'baseqranksum': -1.2489999532699585,\n",
       "      'dp': 51143,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 6.3429999351501465,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.8600000143051147,\n",
       "      'mq_dp': 50,\n",
       "      'qd': 6.369999885559082,\n",
       "      'raw_mq': 180000.0,\n",
       "      'readposranksum': 0.7990000247955322,\n",
       "      'sor': 1.253000020980835,\n",
       "      'vqslod': 13.289999961853027,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'missing',\n",
       "      'clinvar_stars': 0,\n",
       "      'clinvar_allele': 0,\n",
       "      'clinvar_aip': 0,\n",
       "      'clinvar_aip_strong': 0,\n",
       "      'gnomad_ex_af': 3.97927988160518e-06,\n",
       "      'gnomad_ex_an': 251302,\n",
       "      'gnomad_ex_ac': 1,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 2.184999942779541,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'N,N,N,N,N',\n",
       "      'categoryboolean1': False,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000163631'},\n",
       "     'transcript_consequences': [{'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000295897',\n",
       "       'mane_select': 'NM_000477.7',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/15',\n",
       "       'hgvsc': 'ENST00000295897.9:c.1128A>G',\n",
       "       'hgvsp': 'ENSP00000295897.4:p.Thr376=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000295897',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000401494',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/13',\n",
       "       'hgvsc': 'ENST00000401494.7:c.783A>G',\n",
       "       'hgvsp': 'ENSP00000384695.3:p.Thr261=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000384695',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000415165',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/11',\n",
       "       'hgvsc': 'ENST00000415165.6:c.552A>G',\n",
       "       'hgvsp': 'ENSP00000401820.2:p.Thr184=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000401820',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000503124',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/13',\n",
       "       'hgvsc': 'ENST00000503124.5:c.678A>G',\n",
       "       'hgvsp': 'ENSP00000421027.1:p.Thr226=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000421027',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000509063',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/14',\n",
       "       'hgvsc': 'ENST00000509063.5:c.1128A>G',\n",
       "       'hgvsp': 'ENSP00000422784.1:p.Thr376=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000422784',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000511370',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/11',\n",
       "       'hgvsc': 'ENST00000511370.1:c.663A>G',\n",
       "       'hgvsp': 'ENSP00000426179.1:p.Thr221=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000426179',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_acceptor_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000621085',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000621085.4:c.491-2A>G',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000483421',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_region_variant&synonymous_variant',\n",
       "       'symbol': 'ALB',\n",
       "       'gene': 'ENSG00000163631',\n",
       "       'feature': 'ENST00000621628',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/12',\n",
       "       'hgvsc': 'ENST00000621628.4:c.489A>G',\n",
       "       'hgvsp': 'ENSP00000480485.1:p.Thr163=',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000480485',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_854_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027447_rgp_854',\n",
       "   'family_id': 'RGP_854',\n",
       "   'members': {'RGP_854_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027447_rgp_854'}},\n",
       "   'phenotypes': ['HP:0003236', 'HP:0004303', 'HP:0001324', 'HP:0009055'],\n",
       "   'panel_ids': [3101, 328, 137, 141, 329],\n",
       "   'panel_names': ['Myopathy Superpanel',\n",
       "    'Myopathy - adult onset',\n",
       "    'Mendeliome',\n",
       "    'Muscular dystrophy_Paediatric',\n",
       "    'Myopathy - paediatric onset']}},\n",
       " 'RGP_904_3': {'variants': [{'sample': 'RGP_904_3',\n",
       "    'family': 'RGP_904',\n",
       "    'gene': 'ENSG00000134853',\n",
       "    'var_data': {'coords': {'chrom': '4',\n",
       "      'pos': 54281602,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 227,\n",
       "      'af': 0.07699999958276749,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.011900000274181366,\n",
       "      'as_qd': 13.989999771118164,\n",
       "      'baseqranksum': 0.7689999938011169,\n",
       "      'db': True,\n",
       "      'dp': 53658,\n",
       "      'excesshet': 1.6013000011444092,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': 0.011900000274181366,\n",
       "      'mleac': 227,\n",
       "      'mleaf': 0.07699999958276749,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.125,\n",
       "      'mq_dp': 9171,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 13.989999771118164,\n",
       "      'raw_mq': 33011800.0,\n",
       "      'readposranksum': 0.3659999966621399,\n",
       "      'sor': 0.6930000185966492,\n",
       "      'vqslod': 19.010000228881836,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 137113,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.07246240228414536,\n",
       "      'gnomad_ex_an': 134042,\n",
       "      'gnomad_ex_ac': 9713,\n",
       "      'gnomad_ex_hom': 411,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.08386579900979996,\n",
       "      'gnomad_an': 152112,\n",
       "      'gnomad_ac': 12757,\n",
       "      'gnomad_hom': 582,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 5.215000152587891,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000134853'},\n",
       "     'transcript_consequences': [{'consequence': 'intron_variant',\n",
       "       'symbol': 'PDGFRA',\n",
       "       'gene': 'ENSG00000134853',\n",
       "       'feature': 'ENST00000257290',\n",
       "       'mane_select': 'NM_006206.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000257290.10:c.2323+1120C>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000257290',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_904_1': 'WT', 'RGP_904_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_904_3',\n",
       "    'family': 'RGP_904',\n",
       "    'gene': 'ENSG00000108312',\n",
       "    'var_data': {'coords': {'chrom': '17',\n",
       "      'pos': 44212851,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 12.680000305175781,\n",
       "      'baseqranksum': -0.47200000286102295,\n",
       "      'dp': 54616,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 4.433000087738037,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.49000000953674316,\n",
       "      'mq_dp': 53,\n",
       "      'qd': 12.6899995803833,\n",
       "      'raw_mq': 190800.0,\n",
       "      'readposranksum': 0.7929999828338623,\n",
       "      'sor': 1.5549999475479126,\n",
       "      'vqslod': 13.90999984741211,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 431537,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 25.0,\n",
       "      'revel': 0.6830000281333923,\n",
       "      'mutationtaster': 'D,D,D,D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000108312'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000302904',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/21',\n",
       "       'hgvsc': 'ENST00000302904.8:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000302640.4:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000302640',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000343638',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/20',\n",
       "       'hgvsc': 'ENST00000343638.9:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000345297.5:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000345297',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000393606',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/20',\n",
       "       'hgvsc': 'ENST00000393606.7:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000377231.3:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000377231',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000436088',\n",
       "       'mane_select': 'NM_014233.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/21',\n",
       "       'hgvsc': 'ENST00000436088.6:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000390669.1:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000390669',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000526094',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/19',\n",
       "       'hgvsc': 'ENST00000526094.5:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000432925.1:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000432925',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000527034',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/20',\n",
       "       'hgvsc': 'ENST00000527034.5:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000431539.1:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000431539',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000529373',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000431295',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000529383',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/20',\n",
       "       'hgvsc': 'ENST00000529383.5:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000435708.1:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000435708',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000530828',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000433046',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'UBTF',\n",
       "       'gene': 'ENSG00000108312',\n",
       "       'feature': 'ENST00000533177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/20',\n",
       "       'hgvsc': 'ENST00000533177.5:c.628G>A',\n",
       "       'hgvsp': 'ENSP00000437180.1:p.Glu210Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000437180',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_904_1': 'WT', 'RGP_904_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic',\n",
       "     'Regression'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_904_3',\n",
       "    'family': 'RGP_904',\n",
       "    'gene': 'ENSG00000258366',\n",
       "    'var_data': {'coords': {'chrom': '20',\n",
       "      'pos': 63694739,\n",
       "      'ref': 'A',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 12.25,\n",
       "      'baseqranksum': 1.0199999809265137,\n",
       "      'db': True,\n",
       "      'dp': 58764,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 4.998000144958496,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.4749999940395355,\n",
       "      'mq_dp': 100,\n",
       "      'qd': 12.25,\n",
       "      'raw_mq': 360000.0,\n",
       "      'readposranksum': 2.640000104904175,\n",
       "      'sor': 0.5709999799728394,\n",
       "      'vqslod': 12.449999809265137,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 549409,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 4.183019882475492e-06,\n",
       "      'gnomad_ex_an': 239062,\n",
       "      'gnomad_ex_ac': 1,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 3.9426999137504026e-05,\n",
       "      'gnomad_an': 152180,\n",
       "      'gnomad_ac': 6,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.8999999761581421,\n",
       "      'splice_ai_csq': 'Acceptor_loss',\n",
       "      'cadd': 25.5,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000258366'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_acceptor_variant',\n",
       "       'symbol': 'RTEL1',\n",
       "       'gene': 'ENSG00000258366',\n",
       "       'feature': 'ENST00000318100',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000318100.9:c.2441-2A>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000322287',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_acceptor_variant',\n",
       "       'symbol': 'RTEL1',\n",
       "       'gene': 'ENSG00000258366',\n",
       "       'feature': 'ENST00000360203',\n",
       "       'mane_select': 'NM_001283009.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000360203.11:c.3110-2A>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000353332',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_acceptor_variant',\n",
       "       'symbol': 'RTEL1',\n",
       "       'gene': 'ENSG00000258366',\n",
       "       'feature': 'ENST00000370018',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000370018.7:c.3110-2A>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000359035',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'RTEL1',\n",
       "       'gene': 'ENSG00000258366',\n",
       "       'feature': 'ENST00000425905',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000388063',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_acceptor_variant',\n",
       "       'symbol': 'RTEL1',\n",
       "       'gene': 'ENSG00000258366',\n",
       "       'feature': 'ENST00000508582',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000508582.7:c.3182-2A>T',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000424307',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3', '5']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_904_1': 'Het', 'RGP_904_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027621_rgp_904',\n",
       "   'family_id': 'RGP_904',\n",
       "   'members': {'RGP_904_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F027621_rgp_904'},\n",
       "    'RGP_904_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027621_rgp_904'}},\n",
       "   'phenotypes': ['HP:0002376', 'HP:0001249', 'HP:0002059', 'HP:0001344'],\n",
       "   'panel_ids': [137, 250, 206],\n",
       "   'panel_names': ['Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Regression']}},\n",
       " 'RGP_910_3': {'variants': [{'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000162430',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 25808786,\n",
       "      'ref': 'GGAGGTGAGGACAGCTGGGGTGCGACGTGGGGCCCCTCC',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 13.199999809265137,\n",
       "      'baseqranksum': 1.5199999809265137,\n",
       "      'dp': 56861,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.2409999966621399,\n",
       "      'mq_dp': 45,\n",
       "      'qd': 13.210000038146973,\n",
       "      'raw_mq': 162000.0,\n",
       "      'readposranksum': 0.041999999433755875,\n",
       "      'sor': 0.7450000047683716,\n",
       "      'vqslod': 2.9000000953674316,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918201,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'missing',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162430'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000354177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000354177.9:c.644_645+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000346109',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000361547',\n",
       "       'mane_select': 'NM_020451.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/13',\n",
       "       'hgvsc': 'ENST00000361547.7:c.746_747+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000355141',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000374315',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000374315.1:c.644_645+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000363434',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Myopathy - paediatric onset',\n",
       "     'Muscular dystrophy_Paediatric',\n",
       "     'Myopathy Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000162430',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 25808786,\n",
       "      'ref': 'GGAGGTGAGGACAGCTGGGGTGCGACGTGGGGCCCCTCC',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 13.199999809265137,\n",
       "      'baseqranksum': 1.5199999809265137,\n",
       "      'dp': 56861,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 0.0,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.2409999966621399,\n",
       "      'mq_dp': 45,\n",
       "      'qd': 13.210000038146973,\n",
       "      'raw_mq': 162000.0,\n",
       "      'readposranksum': 0.041999999433755875,\n",
       "      'sor': 0.7450000047683716,\n",
       "      'vqslod': 2.9000000953674316,\n",
       "      'culprit': 'FS',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 918201,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'missing',\n",
       "      'cadd': 0.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162430'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000354177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000354177.9:c.644_645+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000346109',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000361547',\n",
       "       'mane_select': 'NM_020451.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/13',\n",
       "       'hgvsc': 'ENST00000361547.7:c.746_747+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000355141',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'splice_donor_variant&splice_donor_5th_base_variant&coding_sequence_variant&intron_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000374315',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/12',\n",
       "       'hgvsc': 'ENST00000374315.1:c.644_645+36del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000363434',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['1-25813890-G-A'],\n",
       "    'flags': ['Myopathy - paediatric onset',\n",
       "     'Muscular dystrophy_Paediatric',\n",
       "     'Myopathy Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000162430',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 25813890,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 3,\n",
       "      'af': 0.0010179999517276883,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.0010999999940395355,\n",
       "      'as_qd': 15.529999732971191,\n",
       "      'baseqranksum': 0.7990000247955322,\n",
       "      'db': True,\n",
       "      'dp': 53953,\n",
       "      'excesshet': 3.014699935913086,\n",
       "      'fs': 1.406000018119812,\n",
       "      'inbreedingcoeff': -0.0010999999940395355,\n",
       "      'mleac': 3,\n",
       "      'mleaf': 0.0010179999517276883,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.12200000137090683,\n",
       "      'mq_dp': 130,\n",
       "      'qd': 15.529999732971191,\n",
       "      'raw_mq': 468000.0,\n",
       "      'readposranksum': 0.5870000123977661,\n",
       "      'sor': 0.5690000057220459,\n",
       "      'vqslod': 14.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 19531,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 4.8135199904209e-05,\n",
       "      'gnomad_ex_an': 249298,\n",
       "      'gnomad_ex_ac': 12,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 5.9144400438526645e-05,\n",
       "      'gnomad_an': 152170,\n",
       "      'gnomad_ac': 9,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 24.200000762939453,\n",
       "      'revel': 0.7289999723434448,\n",
       "      'mutationtaster': 'A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162430'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000354177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/12',\n",
       "       'hgvsc': 'ENST00000354177.9:c.1226G>A',\n",
       "       'hgvsp': 'ENSP00000346109.5:p.Arg409Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000346109',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000361547',\n",
       "       'mane_select': 'NM_020451.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/13',\n",
       "       'hgvsc': 'ENST00000361547.7:c.1397G>A',\n",
       "       'hgvsp': 'ENSP00000355141.2:p.Arg466Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000355141',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000374315',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/12',\n",
       "       'hgvsc': 'ENST00000374315.1:c.1295G>A',\n",
       "       'hgvsp': 'ENSP00000363434.1:p.Arg432Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000363434',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Myopathy - paediatric onset',\n",
       "     'Muscular dystrophy_Paediatric',\n",
       "     'Myopathy Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000162430',\n",
       "    'var_data': {'coords': {'chrom': '1',\n",
       "      'pos': 25813890,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 3,\n",
       "      'af': 0.0010179999517276883,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.0010999999940395355,\n",
       "      'as_qd': 15.529999732971191,\n",
       "      'baseqranksum': 0.7990000247955322,\n",
       "      'db': True,\n",
       "      'dp': 53953,\n",
       "      'excesshet': 3.014699935913086,\n",
       "      'fs': 1.406000018119812,\n",
       "      'inbreedingcoeff': -0.0010999999940395355,\n",
       "      'mleac': 3,\n",
       "      'mleaf': 0.0010179999517276883,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.12200000137090683,\n",
       "      'mq_dp': 130,\n",
       "      'qd': 15.529999732971191,\n",
       "      'raw_mq': 468000.0,\n",
       "      'readposranksum': 0.5870000123977661,\n",
       "      'sor': 0.5690000057220459,\n",
       "      'vqslod': 14.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 19531,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 4.8135199904209e-05,\n",
       "      'gnomad_ex_an': 249298,\n",
       "      'gnomad_ex_ac': 12,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 5.9144400438526645e-05,\n",
       "      'gnomad_an': 152170,\n",
       "      'gnomad_ac': 9,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 24.200000762939453,\n",
       "      'revel': 0.7289999723434448,\n",
       "      'mutationtaster': 'A,A,A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000162430'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000354177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/12',\n",
       "       'hgvsc': 'ENST00000354177.9:c.1226G>A',\n",
       "       'hgvsp': 'ENSP00000346109.5:p.Arg409Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000346109',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000361547',\n",
       "       'mane_select': 'NM_020451.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/13',\n",
       "       'hgvsc': 'ENST00000361547.7:c.1397G>A',\n",
       "       'hgvsp': 'ENSP00000355141.2:p.Arg466Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000355141',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SELENON',\n",
       "       'gene': 'ENSG00000162430',\n",
       "       'feature': 'ENST00000374315',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/12',\n",
       "       'hgvsc': 'ENST00000374315.1:c.1295G>A',\n",
       "       'hgvsp': 'ENSP00000363434.1:p.Arg432Gln',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000363434',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Recessive Compound-Het'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': True,\n",
       "    'support_vars': ['1-25808786-GGAGGTGAGGACAGCTGGGGTGCGACGTGGGGCCCCTCC-G'],\n",
       "    'flags': ['Myopathy - paediatric onset',\n",
       "     'Muscular dystrophy_Paediatric',\n",
       "     'Myopathy Superpanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000213689',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 48467070,\n",
       "      'ref': 'GC',\n",
       "      'alt': 'G'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00039999998989515007,\n",
       "      'as_qd': 11.9399995803833,\n",
       "      'baseqranksum': -0.6869999766349792,\n",
       "      'dp': 55998,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 5.940000057220459,\n",
       "      'inbreedingcoeff': -0.00039999998989515007,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.2919999957084656,\n",
       "      'mq_dp': 52,\n",
       "      'qd': 11.949999809265137,\n",
       "      'raw_mq': 187200.0,\n",
       "      'readposranksum': 1.1200000047683716,\n",
       "      'sor': 0.9459999799728394,\n",
       "      'vqslod': -0.24410000443458557,\n",
       "      'culprit': 'SOR',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 828240,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 2.788580059132073e-05,\n",
       "      'gnomad_ex_an': 251024,\n",
       "      'gnomad_ex_ac': 7,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 3.284939884906635e-05,\n",
       "      'gnomad_an': 152210,\n",
       "      'gnomad_ac': 5,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.009999999776482582,\n",
       "      'splice_ai_csq': 'Donor_gain',\n",
       "      'cadd': 26.399999618530273,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000213689'},\n",
       "     'transcript_consequences': [{'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000433541',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '4/4',\n",
       "       'hgvsc': 'ENST00000433541.1:c.-2del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000412404',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000444177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000444177.1:c.386del',\n",
       "       'hgvsp': 'ENSP00000415972.1:p.Ala129ValfsTer21',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000415972',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000456089',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000456089.1:c.-2del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000411331',\n",
       "       'lof': ''},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000492235',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000492235.2:c.-2del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000494511',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'frameshift_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000625293',\n",
       "       'mane_select': 'NM_033629.6',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000625293.3:c.416del',\n",
       "       'hgvsp': 'ENSP00000486676.2:p.Ala139ValfsTer21',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000486676',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': '5_prime_UTR_variant',\n",
       "       'symbol': 'TREX1',\n",
       "       'gene': 'ENSG00000213689',\n",
       "       'feature': 'ENST00000635452',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/2',\n",
       "       'hgvsc': 'ENST00000635452.2:c.-2del',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'deletion',\n",
       "       'ensp': 'ENSP00000492023',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000187098',\n",
       "    'var_data': {'coords': {'chrom': '3',\n",
       "      'pos': 69964940,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 2,\n",
       "      'af': 0.000678399985190481,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.000699999975040555,\n",
       "      'as_qd': 11.970000267028809,\n",
       "      'baseqranksum': 0.11599999666213989,\n",
       "      'db': True,\n",
       "      'dp': 50805,\n",
       "      'excesshet': 3.0118000507354736,\n",
       "      'fs': 2.059000015258789,\n",
       "      'inbreedingcoeff': -0.000699999975040555,\n",
       "      'mleac': 2,\n",
       "      'mleaf': 0.000678399985190481,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 1.159999966621399,\n",
       "      'mq_dp': 65,\n",
       "      'qd': 11.960000038146973,\n",
       "      'raw_mq': 234000.0,\n",
       "      'readposranksum': 1.159999966621399,\n",
       "      'sor': 0.45500001311302185,\n",
       "      'vqslod': 14.649999618530273,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 38747,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0013604500563815236,\n",
       "      'gnomad_ex_an': 251388,\n",
       "      'gnomad_ex_ac': 342,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0015777599764987826,\n",
       "      'gnomad_an': 152114,\n",
       "      'gnomad_ac': 240,\n",
       "      'gnomad_hom': 1,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 28.100000381469727,\n",
       "      'revel': 0.43799999356269836,\n",
       "      'mutationtaster': 'D,D,D,D,D,D,D,D,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000187098'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000314557',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/9',\n",
       "       'hgvsc': 'ENST00000314557.10:c.934G>A',\n",
       "       'hgvsp': 'ENSP00000324246.6:p.Glu312Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000324246',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000314589',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000314589.10:c.1207G>A',\n",
       "       'hgvsp': 'ENSP00000324443.5:p.Glu403Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000324443',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000352241',\n",
       "       'mane_select': 'NM_001354604.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000352241.9:c.1273G>A',\n",
       "       'hgvsp': 'ENSP00000295600.8:p.Glu425Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000295600',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000394351',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/9',\n",
       "       'hgvsc': 'ENST00000394351.9:c.952G>A',\n",
       "       'hgvsp': 'ENSP00000377880.3:p.Glu318Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000377880',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000448226',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000448226.9:c.1252G>A',\n",
       "       'hgvsp': 'ENSP00000391803.3:p.Glu418Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000391803',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000472437',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000472437.5:c.1099G>A',\n",
       "       'hgvsp': 'ENSP00000418845.1:p.Glu367Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418845',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000531774',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/9',\n",
       "       'hgvsc': 'ENST00000531774.1:c.766G>A',\n",
       "       'hgvsp': 'ENSP00000435909.1:p.Glu256Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000435909',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000642352',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000642352.1:c.1255G>A',\n",
       "       'hgvsp': 'ENSP00000494105.1:p.Glu419Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000494105',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000687384',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/11',\n",
       "       'hgvsc': 'ENST00000687384.1:c.1204G>A',\n",
       "       'hgvsp': 'ENSP00000510225.1:p.Glu402Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000510225',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000693031',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/11',\n",
       "       'hgvsc': 'ENST00000693031.1:c.1180G>A',\n",
       "       'hgvsp': 'ENSP00000509845.1:p.Glu394Lys',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509845',\n",
       "       'lof': ''},\n",
       "      {'consequence': '3_prime_UTR_variant',\n",
       "       'symbol': 'MITF',\n",
       "       'gene': 'ENSG00000187098',\n",
       "       'feature': 'ENST00000693549',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '10/10',\n",
       "       'hgvsc': 'ENST00000693549.1:c.*18G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000509358',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_910_3',\n",
       "    'family': 'RGP_910',\n",
       "    'gene': 'ENSG00000164266',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 147828115,\n",
       "      'ref': 'T',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 20,\n",
       "      'af': 0.006783999968320131,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.02449999935925007,\n",
       "      'as_qd': 12.130000114440918,\n",
       "      'baseqranksum': 0.6790000200271606,\n",
       "      'db': True,\n",
       "      'dp': 51018,\n",
       "      'excesshet': 3.292099952697754,\n",
       "      'fs': 1.093999981880188,\n",
       "      'inbreedingcoeff': 0.02449999935925007,\n",
       "      'mleac': 20,\n",
       "      'mleaf': 0.006783999968320131,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.40299999713897705,\n",
       "      'mq_dp': 729,\n",
       "      'positive_train_site': True,\n",
       "      'qd': 12.130000114440918,\n",
       "      'raw_mq': 2624400.0,\n",
       "      'readposranksum': 0.5070000290870667,\n",
       "      'sor': 0.8069999814033508,\n",
       "      'vqslod': 15.890000343322754,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 28799,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.009062569588422775,\n",
       "      'gnomad_ex_an': 249598,\n",
       "      'gnomad_ex_ac': 2262,\n",
       "      'gnomad_ex_hom': 20,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.008252410218119621,\n",
       "      'gnomad_an': 152198,\n",
       "      'gnomad_ac': 1256,\n",
       "      'gnomad_hom': 7,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 0.0010000000474974513,\n",
       "      'revel': 0.19900000095367432,\n",
       "      'mutationtaster': 'N,N',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000164266'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'SPINK1',\n",
       "       'gene': 'ENSG00000164266',\n",
       "       'feature': 'ENST00000296695',\n",
       "       'mane_select': 'NM_001379610.1',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/4',\n",
       "       'hgvsc': 'ENST00000296695.10:c.101A>G',\n",
       "       'hgvsp': 'ENSP00000296695.5:p.Asn34Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000296695',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'missense_variant',\n",
       "       'symbol': 'SPINK1',\n",
       "       'gene': 'ENSG00000164266',\n",
       "       'feature': 'ENST00000510027',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '3/3',\n",
       "       'hgvsc': 'ENST00000510027.2:c.101A>G',\n",
       "       'hgvsp': 'ENSP00000427376.1:p.Asn34Ser',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000427376',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_910_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027630_rgp_910',\n",
       "   'family_id': 'RGP_910',\n",
       "   'members': {'RGP_910_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027630_rgp_910'}},\n",
       "   'phenotypes': ['HP:0003560', 'HP:0001324', 'HP:0002093', 'HP:0009055'],\n",
       "   'panel_ids': [3101, 328, 137, 141, 329],\n",
       "   'panel_names': ['Myopathy Superpanel',\n",
       "    'Myopathy - adult onset',\n",
       "    'Mendeliome',\n",
       "    'Muscular dystrophy_Paediatric',\n",
       "    'Myopathy - paediatric onset']}},\n",
       " 'RGP_930_3': {'variants': [{'sample': 'RGP_930_3',\n",
       "    'family': 'RGP_930',\n",
       "    'gene': 'ENSG00000175084',\n",
       "    'var_data': {'coords': {'chrom': '2',\n",
       "      'pos': 219421350,\n",
       "      'ref': 'T',\n",
       "      'alt': 'C'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.3084000051021576,\n",
       "      'as_qd': 13.920000076293945,\n",
       "      'baseqranksum': 1.100000023841858,\n",
       "      'db': True,\n",
       "      'dp': 52322,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 1.3029999732971191,\n",
       "      'inbreedingcoeff': 0.3084000051021576,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.9210000038146973,\n",
       "      'mq_dp': 38,\n",
       "      'qd': 13.930000305175781,\n",
       "      'raw_mq': 136800.0,\n",
       "      'readposranksum': 2.4100000858306885,\n",
       "      'sor': 1.1150000095367432,\n",
       "      'vqslod': 14.380000114440918,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 31864,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.0,\n",
       "      'splice_ai_csq': 'No_consequence',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.9819999933242798,\n",
       "      'mutationtaster': 'A',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': True,\n",
       "      'gene_id': 'ENSG00000175084'},\n",
       "     'transcript_consequences': [{'consequence': 'missense_variant',\n",
       "       'symbol': 'DES',\n",
       "       'gene': 'ENSG00000175084',\n",
       "       'feature': 'ENST00000373960',\n",
       "       'mane_select': 'NM_001927.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/9',\n",
       "       'hgvsc': 'ENST00000373960.4:c.1034T>C',\n",
       "       'hgvsp': 'ENSP00000363071.3:p.Leu345Pro',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000363071',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', 'support']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_930_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Myopathy - adult onset',\n",
       "     'Arrhythmogenic Cardiomyopathy',\n",
       "     'Myopathy Superpanel',\n",
       "     'Arrhythmia_SuperPanel'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027646_rgp_930',\n",
       "   'family_id': 'RGP_930',\n",
       "   'members': {'RGP_930_3': {'sex': 'male',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027646_rgp_930'}},\n",
       "   'phenotypes': ['HP:0011675', 'HP:0001324', 'HP:0009055'],\n",
       "   'panel_ids': [48, 329, 328, 137, 3101, 254],\n",
       "   'panel_names': ['Arrhythmogenic Cardiomyopathy',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Myopathy - adult onset',\n",
       "    'Mendeliome',\n",
       "    'Myopathy Superpanel',\n",
       "    'Arrhythmia_SuperPanel']}},\n",
       " 'RGP_956_3': {'variants': [{'sample': 'RGP_956_3',\n",
       "    'family': 'RGP_956',\n",
       "    'gene': 'ENSG00000169247',\n",
       "    'var_data': {'coords': {'chrom': '5',\n",
       "      'pos': 149026872,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 3,\n",
       "      'af': 0.0010179999517276883,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.15950000286102295,\n",
       "      'as_qd': 14.800000190734863,\n",
       "      'baseqranksum': 1.0299999713897705,\n",
       "      'db': True,\n",
       "      'dp': 53431,\n",
       "      'excesshet': 3.014699935913086,\n",
       "      'fs': 10.439000129699707,\n",
       "      'inbreedingcoeff': 0.15950000286102295,\n",
       "      'mleac': 3,\n",
       "      'mleaf': 0.0010179999517276883,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': -0.18199999630451202,\n",
       "      'mq_dp': 115,\n",
       "      'qd': 14.800000190734863,\n",
       "      'raw_mq': 414000.0,\n",
       "      'readposranksum': 0.7120000123977661,\n",
       "      'sor': 1.0679999589920044,\n",
       "      'vqslod': 14.899999618530273,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 17521,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.000748021004255861,\n",
       "      'gnomad_ex_an': 251330,\n",
       "      'gnomad_ex_ac': 188,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.000643990992102772,\n",
       "      'gnomad_an': 152176,\n",
       "      'gnomad_ac': 98,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.15000000596046448,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 41.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,A,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000169247'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'SH3TC2',\n",
       "       'gene': 'ENSG00000169247',\n",
       "       'feature': 'ENST00000512049',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000512049.5:c.2839C>T',\n",
       "       'hgvsp': 'ENSP00000421860.1:p.Arg947Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000421860',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'SH3TC2',\n",
       "       'gene': 'ENSG00000169247',\n",
       "       'feature': 'ENST00000515425',\n",
       "       'mane_select': 'NM_024577.4',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '11/17',\n",
       "       'hgvsc': 'ENST00000515425.6:c.2860C>T',\n",
       "       'hgvsp': 'ENSP00000423660.1:p.Arg954Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000423660',\n",
       "       'lof': 'HC'}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_956_1': 'WT', 'RGP_956_2': 'Het', 'RGP_956_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_956_3',\n",
       "    'family': 'RGP_956',\n",
       "    'gene': 'ENSG00000167508',\n",
       "    'var_data': {'coords': {'chrom': '16',\n",
       "      'pos': 88663006,\n",
       "      'ref': 'C',\n",
       "      'alt': 'T'},\n",
       "     'info': {'ac': 8,\n",
       "      'af': 0.002713999943807721,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': -0.00279999990016222,\n",
       "      'as_qd': 12.460000038146973,\n",
       "      'baseqranksum': -0.03200000151991844,\n",
       "      'db': True,\n",
       "      'dp': 59648,\n",
       "      'excesshet': 3.0517001152038574,\n",
       "      'fs': 3.191999912261963,\n",
       "      'inbreedingcoeff': -0.00279999990016222,\n",
       "      'mleac': 8,\n",
       "      'mleaf': 0.002713999943807721,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.017000000923871994,\n",
       "      'mq_dp': 434,\n",
       "      'qd': 12.460000038146973,\n",
       "      'raw_mq': 1562400.0,\n",
       "      'readposranksum': 0.9789999723434448,\n",
       "      'sor': 0.7110000252723694,\n",
       "      'vqslod': 12.649999618530273,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 1336585,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0019698599353432655,\n",
       "      'gnomad_ex_an': 225904,\n",
       "      'gnomad_ex_ac': 445,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0029553500935435295,\n",
       "      'gnomad_an': 152266,\n",
       "      'gnomad_ac': 450,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.8600000143051147,\n",
       "      'splice_ai_csq': 'Donor_loss',\n",
       "      'cadd': 32.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'missing',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': False,\n",
       "      'categoryboolean5': True,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000167508'},\n",
       "     'transcript_consequences': [{'consequence': 'splice_donor_5th_base_variant&intron_variant',\n",
       "       'symbol': 'MVD',\n",
       "       'gene': 'ENSG00000167508',\n",
       "       'feature': 'ENST00000301012',\n",
       "       'mane_select': 'NM_002461.3',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000301012.8:c.70+5G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000301012',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'splice_donor_5th_base_variant&intron_variant',\n",
       "       'symbol': 'MVD',\n",
       "       'gene': 'ENSG00000167508',\n",
       "       'feature': 'ENST00000569177',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': 'ENST00000569177.5:c.70+5G>A',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000455131',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '5']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_956_1': 'Het', 'RGP_956_2': 'WT', 'RGP_956_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': [],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'},\n",
       "   {'sample': 'RGP_956_3',\n",
       "    'family': 'RGP_956',\n",
       "    'gene': 'ENSG00000196998',\n",
       "    'var_data': {'coords': {'chrom': 'X',\n",
       "      'pos': 49075573,\n",
       "      'ref': 'G',\n",
       "      'alt': 'A'},\n",
       "     'info': {'ac': 1,\n",
       "      'af': 0.0003391999925952405,\n",
       "      'an': 2948,\n",
       "      'as_inbreedingcoeff': 0.5623000264167786,\n",
       "      'as_qd': 15.770000457763672,\n",
       "      'baseqranksum': 0.824999988079071,\n",
       "      'dp': 42715,\n",
       "      'excesshet': 3.0102999210357666,\n",
       "      'fs': 7.9019999504089355,\n",
       "      'inbreedingcoeff': 0.5622000098228455,\n",
       "      'mleac': 1,\n",
       "      'mleaf': 0.0003391999925952405,\n",
       "      'mq': 60.0,\n",
       "      'mqranksum': 0.21899999678134918,\n",
       "      'mq_dp': 35,\n",
       "      'qd': 15.779999732971191,\n",
       "      'raw_mq': 126000.0,\n",
       "      'readposranksum': 0.45500001311302185,\n",
       "      'sor': 0.12399999797344208,\n",
       "      'vqslod': 14.109999656677246,\n",
       "      'culprit': 'MQ',\n",
       "      'clinvar_sig': 'Pathogenic',\n",
       "      'clinvar_stars': 1,\n",
       "      'clinvar_allele': 51081,\n",
       "      'clinvar_aip': 1,\n",
       "      'clinvar_aip_strong': 1,\n",
       "      'gnomad_ex_af': 0.0,\n",
       "      'gnomad_ex_an': 0,\n",
       "      'gnomad_ex_ac': 0,\n",
       "      'gnomad_ex_hom': 0,\n",
       "      'gnomad_ex_hemi': 0,\n",
       "      'gnomad_af': 0.0,\n",
       "      'gnomad_an': 0,\n",
       "      'gnomad_ac': 0,\n",
       "      'gnomad_hom': 0,\n",
       "      'gnomad_hemi': 0,\n",
       "      'splice_ai_delta': 0.019999999552965164,\n",
       "      'splice_ai_csq': 'Acceptor_gain',\n",
       "      'cadd': 35.0,\n",
       "      'revel': 0.0,\n",
       "      'mutationtaster': 'A,A,A,A,A,A,A,D',\n",
       "      'categoryboolean1': True,\n",
       "      'categoryboolean2': False,\n",
       "      'categoryboolean3': True,\n",
       "      'categoryboolean5': False,\n",
       "      'categorysample4': [],\n",
       "      'categorysupport': False,\n",
       "      'gene_id': 'ENSG00000196998'},\n",
       "     'transcript_consequences': [{'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000322995',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/11',\n",
       "       'hgvsc': 'ENST00000322995.13:c.730C>T',\n",
       "       'hgvsp': 'ENSP00000365543.5:p.Arg244Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000365543',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000356463',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '9/12',\n",
       "       'hgvsc': 'ENST00000356463.7:c.700C>T',\n",
       "       'hgvsp': 'ENSP00000348848.3:p.Arg234Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000348848',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000367375',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/8',\n",
       "       'hgvsc': 'ENST00000367375.8:c.478C>T',\n",
       "       'hgvsp': 'ENSP00000356345.4:p.Arg160Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000356345',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000376368',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/11',\n",
       "       'hgvsc': 'ENST00000376368.7:c.700C>T',\n",
       "       'hgvsp': 'ENSP00000365546.2:p.Arg234Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000365546',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000376372',\n",
       "       'mane_select': 'NM_001029896.2',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/11',\n",
       "       'hgvsc': 'ENST00000376372.9:c.697C>T',\n",
       "       'hgvsp': 'ENSP00000365551.3:p.Arg233Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000365551',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000396681',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/10',\n",
       "       'hgvsc': 'ENST00000396681.9:c.592C>T',\n",
       "       'hgvsp': 'ENSP00000379913.5:p.Arg198Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000379913',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000419567',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000393640',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000423215',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000397657',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000465382',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420534',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000471338',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418466',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000473974',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/9',\n",
       "       'hgvsc': 'ENST00000473974.5:c.697C>T',\n",
       "       'hgvsp': 'ENSP00000417211.1:p.Arg233Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417211',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000474053',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000420728',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000475880',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/8',\n",
       "       'hgvsc': 'ENST00000475880.6:c.595C>T',\n",
       "       'hgvsp': 'ENSP00000418919.2:p.Arg199Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000418919',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000475977',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '2/4',\n",
       "       'hgvsc': 'ENST00000475977.2:c.193C>T',\n",
       "       'hgvsp': 'ENSP00000417754.2:p.Arg65Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000417754',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000476728',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419324',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000485908',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/10',\n",
       "       'hgvsc': 'ENST00000485908.6:c.592C>T',\n",
       "       'hgvsp': 'ENSP00000419897.1:p.Arg198Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419897',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'upstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000486337',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000419453',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634559',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/9',\n",
       "       'hgvsc': 'ENST00000634559.1:c.496C>T',\n",
       "       'hgvsp': 'ENSP00000488986.1:p.Arg166Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000488986',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634671',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489040',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634736',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '5/7',\n",
       "       'hgvsc': 'ENST00000634736.1:c.391C>T',\n",
       "       'hgvsp': 'ENSP00000489561.1:p.Arg131Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489561',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634838',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/11',\n",
       "       'hgvsc': 'ENST00000634838.1:c.697C>T',\n",
       "       'hgvsp': 'ENSP00000489268.1:p.Arg233Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489268',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634849',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489150',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000634944',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '8/11',\n",
       "       'hgvsc': 'ENST00000634944.1:c.697C>T',\n",
       "       'hgvsp': 'ENSP00000488972.1:p.Arg233Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000488972',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000635003',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '6/9',\n",
       "       'hgvsc': 'ENST00000635003.1:c.496C>T',\n",
       "       'hgvsp': 'ENSP00000489080.1:p.Arg166Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489080',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'stop_gained',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000635666',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '7/9',\n",
       "       'hgvsc': 'ENST00000635666.1:c.625C>T',\n",
       "       'hgvsp': 'ENSP00000489128.1:p.Arg209Ter',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000489128',\n",
       "       'lof': 'HC'},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000636049',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490378',\n",
       "       'lof': ''},\n",
       "      {'consequence': 'downstream_gene_variant',\n",
       "       'symbol': 'WDR45',\n",
       "       'gene': 'ENSG00000196998',\n",
       "       'feature': 'ENST00000636645',\n",
       "       'mane_select': '',\n",
       "       'biotype': 'protein_coding',\n",
       "       'exon': '',\n",
       "       'hgvsc': '',\n",
       "       'hgvsp': '',\n",
       "       'variant_class': 'SNV',\n",
       "       'ensp': 'ENSP00000490130',\n",
       "       'lof': ''}],\n",
       "     'phased': {},\n",
       "     'categories': ['1', '3']},\n",
       "    'reasons': ['Autosomal Dominant'],\n",
       "    'genotypes': {'RGP_956_1': 'WT', 'RGP_956_2': 'WT', 'RGP_956_3': 'Het'},\n",
       "    'supported': False,\n",
       "    'support_vars': [],\n",
       "    'flags': ['Early-onset Dementia',\n",
       "     'Genetic Epilepsy',\n",
       "     'Intellectual disability syndromic and non-syndromic'],\n",
       "    'phenotypes': [],\n",
       "    'first_seen': '2023-01-31'}],\n",
       "  'metadata': {'ext_id': 'F027745_rgp_956',\n",
       "   'family_id': 'RGP_956',\n",
       "   'members': {'RGP_956_1': {'sex': 'female',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F027745_rgp_956'},\n",
       "    'RGP_956_2': {'sex': 'male',\n",
       "     'affected': False,\n",
       "     'ext_id': 'F027745_rgp_956'},\n",
       "    'RGP_956_3': {'sex': 'female',\n",
       "     'affected': True,\n",
       "     'ext_id': 'F027745_rgp_956'}},\n",
       "   'phenotypes': ['HP:0100543',\n",
       "    'HP:0001260',\n",
       "    'HP:0001263',\n",
       "    'HP:0000821',\n",
       "    'HP:0001252',\n",
       "    'HP:0010865',\n",
       "    'HP:0002370',\n",
       "    'HP:0001250',\n",
       "    'HP:0000505'],\n",
       "   'panel_ids': [3796, 3078, 202, 24, 137, 250, 329, 3471],\n",
       "   'panel_names': ['Familial Generalised Epilepsy',\n",
       "    'Congenital Myasthenia',\n",
       "    'Genetic Epilepsy',\n",
       "    'Early-onset Dementia',\n",
       "    'Mendeliome',\n",
       "    'Intellectual disability syndromic and non-syndromic',\n",
       "    'Myopathy - paediatric onset',\n",
       "    'Congenital hypothyroidism']}}}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "aip_results['results']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "aip",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.6"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "76942bb33a17c5e75269ab1a60bfa6814cd5a4e177669d2f23fc673192dea4db"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
